%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Preamble
\documentclass{article}

%Dependencies
\usepackage[left]{lineno}
\usepackage{titlesec}
\usepackage{xcolor}

\newcommand\hl[1]{%
  \bgroup
  \hskip0pt\color{blue!80!black}%
  #1%
  \egroup
}
\usepackage{ogonek}
\usepackage{float}
%\usepackage{times}
\usepackage{amsmath}
\usepackage{old-arrows}
\usepackage{enumitem}
\usepackage{wasysym}
\usepackage[titletoc,title]{appendix}
\usepackage{ulem}

% Other Packages
\usepackage{times}
\RequirePackage{fullpage}
\linespread{1.2}
\RequirePackage[colorlinks=true, allcolors=black, linktoc=page]{hyperref}
\RequirePackage[english]{babel}
\RequirePackage{amsmath,amsfonts,amssymb}
\usepackage{mathptmx}
\RequirePackage[sc]{mathpazo}
\RequirePackage[T1]{fontenc}
\RequirePackage{url}
\usepackage{tabu}

% Bibliography
%\usepackage[authoryear,sectionbib,sort]{natbib}
\usepackage{natbib} \bibpunct{(}{)}{;}{author-year}{}{,}
\bibliographystyle{evolution}
\addto{\captionsenglish}{\renewcommand{\refname}{References}}
\setlength{\bibsep}{0.0pt}

% Graphics package
\usepackage{graphicx}
\graphicspath{{../../output/figures/}.pdf}

% Change Appendix Numbering
%\renewcommand\thesection{Appendix \Alph{section}}
%\addcontentsline{toc,page}{section}{Supplement~\ref{section}}
%\renewcommand\thesection{S\arabic{section}}
%\renewcommand\thesubsection{S\arabic{section}.\arabic{subsection}}
\addcontentsline{toc,page}{section}{Appendix~\ref{section}}
\renewcommand\thesubsection{\Alph{section}.\arabic{subsection}}

% New commands: fonts
\def\mbf#1{\mathbf{#1}}

%\newcommand{\code}{\fontfamily{pcr}\selectfont}
%\newcommand*\chem[1]{\ensuremath{\mathrm{#1}}}
\newcommand\numberthis{\addtocounter{equation}{1}\tag{\theequation}}
%\titleformat{\subsubsection}[runin]{\bfseries\itshape}{\thesubsubsection.}{0.5em}{}




%%%%%%%%%%%%%%%%%%%%%
% Header
%%%%%%%%%%%%%%%%%%%%%
%
% Customize the line below with the last name of your first author and
% the short title of your MS. You can comment authorship information out
% while your MS is undergoing double-blind review.
%
%\lhead{\thesection}
%\rhead{Olito \& Abbott, "Inversions on Proto Sex Chromosomes"}
%\setlength{\headheight}{15pt}
%\setlength{\headsep}{0.15in}  

%%%%%%%%%%%%%%%%%%%%%
% Line numbering
%%%%%%%%%%%%%%%%%%%%%
%
% Please use line numbering with your initial submission and
% subsequent revisions. After acceptance, please turn line numbering
% off by adding percent signs to the lines %\usepackage{lineno} and
% to %\linenumbers{} and %\modulolinenumbers[3] below.
%
% To avoid line numbering being thrown off around math environments,
% the math environments have to be wrapped using
% \begin{linenomath*} and \end{linenomath*}
%
% (Thanks to Vlastimil Krivan for pointing this out to us!)
\begin{document}

\title{Supporting Information (Appendixes A--D) for: The evolution of suppressed recombination between sex chromosomes by chromosomal inversions.\\ 
%\LaTeX{} Template for Author-Supplied Supplementary PDFs, \\ 
\textit{Evolution}}

% This version of the LaTeX supplementary template was last updated on
% November 8, 2019.

%%%%%%%%%%%%%%%%%%%%%
% Authorship
%%%%%%%%%%%%%%%%%%%%%
% Please remove authorship information while your paper is under review,
% unless you wish to waive your anonymity under double-blind review. You
% will need to add this information back in to your final files after
% acceptance.
%
% Once accepted for publication, author-supplied PDFs should have a 
% title page that includes (at least) the authors' names, the title of 
% the MS, and the name of the journal. It should also have a header and
% page numbers.

%\author{Colin Olito$^{\ast}$ \\ 
%Jessica K.~Abbott}
\date{\today}
\maketitle

%\noindent Department of Biology, Lund University, Lund 223 62, Sweden. \\
%\noindent $\ast$ Corresponding author; e-mail: \url{colin.olito@gmail.com}\\

%\noindent ORCiDs\\
%\noindent CO: https://orcid.org/0000-0001-6883-0367\\
%\noindent JKA: https://orcid.org/0000-0002-8743-2089


\bigskip
%\linenumbers{}
%\modulolinenumbers[3]

\newpage{}
\tableofcontents
\newpage

\begin{appendices}
% In many cases, The American Naturalist allows authors to typeset their
% own supplementary material in an author-supplied PDF. This template
% applies to such cases. 
% 
% For appendices that will be typeset by the AmNat editorial staff, 
% please see the main LaTeX template, available from
% https://www.journals.uchicago.edu/journals/an/instruct
% Such appendices typically include descriptions of methods and tables
% defining parameters.
%
% In general, you have wide discretion for how you want to format an
% author-supplied PDF. They should in any case have a title page, 
% page numbers, and a header identifying the MS's (short) title.
%
% Counters for the online supplement should normally begin with an S
% (thus normally figure S1, figure S2, table S1, equation S1, etc.).

% redefine the commands that create equation, table, section, figure numbers.
%\renewcommand{\appendixname}{Supplement}
%\renewcommand{\appendixtocname}{Supplement}
%\renewcommand{\theequation}{S\arabic{section}.\arabic{equation}}
%\renewcommand{\thetable}{S\arabic{table}}
%\renewcommand{\thesection}{\arabic{section}}
%\renewcommand{\thefigure}{S\arabic{section}.\arabic{figure}}
\setcounter{equation}{0}  % reset counter 
\setcounter{figure}{0}
\setcounter{table}{0}

% In online supplementary PDFs, sections can be numbered or not
% (at your discretion). If they are numbered, sections should usually
% begin with an S.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Neutral inversions}\label{neutralInvSupp}

A full derivation of the deterministic two-locus and multi-locus recursions necessary to model the evolution of SLR-expanding inversions under the 'sheltering hypothesis' (neutral inversions) is provided in \citet{Olito-etal-2022} SI Appendix B. Here, we provide abbreviated derivations, emphasizing the key results that are used for Wright-Fisher simulations for all three selection scenarios in the present paper.


%%%%%%%%%%%
\subsection{Deterministic two-locus recursions}\label{Det2locus}

To model the evolutionary dynamics of SLR-expanding inversions under partially recessive deleterious mutation pressure, one needs recursions describing per-generation gene frequency changes for two categories of loci: (1) loci where the inversion intially captures a wild-type allele and (2) those where it captures a deleterious allele, within four different chromosome classes, (1) X's in ovules/eggs, (2) X's in pollen/sperm, (3) non-inverted Y's, and (4) inverted Y's. Under our assumption that the selected loci are unlinked in all but the inverted Y chromosome class, we can derive these recursions from a simple two-locus PAR model involving a sex-determining locus and a second selected locus under deleterious mutation pressure. The recursions can then be used to formulate a multi-lous model at linkage equilibrium. All of the same model assumptions noted in the main text still apply (i.e., large population size, discrete generations, etc.)

The relevant two-locus genetic system involves (1) a sex-determining locus, where XX individuals are female, and XY individuals are male, and (2) a selected locus (with wild-type $A$ and partially recessive deleterious variant $a$ alleles) that may or may not be genetically linked to the sex-determining region. Mutation from $A \rightarrow a$ occurs at a rate $\mu$ per meiosis, and we ignore backmutation. The three relevant female genotypes at the selected locus are: $AA$, $Aa$, and $aa$, with frequencies denoted $x_1$, $x_2$, and $x_3$, and general fitness expressions at selection denoted $w_{f,1}$, $w_{f,2}$, $w_{f,3}$. The eight relevant genotypes for males are: $AA$, $AA^I$, $Aa$ (cis-), $Aa^I$ (cis-), $aA$ (trans-), $aA^I$ (trans-), $aa$, and $aa^I$, with frequencies $y_{1}$, $y^I_{1}$, $y_{2c}$, $y^I_{2c}$, $y_{2t}$, $y^I_{2t}$, $y_{3}$, and $y^I_{3}$, and relative fitness terms $w_{m,1}$, $w_{m,2}$, $w_{m,3}$. Note that male heterozygote genotype labels indicate whether the $A$ allele is located on the X chromosome ($y_{2c}$), or on the Y chromosome ($y_{2t}$), and "$I$" superscripts denote inverted haplotypes. 

The frequency of each of the relevant haplotypes are denoted: 
\begin{align*}
	&X_{A,Ov}:~\text{the frequency of proto-X chromosomes carrying the {\itshape A} allele in ovules/eggs,}\\ 
	&X_{a,Ov}:~\text{the frequency of proto-X chromosomes carrying the {\itshape a} allele in ovules/eggs,}\\ 
	&X_{A,Sp}:~\text{the frequency of proto-X chromosomes carrying the {\itshape A} allele in pollen/sperm,}\\ 
	&X_{a,Sp}:~\text{the frequency of proto-X chromosomes carrying the {\itshape a} allele in pollen/sperm, and}\\ 
	&Y_A, Y_A^{I}, Y_a, Y_a^{I}:~\text{the frequency of inverted \& non-inverted proto-Y chromosomes}\\ &~~~~~~~~~~~~~~~~~~~~~~~~\text{carrying each allele in pollen/sperm.}
\end{align*}

The life cycle proceeds as follows: random mating $\rightarrow$ mutation $\rightarrow$ selection/meiosis. The resulting haplotype frequencies among female gametes (ovules/eggs) after selection and meiosis are:
\begin{align*}
	X_{A,Ov}^{\prime} &= \Big( x^u_{1} w_{f,1} + \frac{x^u_{2} w_{f,2} }{2} \Big) \Big/ \overline{w}_f \\
	X_{a,Ov}^{\prime} &= \Big( x^u_{3} w_{f,3} + \frac{x^u_{2} w_{f,2} }{2} \Big) \Big/ \overline{w}_f  \numberthis
\end{align*}

\noindent where $\overline{w}_f = x^u_{1} w_{f,1} + x^u_{2} w_{f,2} + x^u_{3} w_{f,3}$. The corresponding haplotype frequencies among pollen/sperm are:
\begin{align*}
	X^{\prime}_{A,Sp} &= \big(y^u_1 w_{m,1} + y^{I,u}_1 w_{m,1} + y^u_{2c} w_{m,2} (1 - r) + y^{I,u}_{2c} w_{m,2} + y^u_{2t} w_{m,2} r \big) \big/ \overline{w}_m \\
	X^{\prime}_{a,Sp} &= \big(y^u_{2c} w_{m,2} r + y^u_{2t} w_{m,2} (1 - r) + y^{I,u}_{2t} w_{m,2} + y^u_3 w_{m,3} + y^{I,u}_3 w_{m,3} \big) \big/ \overline{w}_m \\
	Y^{\prime}_{A} &= \big( y^u_1 w_{m,1} + y^u_{2c} w_{m,2} r + y^u_{2t} w_{m,2} (1 - r) \big) \big/ \overline{w}_m \\
	Y^{I \prime}_{A} &= \big( y^{I,u}_1 w_{m,1} + (y^{I,u}_{2t} w_{m,2}) \big) \big/ \overline{w}_m \\
	Y^{\prime}_{a} &= \big( y^u_{2c} w_{m,2} (1 - r) + y^u_{2t} w_{m,2} r + y^u_3 w_{m,3} \big) \big/ \overline{w}_m \\
	Y^{I \prime}_{a} &= \big( y^{I,u}_{2c} w_{m,2} + y^{I,u}_3 w_{m,3} \big) \big/ \overline{w}_m \numberthis
\end{align*}

\noindent where $\overline{w}_{m} = y^u_1 w_{m,1} + y^{I,u}_1 w_{m,1} + y^u_{2c} w_{m,2} + y^{I,u}_{2c} w_{m,2} + y^u_{2t} w_{m,2} + y^{I,u}_{2t} w_{m,2} + y^u_3 w_{m,3} + y^{I,u}_3 w_{m,3}$.

\bigskip

\noindent These recursions can be simplified using the following substitutions: $X_{A,Ov} = 1 - X_{a,Ov}$, $X_{A,Sp} = 1 - X_{a,Sp}$, $Y_{A} = 1 - Y_a - Y_A^I - Y_a^I$, and $r = 1/2$ (recall our assumption that the selected locus is initially in linkage equilibrium with the sex-linked region). The frequency dynamics can now be described with a system of $5$ simplified haplotype recursions: $X^{\prime}_{a,Ov}$, $X^{\prime}_{a,Sp}$, $Y^{\prime}_{a}$, $Y^{\prime I}_{A}$, and $Y^{\prime I}_{a}$.
\bigskip


In order to express the recursions in terms of the frequency of deleterious alleles in each of the different chromosome classes, it is necessary to transform the haplotype recursions onto a new coordinate system by introducing $3$ new variables: $Y_I$, the frequency of the inversion among $Y$ chromosomes; $q_Y$, the frequency of the deleterious $a$ allele at the selected locus among non-inverted $Y$ chromosomes; and $q^I_Y$, the frequency of the deleterious $a$ allele at the selected locus among inverted $Y$ chromosomes. We can then make the following substitutions: $Y_a = (1 - Y_I)q_Y$, $Y^I_a = Y^I q_Y$, and $Y^I_A = Y_I (1 - q_Y)$, and define three new recursions:
\begin{align*}
	q^{\prime}_Y &= Y^{\prime}_a / (Y^{\prime}_A + Y^{\prime}_a) \\
	q^{\prime}_I &= Y^{\prime I}_a / (Y^{\prime I}_A + Y^{\prime I}_a) \\
	Y^{\prime}_I &= (Y^{\prime I}_A + Y^{\prime I}_a) / (Y^{\prime}_A + Y^{\prime}_a + Y^{\prime I}_A + Y^{\prime I}_a)\numberthis
\end{align*}

\noindent For consistency of notation, we relabel $X_{a,Ov} = q_{X_f}$ and $X_{a,Sp} = q_{X_m}$. The frequency dynamics can now be described by the following $5$ recursions: $q^{\prime}_{X_f}$, $q^{\prime}_{X_m}$, $q^{\prime}_Y$, $q^{\prime}_I$, and $Y^{\prime I}$. We develop the multilocus recursion for $Y^{\prime I}$ below in \ref{subsec:multilocYI}.

%%%%%%%%%%%
\subsection{Recursions when inversion captures either a wild-type or deleterious allele}

A new single-copy SLR-expanding inversion will capture either a wild-type ($A$) or deleterious ($a$) allele at the selected locus. When a new (rare) inversion captures a wild-type allele, the above recursions can be used to describe the resulting per-generation allele frequency changes in each chromosome class. When a new inversion captures the deleterious $a$ allele, we substitute $Y^I_A = 0$ into the above recursion system. In this case all descendent copies of the inversion will carry the deleterious allele ({\itshape i.e.}, $q_I = 1$ for all $t$ generations in the future). 

The full two-locus recursions for $q^{\prime}_{X_f}$, $q^{\prime}_{X_m}$, $q^{\prime}_Y$, and $q^{\prime}_I$ when the inversion captures either a wild-type or deleterious allele are unwieldy, and are presented in the accompanying Supplementary Mathematica notebook file to \citet{Olito-etal-2022}. 

%%%%%%%%%%%
\subsection{Multilocus recursion for inversion frequency} \label{subsec:multilocYI}

We make several simplifying assumptions in order to describe the frequency dynamics of an inversion that captures any number of unlinked selected loci (all described in the main text). In brief, we assume the inversion captures $n$ selected loci, and that the mutation selection parameters are constant across all captured loci ({\itshape i.e.,} $\mu_i = \mu$, $s_i = s$, and $h_i = h$). We can then categorize the PAR loci spanned by the inversion by which allele is captured. The gene frequencies at each of the $n$ loci at time $t$ can be described using the following notation: $q^{D}_{X_f,t}$, $q^{D}_{X_m,t}$, $q^{D}_{Y,t}$, $q^{D}_{Y^I,t}$, and $q^{W}_{X_f,t}$, $q^{W}_{X_m,t}$, $q^{W}_{Y,t}$, $q^{W}_{Y^I,t}$, where $q$ refers to the deleterious allele frequency, and $D$ and $W$ superscripts denote which allele was initially captured by the inversion at the $i^{th}$ locus ($D$ denotes loci where a deleterious allele was initially captured, $W$ denotes loci where a wild-type allele was initially captured). To simplify notation, we use the convention $p^{\cdot}_{\cdot,t} = 1 - q^{\cdot}_{\cdot,t}$. 

We can now define the following recursion for $Y^{\prime}_I$ that takes into account the fitness effects of the alleles it captures at all $n$ loci:
\begin{equation} \label{eq:YIprime-multiLoc}
	Y^I_{(t + 1)} = Y^I_t \Bigg[ \Big(1 - s \big(h p^{D}_{X_f,t} + q^{D}_{X_f,t} \big) \Big)^{r}\Big(1 - s \big(h (p^{W}_{Y^I,t} q^{W}_{X_f,t} + q^{W}_{Y^I,t} p^{W}_{X_f,t}) + q^{W}_{Y^I,t} q^{W}_{X_f,t} \big) \Big)^{n-r} \Bigg] \Bigg/ \overline{w}^Y
\end{equation}

\noindent where 
\begin{align*}\label{eq:wBarY-multiLoc}
	\overline{w}^Y = Y^I_t &\Bigg[ \Big(1 - s \big(h p^{D}_{X_f,t} + q^{D}_{X_f,t} \big) \Big)^{r}\Big(1 - s \big(h (p^{W}_{Y^I,t} q^{W}_{X_f,t} + q^{W}_{Y^I,t} p^{W}_{X_f,t}) + q^{W}_{Y^I,t} q^{W}_{X_f,t} \big) \Big)^{n-r} \Bigg] + \\
	&(1 - Y^I_t) \left[\begin{array}{c}
											\bigg( 1 - s \Big( h \big( p^{D}_{X_f,t} q^{D}_{Y,t} +  q^{D}_{X_f,t} p^{D}_{Y,t} \big) + q^{D}_{X_f,t} q^{D}_{Y,t} \Big) \bigg)^r\\
											\bigg( 1 - s \Big( h \big( p^{W}_{X_f,t} q^{W}_{Y,t} +  q^{W}_{X_f,t} p^{W}_{Y,t} \big) + q^{W}_{X_f,t} q^{W}_{Y,t} \Big) \bigg)^{n-r}
											\end{array} \right] \numberthis
\end{align*}

\noindent Noting that $q^{D}_{I,{t+1}} = 1$. Equations (\ref{eq:YIprime-multiLoc}) and (\ref{eq:wBarY-multiLoc}) are presented as equations (1) and (2) in the main text of \citet{Olito-etal-2022}.

\bigskip

\noindent {\bf \itshape Note:} The effects of partially deleterious mutational variation will influence the evolutionary dynamics not only of neutral inversions, but also inversions under other forms of selection. The above multilocus recursions for neutral inversions therefore provide a baseline model which can be modified to accommodate other selection scenarios.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Beneficial inversions}\label{BenInvSupp}

\subsection{Multilocus recursions for inversion frequency} \label{subsec:multilocYI-ben}

For unconditionally beneficial inversions, only small changes are required to the model described in Supplement \ref{neutralInvSupp}. As described in the main text, we denote the heterozygote selection coefficient for a new beneficial inversion as $s_b$. Using the same notation as Supplement \ref{neutralInvSupp}, the deterministic inversion frequency dynamics under the joint effects of a constant fitness benefit and partially recessive deleterious mutational variation can be described by the following multilocus recursion:
\begin{equation} \label{eq:YIprime-ben-multiLoc}
	Y^I_{(t + 1)} = Y^I_t (1 + s_b) \Bigg[ \Big(1 - s \big(h p^{D}_{X_f,t} + q^{D}_{X_f,t} \big) \Big)^{r}\Big(1 - s \big(h (p^{W}_{Y^I,t} q^{W}_{X_f,t} + q^{W}_{Y^I,t} p^{W}_{X_f,t}) + q^{W}_{Y^I,t} q^{W}_{X_f,t} \big) \Big)^{n-r} \Bigg] \Bigg/ \overline{w}^Y
\end{equation}

\noindent where 
\begin{align*}\label{eq:wBarY-ben-multiLoc}
	\overline{w}^Y = Y^I_t (1 + s_b) &\Bigg[ \Big(1 - s \big(h p^{D}_{X_f,t} + q^{D}_{X_f,t} \big) \Big)^{r}\Big(1 - s \big(h (p^{W}_{Y^I,t} q^{W}_{X_f,t} + q^{W}_{Y^I,t} p^{W}_{X_f,t}) + q^{W}_{Y^I,t} q^{W}_{X_f,t} \big) \Big)^{n-r} \Bigg] + \\
	&(1 - Y^I_t) \left[\begin{array}{c}
											\bigg( 1 - s \Big( h \big( p^{D}_{X_f,t} q^{D}_{Y,t} +  q^{D}_{X_f,t} p^{D}_{Y,t} \big) + q^{D}_{X_f,t} q^{D}_{Y,t} \Big) \bigg)^r\\
											\bigg( 1 - s \Big( h \big( p^{W}_{X_f,t} q^{W}_{Y,t} +  q^{W}_{X_f,t} p^{W}_{Y,t} \big) + q^{W}_{X_f,t} q^{W}_{Y,t} \Big) \bigg)^{n-r}
											\end{array} \right] \numberthis
\end{align*}




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Inversions capturing sexually antagonistic loci}\label{SAInvSupp}


In the main text we present analyses of simplified two-locus models of sexually antagonistic selection. These results represent modest extensions of earlier population genetic models of the PAR including \citet{Clark1987}, \citet{Otto2011}, and \citet{Otto2014, Otto2019}. We refer readers to these papers for comprehensive background on two-locus PAR models. 

Here, we use two-locus models to examine the invasion conditions and to approximate the overall selection coefficient for rare SLR-expanding inversions while taking into account the possible effects of linkage disequilibrium between the SLR and the SA locus prior to the inversion mutation. Below, we develop the two-locus recursions for SLR-expanding inversions that also capture a single SA locus, and provide a detailed description of the invasion analysis and approximation of the invasion fitness (i.e., the overall selection coefficient for the inveresion when rare). We then describe the corresponding multilocus recursion which also takes into account the time-dependent effects of partially recessive deleterious mutation pressure. Key analytic results for these two-locus models are presented in the accompanying Mathematica notebook (.nb) available in the Online Supplementary Information.


%%%%%%%%%%%%
\subsection{Deterministic two-locus recursions} \label{SA-2loc-Supp}

Consider the two-locus genetic system described in the main text: one locus determines whether a chromosome is considered X or Y, with XX individuals being female, and XY individuals being male (YY are considered lethal); and a second locus having alleles $A$ and $a$ that may be genetically linked to the SLR and is subject to natural selection (we substitute fitness expressions for sexually antagonistic selection later). Recombination between the two loci occurs at a rate $r$ per meiosis. Generations are discrete, and the population size is assumed to be large enough that drift is negligible. The life cycle proceeds: meiosis $\rightarrow$ random mating $\rightarrow$ diploid selection.

When studying the invasion of an inversion capturing the sex determining locus and one allele at a selected locus on the Y chromosome, there are three relevant female genotypes: $AA$, $Aa$, and $aa$, with frequencies denoted $x_1$, $x_2$, and $x_3$, and general fitness expressions at selection denoted $w_{f,1}$, $w_{f,2}$, $w_{f,3}$. However, there are six relevant genotypes for males: $AA$, $Aa$ (cis-), $Aa^I$ (cis-), $aA$ (trans-), $aa$, and $aa^I$, with frequencies $y_{1}$, $y_{2c}$, $y^I_{2c}$, $y_{2t}$, $y_{3}$, $y_{3I}$, but fitness expressions $w_{m,1}$, $w_{m,2}$, $w_{m,3}$. Note that male heterozygote genotype labels indicate whether the $A$ allele is located on the X chromosome ($y_{2c}$), or on the Y chromosome ($y_{2t}$), and "$I$" superscripts denote inverted haplotypes. As before, we assume recombination is completely suppressed between inverted and non-inverted chromosomes.

\begin{equation*}
	\text{Female genotypes}:\left( \begin{array}{cc|c}
		x_1: & XA & XA \\
		x_2: & XA & Xa \\
		x_3: & Xa & Xa 
	\end{array} \right)
\end{equation*}

\begin{equation*}
	\text{Male genotypes}:\left( \begin{array}{cc|c}
		y_1:     & XA & YA \\
		y_{2c}:   & XA & Ya \\
		y^I_{2c}: & XA & Ya^I \\
		y_{2t}:   & Xa & YA \\
		y_{3}:    & Xa & Ya \\
		y^I_{3}:  & Xa & Ya^I 
	\end{array} \right)
\end{equation*}

\noindent The genotypic frequencies among females after recombination and random mating are:
\begin{align*}
	x^m_{1} &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^m_{2} &= \Big( x_1 + \frac{x_2}{2} \Big) \big( y_{2c} \cdot r + y_{2t}(1 - r) + y_3 + y^I_3 \big)~+ \\
	&~~~~\Big( x_3 + \frac{x_2}{2} \Big) \big( y_{1} + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big)      \\
	x^m_{3} &= \Big( x_3 + \frac{x_2}{2} \Big) \big(y_{2c} \cdot r + y{2t}(1 - r) + y_{3} + y^I_3  \big) \numberthis
\end{align*}

\noindent and among males:
\begin{align*}
	y^m_{1}      &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y_1 + y_{2c} \cdot r + y_{2t}(1 - r)  \big) \\
	y^m_{2c}     &= \Big( x_1 + \frac{x_2}{2} \Big) \big( y_{2c} (1 - r) + y_{2t} \cdot r + y_3 \big) \\
	y^{I,m}_{2c} &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y^I_{2c} + y^I_{3} \big)  \\
	y^m_{2t}     &= \Big( x_3 + \frac{x_2}{2} \Big) \big( y_{2c} \cdot r + y_{2t}(1 - r) + y_1 \\
	y^m_{3}      &= \Big( x_3 + \frac{x_2}{2} \Big) \big( y_{2c}(1 - r) + y_{2t} \cdot r + y_3 \\
	y^{I,m}_{3}  &= \Big( x_3 + \frac{x_2}{2} \Big) \big(y^I_{2c} + y^I_{3} \big)  \numberthis
\end{align*}

\noindent After selection, the genotypic frequencies among females will be:
\begin{align*}
	x'_{1} &= \frac{x^m_1 w_{f,1}}{\overline{w}_{f}}\\
	x'_{2} &= \frac{x^m_2 w_{f,2}}{\overline{w}_{f}}\\
	x'_{3} &= \frac{x^m_3 w_{f,3}}{\overline{w}_{f}} \numberthis
\end{align*}

\noindent where $\overline{w}_{f} = x^m_1 w_{f,1} + x^m_2 w_{f,2} + x^m_3 w_{f,3}$. For males, the genotypic frequencies after selection are:
\begin{align*}
	y'_{1}       &= \frac{y^m_1 w_{m,1}}{\overline{w}_{m}} \\
	y'_{2c}      &= \frac{y^m_{2c} w_{m,2}}{\overline{w}_{m}} \\
	y'^{I}_{2c}  &= \frac{y^{I,m}_{2c} w_{m,2}}{\overline{w}_{m}}  \\
	y'_{2t}      &= \frac{y^m_{2t} w_{m,2}}{\overline{w}_{m}} \\
	y'_{3}       &= \frac{y^m_3 w_{m,3}}{\overline{w}_{m}} \\
	y'^{I}_{3}   &= \frac{y^{I,m}_3 w_{m,3}}{\overline{w}_{m}} \numberthis
\end{align*}

\noindent where $\overline{w}_{m} = y^m_1 w_{m,1} + y^m_{2c} w_{m,2} + y^{I,m}_{2c} w_{m,2} + y^m_{2t} w_{m,2} + y^m_3 w_{m,3} + y^{I,m}_3 w_{m,3}$.

\bigskip


%%%%%%%%%%%%
\subsection{Eigenvalues and approximation for inversion invasion selection coefficient, \texorpdfstring{$s_I$}{sI}} \label{SA-eigenSI-Supp}

Under our stated assumptions, a new inversion on the proto Y chromosome must capture the SLR and a male-beneficial allele at the SA locus in order to invade.  We are interested in the evolutionary fate of rare inversion genotypes capturing the $a$ allele at the SA locus. We therefore substitute SA fitness expressions {\itshape sensu} \citealt{Kidwell1977,ConnallonClark2012,Otto2011}), where $w_{f,1} = 1$, $w_{f,2} = 1 - h_f s_f$, and $w_{f,3} = 1 - s_f$, and $w_{m,1} = 1 - s_m$, $w_{m,2} = 1 - h_m s_m$, and $w_{m,3} = 1$.

Substituting $x_3 = 1 - x_1 - x_2$ and $y_3 = 1 - y_1 - y_{2c} - y^I_{2c} - y_{2t} - y^I_3$, yields a system of $7$ genotypic frequency recursions. A key assumption in the analysis is that all non-inversion genotypes are initially at equilibrium when a new inversion mutation occurs. Specifically, we assume that $x_i = \hat{x}_i$ for $i \in \{1,\,2 \}$ and $y_j = \hat{y}_j$ where $j \in \{ 1,\,2c,\,2t \}$. To solve for the conditions under which a rare inversion capturing the SDR and the male-beneficial allele at the SA locus, we evaluate the Jacobian at the boundary equilibrium where the initial frequency of inverted genotypes are $y^{I}_{2c} = y^{I}_{3} = 0$:

\begin{equation*}
	\mathbb{J} = \left( \begin{array}{ccccccc}

					\frac{\partial x'_1}{\partial x_1} &
					\frac{\partial x'_2}{\partial x_1} &
					\frac{\partial y'_1}{\partial x_{1}} &
					\frac{\partial y'_{2c}}{\partial x_1} &
					\frac{\partial y'^I_{2c}}{\partial x_1} &
					\frac{\partial y'_{2t}}{\partial x_1} &
					\frac{\partial y'^I_{3}}{\partial x_1} \\[1ex]

					\frac{\partial x'_1}{\partial x_2} &
					\frac{\partial x'_2}{\partial x_2} &
					\frac{\partial y'_1}{\partial x_{2}} &
					\frac{\partial y'_{2c}}{\partial x_2} &
					\frac{\partial y'^I_{2c}}{\partial x_2} &
					\frac{\partial y'_{2t}}{\partial x_2} &
					\frac{\partial y'^I_{3}}{\partial x_2} \\[1ex]

					\frac{\partial x'_1}{\partial y_1} &
					\frac{\partial x'_2}{\partial y_1} &
					\frac{\partial y'_1}{\partial y_{1}} &
					\frac{\partial y'_{2c}}{\partial y_1} &
					\frac{\partial y'^I_{2c}}{\partial y_1} &
					\frac{\partial y'_{2t}}{\partial y_1} &
					\frac{\partial y'^I_{3}}{\partial y_1} \\[1ex]

					\frac{\partial x'_1}{\partial y_{2c}} &
					\frac{\partial x'_2}{\partial y_{2c}} &
					\frac{\partial y'_1}{\partial y_{2c}} &
					\frac{\partial y'_{2c}}{\partial y_{2c}} &
					\frac{\partial y'^I_{2c}}{\partial y_{2c}} &
					\frac{\partial y'_{2t}}{\partial y_{2c}} &
					\frac{\partial y'^I_{3}}{\partial y_{2c}} \\[1ex]

					\frac{\partial x'_1}{\partial y^I_{2c}} &
					\frac{\partial x'_2}{\partial y^I_{2c}} &
					\frac{\partial y'_1}{\partial y^I_{2c}} &
					\frac{\partial y'_{2c}}{\partial y^I_{2c}} &
					\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
					\frac{\partial y'_{2t}}{\partial y^I_{2c}} &
					\frac{\partial y'^I_{3}}{\partial y^I_{2c}} \\[1ex]

					\frac{\partial x'_1}{\partial y_{2t}} &
					\frac{\partial x'_2}{\partial y_{2t}} &
					\frac{\partial y'_1}{\partial y_{2t}} &
					\frac{\partial y'_{2c}}{\partial y_{2t}} &
					\frac{\partial y'^I_{2c}}{\partial y_{2t}} &
					\frac{\partial y'_{2t}}{\partial y_{2t}} &
					\frac{\partial y'^I_{3}}{\partial y_{2t}} \\[1ex]

					\frac{\partial x'_1}{\partial y^I_{3}} &
					\frac{\partial x'_2}{\partial y^I_{3}} &
					\frac{\partial y'_1}{\partial y^I_{3}} &
					\frac{\partial y'_{2c}}{\partial y^I_{3}} &
					\frac{\partial y'^I_{2c}}{\partial y^I_{3}} &
					\frac{\partial y'_{2t}}{\partial y^I_{3}} &
					\frac{\partial y'^I_{3}}{\partial y^I_{3}} \\[1ex]

				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										y'^I_{2c} = y^I_{3} = 0}} \numberthis
\end{equation*}

\noindent We can isolate the candidate leading eigenvalues associated with the spread of inversion genotypes by rearranging the Jacobian using elementary row and column operations to give a block triangular matrix \cite[Supplement~to~Primer~2]{OttoDay2007}:
\begin{equation}
	\mathbb{J}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{A} = \left( \begin{array}{ccc} 
										\frac{\partial x'_1}{\partial x_1} & \cdots & \frac{\partial y'_{2t}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'_1}{\partial y_{2t}} & \cdots & \frac{\partial y'_{2t}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) &
				\mathbf{B} = \left( \begin{array}{cc} 
										\frac{\partial y'^I_{2c}}{\partial x_1} & \frac{\partial y'^I_{3}}{\partial x_1} \\[1ex]
										\vdots & \vdots \\[1ex]
										\frac{\partial y'^I_{2c}}{\partial y_{2t}} & \frac{\partial y'^I_{3}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{C} = \left( \begin{array}{ccc} 
								0 & \cdots & 0\\[1ex]
								0 & \cdots & 0\\[1ex]
							\end{array} \right) &
		\mathbf{D} = \left( \begin{array}{cc} 
								\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} & \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} \\[1ex]
								\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} & \frac{\partial y'^I_{3}}{\partial y^I_{3}} \\[1ex]
							\end{array} \right) \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										y'^I_{2c} = y^I_{3} = 0}} \\[1ex]
\end{equation}

\noindent The eigenvalues of a block triangular matrix are the eigenvalues of the submatrices along the diagonal, which conveniently isolates the candidate leading eigenvalues for non-inversion genotypes (submatrix $\mathbf{A}$) and inversion genotypes (submatrix $\mathbf{D}$). When evaluated at $y'^I_{2c} = y^I_{3} = 0$, the two elements in the first row of submatrix $\mathbf{D}$ are identical ($ \alpha = \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} = \frac{\partial y'^I_{2c}}{\partial y^I_{3}}$), as are the two elements in the second row ($\beta = \frac{\partial y'^I_{3}}{\partial y^I_{2c}} = \frac{\partial y'^I_{3}}{\partial y^I_{3}}$). The candidate leading eigenvalue of submatrix $\mathbf{D}$ describing the spread of the rare inversion is:
\begin{equation}
	\lambda_{I} = \big( \alpha + \beta \big)
\end{equation}

\noindent $\lambda_I$ is cumbersome when expressed in terms of the adult genotypic frequencies. However, similar to Supplement \ref{neutralInvSupp}, $\lambda_I$ can be transformed onto a new coordinate system of allele frequencies in the three chromosome classes among gametes: X chromosomes in ovules ($X_f$), X chromosomes in sperm ($X_m$), and Y chromosomes in sperm ($Y$) \citep{Clark1987,Otto2011,Otto2014}. $\lambda_I$ simplifies considerably when expressed on this new coordinate system:

\begin{equation}
	\lambda_I = \frac{w_{m,2}(1 - \hat{X}_f) + w_{m,3} \hat{X}_f} {w_{m,1}(1 - \hat{X}_f)(1 - \hat{Y}) + w_{m,3} \hat{X}_f \hat{Y} + w_{m,2} (\hat{X}_f + \hat{Y} - 2 \hat{X}_f \hat{Y})}
\end{equation}

\noindent If $\lambda_I > 1$, and all non-inversion genotypes are initially at equilibrium ($x_i = \hat{x}_i$, $y_j = \hat{y}_j$), as we have assumed, $\lambda_I$ will necessarily be the leading eigenvalue of the system of recursions. This is a strong assumption that does not necessarily hold once the system has been perturbed. 
For inversions expanding the SLR on a Y chromosome, however, there are no internal equilibria; if the inversion can invade (i.e., if $\lambda_I > 1$), it will deterministically fix in the population of Y chromosomes. 

Substituting the SA fitness expessions described above, and calculating $s_I \approx \lambda - 1$ provides an estimate of the overall selection coefficient for the rare inversion \citep{OttoYong2002}, which appears in the main text as Eq(2) (when the SA locus is unlinked to the ancestral SLR) and Eq(4) (for arbitrary recombination rates between the SLR and SA locus). 



%%%%%%%%%%%
\subsection{Multilocus recursions for inversion frequency} \label{subsec:multilocYI-SA}

Similar to Supplement \ref{neutralInvSupp}, we can transform our two-locus recursions onto a new coordinate system tracking the frequency of the male-beneficial SA allele on the relevant chromosome classes among gametes. Specifically, we introduce two new variables: the frequency of the inversion among Y chromosomes, $Y^I$, and the frequency of the male-beneficial allele ($a$) among non-inverted Y chromosomes, $q_Y$. For consistency of notation, we also re-label the frequencies on the X chromosomes in ovules and sperm as $q_{X_f}$ and $q_{X_m}$ respectively, and the recombination rate between the SLR and SA locus as $r_{SA}$. The inversion frequency dynamics can now be described by a system of four recursions involving SA allele frequency dynamics:
\begin{align*}
	q^{\prime}_{X_f}       &= \frac{q_{X_f} + q_{X_m} (1 - h_f s_f) - q_{X_f} s_f \big(h_f + 2  q_{X_m} (1 - h_f)\big)}{2 \bigg(1 - s_f \Big(q_{X_f} q_{X_m} + h_f \big(q_{X_m} + q_{X_f} (1 - 2 q_{X_m})\big)\Big)\bigg)} \\
	q^{\prime}_{X_m}      &= \frac{q_{X_f} (1 - h_m s_m (1 - q_Y) (1 - Y^I) - r_{SA} (1 - h_m s_m) (1 - Y^I)) + q_Y r_{SA} (1 - h_m s_m) (1 - Y^I)}{1 - s_m (1 - Y^I (1 - h_m) - q_Y (1 - h_m - Y^I (1 - h_m)) - 
    q_{X_f} (1 - h_m - q_Y (1 - 2 h_m) - Y^I (1 - q_Y) (1 - 2 h_m)))}  \\
	q^{\prime}_{Y}       &= \frac{q_Y + r_{SA} (q_{X_f} - q_Y) - h_m s_m (q_{X_f} r_{SA} + q_Y (1 - q_{X_f} - r_{SA}))}{1 - s_m (1 - q_{X_f} - q_Y (1 - h_m - q_{X_f}) + h_m q_{X_f} (1 - 2 q_Y))} \numberthis
\end{align*}

\noindent and 
\begin{equation} \label{eq:YIprime-SA-multiLoc}
	Y^I_{(t + 1)} = Y^I_t (1 - s_m h_m (1 - q_{X_f,t})) \Bigg[ \Big(1 - s \big(h p^{D}_{X_f,t} + q^{D}_{X_f,t} \big) \Big)^{r}\Big(1 - s \big(h (p^{W}_{Y^I,t} q^{W}_{X_f,t} + q^{W}_{Y^I,t} p^{W}_{X_f,t}) + q^{W}_{Y^I,t} q^{W}_{X_f,t} \big) \Big)^{n-r} \Bigg] \Bigg/ \overline{w}^Y
\end{equation}

\noindent where 
\begin{align*}\label{eq:wBarY-SA-multiLoc}
	\overline{w}^Y = Y^I_t& (1 - s_m h_m (1 - q_{X_f,t})) \Bigg[ \Big(1 - s \big(h p^{D}_{X_f,t} + q^{D}_{X_f,t} \big) \Big)^{r}\Big(1 - s \big(h (p^{W}_{Y^I,t} q^{W}_{X_f,t} + q^{W}_{Y^I,t} p^{W}_{X_f,t}) + q^{W}_{Y^I,t} q^{W}_{X_f,t} \big) \Big)^{n-r} \Bigg] +\\
	&(1 - Y^I_t) (1 - s_m (h_m ((1 - q_{X_f,t}) q_{Y,t} + q_{X_f,t} (1 - q_{t,Y})) \left[\begin{array}{c}
											\bigg( 1 - s \Big( h \big( p^{D}_{X_f,t} q^{D}_{Y,t} +  q^{D}_{X_f,t} p^{D}_{Y,t} \big) + q^{D}_{X_f,t} q^{D}_{Y,t} \Big) \bigg)^r\\
											\bigg( 1 - s \Big( h \big( p^{W}_{X_f,t} q^{W}_{Y,t} +  q^{W}_{X_f,t} p^{W}_{Y,t} \big) + q^{W}_{X_f,t} q^{W}_{Y,t} \Big) \bigg)^{n-r}
											\end{array} \right] \numberthis
\end{align*}

\noindent and we use the same notation for deleterious allele frequencies at other selected loci captured by the inversion as was used in Supplements \ref{neutralInvSupp} and \ref{BenInvSupp}. 

Overall, modelling the deterministic frequency dynamics of an SLR-expanding inversion that captures a male-beneficial allele at a single SA locus, and $n$ other selected loci at which deleterious mutational variation is present requires a system of ten recursion equations: three describing per-generation SA allele frequency changes ($q^{\prime}_{X_f}$, $q^{\prime}_{X_m}$, and $q^{\prime}_{Y}$), seven describing per-generation changes in deleterious allele frequencies ($q^{W}_{class,t}$ and $q^{D}_{class,t}$, where $class \in \{X_f,\,X_m,\,Y,\,Y^I\}$, and recalling that $q^D_{Y^I} = 1$), and finally, the frequency of the inversion itself, $Y^I_{t+1}$. As with the other models, we used these deterministic recursions to predict the expected frequencies in the next generation, and sampled using these probabilities using pseudorandom binomial sampling in Wright-Fisher simulations (see main text for details).














%%%%%%%%%%%%
\subsection{Inversions on X chromosomes} \label{subsec:SA-Xchrom}

The assumptions and development of the recursions for an inversion spanning the SDR and a single SA locus on an X chromosome largely parallel those for Y chromome inversions. We use the same SA fitness expressions described in the previous section. Now, our interest is focused on a rare inversion that captures the female-beneficial $A$ allele at the SA locus, and we have $6$ relevant genotypes for both males and females:

\begin{equation*}
	\text{Female genotypes}:\left( \begin{array}{cc|c}
		x_1:        & XA   & YA \\
		x^I_{1}:    & XA   & XA^I \\
		x^{II}_{1}: & XA^I & XA^I \\
		x_{2}:      & XA   & Xa \\
		x^I_{2}:    & XA^I & Xa \\
		x_{3}:      & Xa   & Xa 
	\end{array} \right)
\end{equation*}

\begin{equation*}
	\text{Male genotypes}:\left( \begin{array}{cc|c}
		y_1:      & XA   & YA \\
		y^I_{1}:  & XA^I & YA \\
		y_{2c}:   & XA   & Ya \\
		y^I_{2c}: & XA^I & Ya \\
		y_{2t}:   & Xa & YA \\
		y_{3}:    & Xa & Ya 
	\end{array} \right)
\end{equation*}


\noindent The genotypic frequencies among females after recombination and random mating are:
\begin{align*}
	x^m_{1}      &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{I,m}_{1}  &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big( y^I_{2c} + y^I_{2c} \big)~+ \\
				 &~~~~\Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{II,m}_{1} &= \Big( \frac{x^I_1}{2} + x^{II}_1 + \frac{x^I_2}{2} \Big) \big( y^I_{2c} + y^I_{2c} \big) \\
	x^m_2        &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big)~+ \\
				 &~~~~\Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{I,m}_2    &= \Big( \frac{x^I_1}{2} + x^{II}_{1} + \frac{x^I_2}{2} \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big)~+ \\
				 &~~~~\Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big( y^I_{2c} + y^I_{2c} \big) \\
	x^m_3        &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big) \numberthis
\end{align*}

\noindent and among males:
\begin{align*}
	y^m_{1}      &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{I,m}_{1}  &= \Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big)  \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{m}_{2c}   &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big)  \\
	y^{I,m}_{2c} &= \Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big) \big( y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big) \\
	y^m_{2t}     &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{m}_{3}    &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big( y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big)  \numberthis
\end{align*}

\noindent After selection, the genotypic frequencies among females will be:
\begin{align*}
	x'_{1}     &= \frac{x^m_1 w_{f,1}     }{\overline{w}_{f}} \\
	x'^{I}_{1}  &= \frac{x^{I,m}_1 w_{f,1} }{\overline{w}_{f}} \\
	x'^{II}_{1} &= \frac{x^{II,m}_1 w_{f,1}}{\overline{w}_{f}} \\
	x'_{2}      &= \frac{x^m_2 w_{f,2}     }{\overline{w}_{f}} \\
	x'^{I}_{2}  &= \frac{x^{I,m}_2 w_{f,2} }{\overline{w}_{f}} \\
	x'_{3}      &= \frac{x^m_3 w_{f,3}     }{\overline{w}_{f}} \\
\end{align*}
    
\noindent where $\overline{w}_{f} = x^m_1 w_{f,1} + x^{I,m}_1 w_{f,1} + x^{II,m}_1 w_{f,1} + x^m_2 w_{f,2} + x^{I,m}_2 w_{f,2} + x^m_3 w_{f,3} $. The genotypic frequencies in males after selection are:
\begin{align*}
	y'_{1}       &= \frac{y^m_1 w_{m,1}       }{\overline{w}_{m}} \\
	y'^I_{1}     &= \frac{y^{I,m}_{1} w_{m,1} }{\overline{w}_{m}} \\
	y'_{2c}      &= \frac{y^{m}_{2c} w_{m,2}  }{\overline{w}_{m}}  \\
	y'^I_{2c}    &= \frac{y^{I,m}_{2c} w_{m,2}}{\overline{w}_{m}} \\
	y'_{2t}      &= \frac{y^m_{2t} w_{m,2}    }{\overline{w}_{m}} \\
	y'_{3}       &= \frac{y^{m}_3 w_{m,3}   }{\overline{w}_{m}} \numberthis
\end{align*}
  
\noindent where $\overline{w}_{m} = y^m_1 w_{f,1} + y^{I,m}_{1} w_{f,1} + y^{m}_{2c} w_{f,2} + y^{I,m}_{2c} w_{f,2} + y^m_{2t} w_{f,2} + y^{I,m}_3 w_{f,3} $.



%%%%%%%%%%%%
\subsubsection*{Eigenvalues (X chromosome inversion)}

Again, our analysis for the model of X chromosome inversions largely paralleles that for the Y.

Substituting $x_3 = 1 - x_{1} - x^I_{1} - x^{II}_{1} - x_{2} - x^I_{2}$ and $y_3 = 1 - y_1 - y^I_1 - y_{2c} - y^I_{2c} - y_{2t}$, yields a system of $10$ genotypic frequency recursions. As before, we make the key assumption that all non-inversion genotypes are initially at equilibrium when a new inversion mutation occurs: $x_i = \hat{x}_i$ for $i \in \{1,\,2 \}$ and $y_j = \hat{y}_j$ where $j \in \{ 1,\,2c,\,2t \}$. To solve for the conditions under which a rare inversion capturing the SDR and the male-beneficial allele at the SA locus, we evaluate the Jacobian at the boundary equilibrium where the initial frequency of inverted genotypes are $x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0$:


\begin{equation*}
	\mathbb{J} = \left( \begin{array}{cccccccccc}

					\frac{\partial x'_1      }{\partial x_1} &
					 \frac{\partial x'^I_1   }{\partial x_1} &
					 \frac{\partial x'^{II}_1}{\partial x_1} &
					 \frac{\partial x'_{2}   }{\partial x_1} &
					 \frac{\partial x'^I_{2} }{\partial x_1} &
					 \frac{\partial y'_1     }{\partial x_1} &
					 \frac{\partial y'^I_1   }{\partial x_1} &
					 \frac{\partial y'_{2c}  }{\partial x_1} &
					 \frac{\partial y'^I_{2c}}{\partial x_1} &
					 \frac{\partial y'_{2t}  }{\partial x_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x^I_1} &
					 \frac{\partial x'^I_1   }{\partial x^I_1} &
					 \frac{\partial x'^{II}_1}{\partial x^I_1} &
					 \frac{\partial x'_{2}   }{\partial x^I_1} &
					 \frac{\partial x'^I_{2} }{\partial x^I_1} &
					 \frac{\partial y'_1     }{\partial x^I_1} &
					 \frac{\partial y'^I_1   }{\partial x^I_1} &
					 \frac{\partial y'_{2c}  }{\partial x^I_1} &
					 \frac{\partial y'^I_{2c}}{\partial x^I_1} &
					 \frac{\partial y'_{2t}  }{\partial x^I_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x^{II}_1} &
					 \frac{\partial x'^I_1   }{\partial x^{II}_1} &
					 \frac{\partial x'^{II}_1}{\partial x^{II}_1} &
					 \frac{\partial x'_{2}   }{\partial x^{II}_1} &
					 \frac{\partial x'^I_{2} }{\partial x^{II}_1} &
					 \frac{\partial y'_1     }{\partial x^{II}_1} &
					 \frac{\partial y'^I_1   }{\partial x^{II}_1} &
					 \frac{\partial y'_{2c}  }{\partial x^{II}_1} &
					 \frac{\partial y'^I_{2c}}{\partial x^{II}_1} &
					 \frac{\partial y'_{2t}  }{\partial x^{II}_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x_2} &
					 \frac{\partial x'^I_1   }{\partial x_2} &
					 \frac{\partial x'^{II}_1}{\partial x_2} &
					 \frac{\partial x'_{2}   }{\partial x_2} &
					 \frac{\partial x'^I_{2} }{\partial x_2} &
					 \frac{\partial y'_1     }{\partial x_2} &
					 \frac{\partial y'^I_1   }{\partial x_2} &
					 \frac{\partial y'_{2c}  }{\partial x_2} &
					 \frac{\partial y'^I_{2c}}{\partial x_2} &
					 \frac{\partial y'_{2t}  }{\partial x_2} \\[1ex]

					\frac{\partial x'_1      }{\partial x'^I_{2}} &
					 \frac{\partial x'^I_1   }{\partial x'^I_{2}} &
					 \frac{\partial x'^{II}_1}{\partial x'^I_{2}} &
					 \frac{\partial x'_{2}   }{\partial x'^I_{2}} &
					 \frac{\partial x'^I_{2} }{\partial x'^I_{2}} &
					 \frac{\partial y'_1     }{\partial x'^I_{2}} &
					 \frac{\partial y'^I_1   }{\partial x'^I_{2}} &
					 \frac{\partial y'_{2c}  }{\partial x'^I_{2}} &
					 \frac{\partial y'^I_{2c}}{\partial x'^I_{2}} &
					 \frac{\partial y'_{2t}  }{\partial x'^I_{2}} \\[1ex]

					\frac{\partial x'_1      }{\partial y_1} &
					 \frac{\partial x'^I_1   }{\partial y_1} &
					 \frac{\partial x'^{II}_1}{\partial y_1} &
					 \frac{\partial x'_{2}   }{\partial y_1} &
					 \frac{\partial x'^I_{2} }{\partial y_1} &
					 \frac{\partial y'_1     }{\partial y_1} &
					 \frac{\partial y'^I_1   }{\partial y_1} &
					 \frac{\partial y'_{2c}  }{\partial y_1} &
					 \frac{\partial y'^I_{2c}}{\partial y_1} &
					 \frac{\partial y'_{2t}  }{\partial y_1} \\[1ex]

					\frac{\partial x'_1      }{\partial y^I_1} &
					 \frac{\partial x'^I_1   }{\partial y^I_1} &
					 \frac{\partial x'^{II}_1}{\partial y^I_1} &
					 \frac{\partial x'_{2}   }{\partial y^I_1} &
					 \frac{\partial x'^I_{2} }{\partial y^I_1} &
					 \frac{\partial y'_1     }{\partial y^I_1} &
					 \frac{\partial y'^I_1   }{\partial y^I_1} &
					 \frac{\partial y'_{2c}  }{\partial y^I_1} &
					 \frac{\partial y'^I_{2c}}{\partial y^I_1} &
					 \frac{\partial y'_{2t}  }{\partial y^I_1} \\[1ex]

					\frac{\partial x'_1      }{\partial y_{2c}} &
					 \frac{\partial x'^I_1   }{\partial y_{2c}} &
					 \frac{\partial x'^{II}_1}{\partial y_{2c}} &
					 \frac{\partial x'_{2}   }{\partial y_{2c}} &
					 \frac{\partial x'^I_{2} }{\partial y_{2c}} &
					 \frac{\partial y'_1     }{\partial y_{2c}} &
					 \frac{\partial y'^I_1   }{\partial y_{2c}} &
					 \frac{\partial y'_{2c}  }{\partial y_{2c}} &
					 \frac{\partial y'^I_{2c}}{\partial y_{2c}} &
					 \frac{\partial y'_{2t}  }{\partial y_{2c}} \\[1ex]

					\frac{\partial x'_1      }{\partial y^I_{2c}} &
					 \frac{\partial x'^I_1   }{\partial y^I_{2c}} &
					 \frac{\partial x'^{II}_1}{\partial y^I_{2c}} &
					 \frac{\partial x'_{2}   }{\partial y^I_{2c}} &
					 \frac{\partial x'^I_{2} }{\partial y^I_{2c}} &
					 \frac{\partial y'_1     }{\partial y^I_{2c}} &
					 \frac{\partial y'^I_1   }{\partial y^I_{2c}} &
					 \frac{\partial y'_{2c}  }{\partial y^I_{2c}} &
					 \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
					 \frac{\partial y'_{2t}  }{\partial y^I_{2c}} \\[1ex]

					\frac{\partial x'_1      }{\partial y_{2t}} &
					 \frac{\partial x'^I_1   }{\partial y_{2t}} &
					 \frac{\partial x'^{II}_1}{\partial y_{2t}} &
					 \frac{\partial x'_{2}   }{\partial y_{2t}} &
					 \frac{\partial x'^I_{2} }{\partial y_{2t}} &
					 \frac{\partial y'_1     }{\partial y_{2t}} &
					 \frac{\partial y'^I_1   }{\partial y_{2t}} &
					 \frac{\partial y'_{2c}  }{\partial y_{2t}} &
					 \frac{\partial y'^I_{2c}}{\partial y_{2t}} &
					 \frac{\partial y'_{2t}  }{\partial y_{2t}} \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation*}

\noindent As before, we can isolate the candidate leading eigenvalues associated with the spread of the inversion by rearranging the Jacobian to give the following block triangular matrix \cite[Supplement~to~Primer~2]{OttoDay2007}:

\begin{equation}
	\mathbb{J}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{A} = \left( \begin{array}{ccc} 
										\frac{\partial x'_1}{\partial x_1} & \cdots & \frac{\partial y'_{2t}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'_1}{\partial y_{2t}} & \cdots & \frac{\partial y'_{2t}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) &
				\mathbf{B} = \left( \begin{array}{ccc} 
										\frac{\partial x'^I_1}{\partial x_1} & \cdots & \frac{\partial y'^I_{2c}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'^I_{1}}{\partial y_{2t}} & \cdots & \frac{\partial y'^I_{2c}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{C} = \left( \begin{array}{ccc} 
								0      & \cdots & 0     \\[1ex]
								\vdots & \ddots & \vdots\\[1ex]
								0      & \cdots & 0     \\[1ex]
							\end{array} \right) &
				\mathbf{D} = \left( \begin{array}{ccc} 
										\frac{\partial x'^I_1}{\partial x^I_1} & \cdots & \frac{\partial y'^I_{2c}}{\partial x^I_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'^I_{1}}{\partial y^I_{2c}} & \cdots & \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} \\[1ex]
									\end{array} \right) \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation}

\noindent The eigenvalues calculations can be further simplified by rearranging submatrix $\mathbf{D}$ so that it is also of block triangular form (this does not affect the eigenvalues associated with changes in non-inversion genotype frequencies from submatrix $\mathbf{A}$). Ultimately, we have 
\begin{equation}
	\mathbf{D}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{W} = \left( \begin{array}{cc}
										\frac{\partial x'^I_1   }{\partial x^I_1} &
										 \frac{\partial x'^I_{2}}{\partial x^I_1} \\[1ex]
										\frac{\partial x'^I_1   }{\partial x^I_2} &
										 \frac{\partial x'^I_{2}}{\partial x^I_2} \\[1ex]
									\end{array} \right) &
				\mathbf{X} = \left( \begin{array}{ccc} 
										\frac{\partial y'^I_1    }{\partial x^I_1} &
										 \frac{\partial y'^I_{2c}}{\partial x^I_1} &
										 \frac{\partial x'^{II}_1}{\partial x^I_1} \\[1ex]
										\frac{\partial y'^I_1    }{\partial x^I_2} &
										 \frac{\partial y'^I_{2c}}{\partial x^I_2} &
										 \frac{\partial x'^{II}_1}{\partial x^I_2} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{Y} = \left( \begin{array}{cc} 
								0      & 0     \\[1ex]
								0      & 0     \\[1ex]
								0      & 0     \\[1ex]
							\end{array} \right) &
				\mathbf{Z} = \left( \begin{array}{ccc} 
										\frac{\partial y'^I_1    }{\partial y^I_1} &
										 \frac{\partial y'^I_{2c}}{\partial y^I_1} &
										 \frac{\partial x'^{II}_1}{\partial y^I_1} \\[1ex]
										\frac{\partial y'^I_1    }{\partial y^I_{2c}} &
										 \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
										 \frac{\partial x'^{II}_1}{\partial y^I_{2c}} \\[1ex]
										\frac{\partial y'^I_1    }{\partial x^{II}_1} &
										 \frac{\partial y'^I_{2c}}{\partial x^{II}_1} &
										 \frac{\partial x'^{II}_1}{\partial x^{II}_1} \\[1ex]
									\end{array} \right) \\
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation}

\noindent The relevant candidate leading eigenvalue comes from submatrix $\mathbf{W}$:

\begin{equation}
	\lambda_{I} = \big( \alpha + \beta \big)
\end{equation}

where $\alpha = \frac{\partial x'^I_1}{\partial x^I_1} = \frac{\partial x'^I_{2}}{\partial x^I_1}$ and $\beta = \frac{\partial x'^I_1}{\partial x^I_2} = \frac{\partial x'^I_{2}}{\partial x^I_2}$. Transforming $\lambda_I$ onto a coordinate system of allele frequencies on the three chromosome classes, $X_f$, $X_m$, and $Y$, yields:


\begin{equation}
	\lambda_I = \frac{w_{f,2} + \hat{X}_f(w_{f,1} - w_{f,2})} {w_{f,3}(1 - \hat{X}_f)(1 - \hat{X}_m) + w_{f,1} \hat{X}_f \hat{Y} + w_{f,2} (\hat{X}_f + \hat{X}_m - 2 \hat{X}_f \hat{X}_m)}
\end{equation}

\noindent If $\lambda_I > 1$, and all non-inversion genotypes are initially at equilibrium ($x_i = \hat{x}_i$, $y_j = \hat{y}_j$), as we have assumed, $\lambda_I$ will necessarily be the leading eigenvalue of the system of recursions. However, for inversions on the X chromosome there exist internal equilibria for the inversion genotypes which we have confirmed with deterministic iteration of the recursions (see fig.~S5 in Supplement 4). 

Our approximation of $s_I \approx \lambda - 1$, must be interpreted cautiously for the model of X chromosome inversions. In particular, $2 s_I$ does not provide a valid approximation of the fixation probability for the inversion, but rather an optimistic approximation of the probability that the inversion escapes stochastic loss when rare to approach an internal deterministic equilibrium frequency. In finite populations, $2 s_I$ will overestimate the probability that a new inversion is maintained as a polymorphism in the long-term due to chance extinction of inversions maintained at low equilibrium frequencies. Hence, inversions on the X chromosome are less likely to be maintained as stable polymorphisms than suggested by the deterministic simulations (fig.~S5).


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Supplementary Figures} \label{SuppFigs}
 \renewcommand{\theequation}{S\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{S\arabic{figure}}
 \setcounter{figure}{0}

%%%%%%%%%%%%%%%
\begin{figure}[H]
 \centering
 \includegraphics[width=0.65\linewidth]{Suppfig-Cartoon-Overview}
 \caption{The spread of SLR-expanding inversions on a Y chromosome under three selection scenarios investigated in the main text: (A) neutral inversions, (B) directly beneficial inversions, and (C) inversions capturing a sexually antagonistic locus. From left to right, each illustration depicts a sample of Y chromosomes at three time points during the spread of an inversion, highlighting several key features of the theoretical models. The left-hand panels show key outcomes when new inversions first arise, and emphasize that lightly-loaded inversions have a temporary fitness advantage (panels A-C) and that inversions capturing a female-beneficial allele in the SA selection scenario (panel C) are unlikely to successfully invade. The center panels illustrate the spread of the inversion and highlight that the fitness advantage enjoyed by mutation-free inversions decays over time as they accumulate new deleterious mutations. Finally, the right-hand panels illustrate the eventual fixation of the inversion and the resulting expansion of the SLR. In all panels, the shading indicates the recombination rate with ancestral SLR $0 \leq r_{SA} < 1/2$.}
 \label{fig:diagramFig}
 \end{figure}



\newpage
%%%%%%%%%%%%%%%
 \begin{figure}[H]
 \centering
 \includegraphics[width=0.95\linewidth]{./PrFix_SA_r_delMut}
 \caption{Fixation probabilities for inversions of different lengths capturing the SLR and a SA locus on the Y-chromosome estimated from Wright-Fisher simulations. Each column of panels show results for different chromosome-arm wide mutation rates relative to selection (i.e., different values of $U$), which influence the average numbers of deleterious mutations carried by a standard-arrangement chromosome ($U/hs$). THe first row of panels (A--C) show results for additive SA fitness effects ($h_{SA} = 1/2$); the second row (D--F) show results for dominance reversals ($h_{SA} = 1/4$); and $s_f = s_m = 0.05$ for all panels. Point shapes indicate different recombination rates bewteen the ancestral SLR and SA locus ($r_{SA} = \{0.002,\, 0.008,\,0.032,\,0.124,\,0.5\}$). Parameters for deleterious mutations are the same as in the main text for all panels: $h = 0.25$, $s = 0.01$, $n_{tot} = 10^4$. Other relevant parameter values are: $A = 1$, $P = 0.05$. All results condition on the inversion spanning the SLR.}
 \label{fig:PrFixFig_SA_r}
 \end{figure}


\newpage
%%%%%%%%%%%%%%%
 \begin{figure}[H]
 \centering
 \includegraphics[width=0.95\linewidth]{./recombEffect}
 \caption{Overall selection coefficient ($s_I$) for an inversion linking a male-beneficial allele at an SA locus to the SLR (see Eq[4] in the main text) as a function of the ancestral recombination rate between the two loci ($r_SA$). Panel A shows $s_I$ when there is equal selection on female- and male-beneficial alleles ($s_f = s_m$) and additive SA fitness effects ($h_f = h_m = 1/2$). Panel B shows the same for female biased selection (where $s_f < s_m$; recall that SA selection coefficients represent the decrease in relative fitness of either SA allele in males and females, after \citealt{Kidwell1977}). Specifically, we model the special case where $s_f$ is equal to the single-locus invasion condition for the male-beneficial allele ($s_f = s_m / (1 - s_m)$).}
 \label{fig:recombEffect}
 \end{figure}


\newpage
%%%%%%%%%%%%%%%
 \begin{figure}[H]
 \centering
 \includegraphics[width=0.95\linewidth]{./PrFix_biasedSA_r_delMut}
 \caption{ Fixation probabilities for inversions of different lengths capturing the SLR and a SA locus on the Y-chromosome estimated from Wright-Fisher simulations. From right to left, each panel shows results for different chromosome-arm wide mutation rates relative to selection (i.e., different values of $U$), which influences the average numbers of deleterious mutations carried by a standard-arrangement chromosome ($U/hs$). Results are shown for additive SA fitness effects ($h_{SA} = 1/2$), with female-biased selection (where $s_f = s_m / (1 - s_m)$). Point shapes indicate different recombination rates bewteen the ancestral SLR and SA locus ($r_{SA} = \{0.002,\, 0.008,\,0.032,\,0.124,\,0.5\}$). Parameters for deleterious mutations are the same as in the main text for all panels: $h = 0.25$, $s = 0.01$, $n_{tot} = 10^4$. Other relevant parameter values are: $A = 1$, $P = 0.05$. All results condition on the inversion spanning the SLR.}
 \label{fig:PrFix-biasedSA}
 \end{figure}



\newpage
%%%%%%%%%%%%%%%
 \begin{figure}[H]
 \centering
 \includegraphics[width=0.95\linewidth]{./detEqInvFreqFig}
 \caption{Deterministic equilibrium frequency of new inversions capturing the SLR and a single sexually antagonistic locus on the Y ($\hat{Y}$, panels A and B) and the X chromosomes ($\hat{X}$, panels C and D), under loose (panels A and C) and tight (panels B and D) genetic linkage between the two loci, and additive SA fitness effects ($h_f = h_m = 1/2$). Initial equilibrium genotypic frequencies were calculated by iterating the two-locus deterministic recursions in the absence of an inversion. Once this initial equilibrium was reached, an (heterozygote) inversion genotype was introduced at low frequency ($10^{-6}$), and the recursions were again iterated until all genotypic frequencies remained unchanged. Note the different color scale for Y and X inversions. Recursions are presented in the Supporting Information.}
 \label{fig:detInvFreqSA}
 \end{figure}


\newpage
%%%%%%%%%%%%%%%
\begin{figure}[H]
 \centering
 \includegraphics[width=0.95\linewidth]{./SuppFig-Pr-ExpandSLR}
 \caption{Probability that a new inversion expands the ancestral SLR as a function of inversion size, $x$, and the location of the SLR on the chromosome arm (see Eq[6] and corresponding assumptions in the main text). We illustrate the same scenarios analysed in the main article: (A) the SLR is located at the exact middle of the chromosome arm, $\text{SLR}_\text{pos} = 0.5$, and (B) the SLR is posated closer to the centromere $\text{SLR}_\text{pos} = 0.1$ (this choice is arbitrary, and results are identical if the SLR is positioned equally close to the distal end of the chromsoome arm, $\text{SLR}_\text{pos} = 0.9$). A cartoon illustration of a Y chromosome arm with the corresponding SLR positions are drawn above each plot. The form of $\Pr (\text{SLR} \mid x)$ changes as $x$ increases, and shaded regions indicate the relevant regions of parameter space where the piecewise function changes form, with the corresponding function shown in boxes.}
 \label{fig:ExpandSLR-SuppFig}
 \end{figure}


\newpage
%%%%%%%%%%%%%%%%%%%%%%%
 \begin{figure}[H]
 \centering
 \includegraphics[width=0.95\linewidth]{./Expected_xDistribution_delMut_Ud0_05}
 \caption{Scaled probability densities for SLR-expanding inversions of different lengths (Eq[7] in the main text). Panels A \& C show results for the Random Breakpoint model, while B \& D show results for the Exponential model of new inversion sizes. Point shapes indicate different selection scenarios, and corresponding black points along the x-axis indicate distribution means. Results are shown for the intermediate deleterious mutation rate presented in other figures, which corresponds to an average deleterious mutation load on standard-arrangement chromsomes of $U/hs = 20$. }
 \label{fig:ExpectedDistFig_Ud0_05}
 \end{figure}


\newpage
%%%%%%%%%%%%%%%%%%%%%%%
 \begin{figure}[H]
 \centering
 \includegraphics[width=0.95\linewidth]{./Expected_xDistribution_delMut_Ud0_1}
 \caption{Scaled probability densities for SLR-expanding inversions of different lengths (Eq[7] in the main text). Panels A \& C show results for the Random Breakpoint model, while B \& D show results for the Exponential model of new inversion sizes. Point shapes indicate different selection scenarios, and corresponding black points along the x-axis indicate distribution means. Results are shown for the highest deleterious mutation rate presented in other figures, which corresponds to an average deleterious mutation load on standard-arrangement chromsomes of $U/hs = 40$.}
 \label{fig:ExpectedDistFig_Ud0_1}
 \end{figure}




%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%
% Any references in this section should also be copied into the main 
% template, under the heading
% \section*{References Cited Only in the Online Enhancements}
% except when they are already cited in the main text.
\newpage
\bibliography{../bibliography-inversionSize-ProtoSexChrom}


\end{appendices}


\end{document}
