\documentclass[11pt]{article}
\usepackage[sc]{mathpazo} %Like Palatino with extensive math support
\usepackage{fullpage}
\usepackage[authoryear,sectionbib,sort]{natbib}

\bibliographystyle{evolution.bst}
\setlength{\bibsep}{0.0pt}
\linespread{1.5}
\usepackage[utf8]{inputenc}
\usepackage{lineno}
\usepackage{titlesec}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage{booktabs}
\titleformat{\section}[block]{\Large\bfseries\filcenter}{\thesection}{1em}{}
\titleformat{\subsection}[block]{\Large\itshape\filcenter}{\thesubsection}{1em}{}
\titleformat{\subsubsection}[block]{\large\itshape}{\thesubsubsection}{1em}{}
\titleformat{\paragraph}[runin]{\itshape}{\theparagraph}{1em}{}[. ]
\usepackage{fancyhdr}
\usepackage{color,soul}
\pagestyle{fancy}
\usepackage[colorlinks=true, allcolors=black]{hyperref}
\setlength{\headheight}{13.6pt}
\renewcommand{\refname}{References}
\usepackage{tabu}
\usepackage{tikz}

% \newcommand\encircle[1]{%
%   \tikz[baseline=(X.base)] 
%     \node (X) [draw, shape=circle, inner sep=0] {\strut #1};}
\newcommand*\circled[1]{\tikz[baseline=(char.base)]{
            \node[shape=circle,draw,inner sep=2pt] (char) {#1};}}

% Equation numbering
\newcommand\numberthis{\addtocounter{equation}{1}\tag{\theequation}}

% Graphics package
\usepackage{graphicx}
\graphicspath{{../../output/figures/}.pdf}

% Change default margins
\usepackage[top=0.75in, bottom=0.75in, left=0.75in, right=0.75in]{geometry}

% Definitions
\def\mathbi#1{\textbf{\em #1}}
\def\mbf#1{\mathbf{#1}}
\def\mbb#1{\mathbb{#1}}
\def\mcal#1{\mathcal{#1}}
\newcommand{\bo}[1]{{\bf #1}}
\newcommand{\tr}{{\mbox{\tiny \sf T}}}
\newcommand{\bm}[1]{\mbox{\boldmath $#1$}}
\newcommand{\om}{\omega}
\DeclareRobustCommand{\firstsecond}[2]{#2}
 \def\linenumberfont{\normalfont\scriptsize}
 \newcommand{\kron}{\otimes}
\usepackage{pdflscape}


%%%%%%%%%%%%%%%%%%%%%
% Header
%%%%%%%%%%%%%%%%%%%%%
%
% Customize the line below with the last name of your first author and
% the short title of your MS. You can comment authorship information out
% while your MS is undergoing double-blind review.
%
\lhead{\thesection}
\rhead{Olito \& Abbott, "Inversions on Proto Sex Chromosomes"}
\setlength{\headheight}{15pt}
\setlength{\headsep}{0.15in}  

%%%%%%%%%%%%%%%%%%%%%
% Line numbering
%%%%%%%%%%%%%%%%%%%%%
%
% Please use line numbering with your initial submission and
% subsequent revisions. After acceptance, please turn line numbering
% off by adding percent signs to the lines %\usepackage{lineno} and
% to %\linenumbers{} and %\modulolinenumbers[3] below.
%
% To avoid line numbering being thrown off around math environments,
% the math environments have to be wrapped using
% \begin{linenomath*} and \end{linenomath*}
%
% (Thanks to Vlastimil Krivan for pointing this out to us!)
\begin{document}
\title{Supplementary Materials (Appendices A -- \hl{X}) for: The evolution of suppressed recombination between sex chromosomes by chromosomal inversions.\\ 
%\LaTeX{} Template for Author-Supplied Supplementary PDFs, \\ 
\textit{Evolution} }

% This version of the LaTeX supplementary template was last updated on
% November 8, 2019.

%%%%%%%%%%%%%%%%%%%%%
% Authorship
%%%%%%%%%%%%%%%%%%%%%
% Please remove authorship information while your paper is under review,
% unless you wish to waive your anonymity under double-blind review. You
% will need to add this information back in to your final files after
% acceptance.
%
% Once accepted for publication, author-supplied PDFs should have a 
% title page that includes (at least) the authors' names, the title of 
% the MS, and the name of the journal. It should also have a header and
% page numbers.
\author{Colin Olito$^{\ast}$ \\ 
Jessica K.~Abbott}
\date{\today}
\maketitle

\noindent Department of Biology, Lund University, Lund 223 62, Sweden. \\
\noindent $\ast$ Corresponding author; e-mail: \url{colin.olito@gmail.com}\\

\noindent ORCiDs\\
\noindent CO: https://orcid.org/0000-0001-6883-0367\\
\noindent JKA: https://orcid.org/0000-0002-8743-2089


\bigskip
%\linenumbers{}
%\modulolinenumbers[3]

\newpage{}
\tableofcontents
\newpage{}
% In many cases, The American Naturalist allows authors to typeset their
% own supplementary material in an author-supplied PDF. This template
% applies to such cases. 
% 
% For appendices that will be typeset by the AmNat editorial staff, 
% please see the main LaTeX template, available from
% https://www.journals.uchicago.edu/journals/an/instruct
% Such appendices typically include descriptions of methods and tables
% defining parameters.
%
% In general, you have wide discretion for how you want to format an
% author-supplied PDF. They should in any case have a title page, 
% page numbers, and a header identifying the MS's (short) title.
%
% Counters for the online supplement should normally begin with an S
% (thus normally figure S1, figure S2, table S1, equation S1, etc.).

% redefine the commands that create equation, table, section, figure numbers.
\renewcommand{\theequation}{S\arabic{section}.\arabic{equation}}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\thesection}{S\arabic{section}}
\renewcommand{\thefigure}{S\arabic{section}.\arabic{figure}}
\setcounter{equation}{0}  % reset counter 
\setcounter{figure}{0}
\setcounter{table}{0}

% In online supplementary PDFs, sections can be numbered or not
% (at your discretion). If they are numbered, sections should usually
% begin with an S.

%%%%%%%%%%%%%%%%%%%%%%
% Appendixes moved from main
% manuscript at Ed. Office's 
% suggestion

%\renewcommand\thesection{Appendix~\Alph{section}}
%\renewcommand\thesubsection{\Alph{section}.\arabic{subsection}}


\setcounter{equation}{0}  % reset counter 
\setcounter{table}{0}  % reset counter 



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Neutral inversions}\label{AppA}
\renewcommand{\theequation}{A\arabic{equation}}
\titleformat{\subsubsection}    
{\normalfont\fontsize{12pt}{17}\itshape}{\thesubsubsection}{12pt}{}
\renewcommand{\thetable}{A\arabic{table}}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% OLD RESULTS TEXT FROM EARLIER DRAFT OF MAIN MS
%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Fixation Probabilities}


\subsubsection*{Neutral Inversions}
For a neutral inversion spanning the SLR on the Y chromosome in a population with an even sex ratio, the effective population size is $N_Y = N_m = N/2$, where $N_m$ is the number of breeding males in the population and $N$ is the total breeding population size. In the absence of deleterious mutational variation the fixation probability for a neutral inversion is independent of inversion length, and equal to its' initial frequency: $\Pr(\text{fix}) = 1/N_Y = 2/N$ for a single copy inversion mutation \citep{Kimura1962, CrowKimura1970}. Under the same assumptions, inversions spanning the SLR on the X chromosome will have an effective population size of $N_X = 3 N_f/2 = 3 N/4 $, and $\Pr(\text{fix}) = 1/N_X = 4/3N$. 

However, under partially recessive deleterious mutation pressure, the evolutionary fate of neutral inversions emerges from time-dependent indirect selection effects which systematically favour smaller inversions (fig.~\ref{PrFixFig}; \cite[see][]{Olito-etal-2022}). Unfortunately, there is no simple analytic solution for the fixation probability under this scenario (\citealt{OhtaKojima1968, KimuraOhta1970, UeckerHermisson2011, Waxman2011}). However, it is possible to approximate the fixation probability for large populations under weak selection \citep{ConnallonOlito2021}. In large populations ($0 < N_Y^{-1},\, N_X^{-1} \ll 1$) an initially deleterious mutation-free inversion will have an initial fitness advantage over non-inverted chromosomes, and will increase in frequency pseudo-deterministically until new deleterious mutations arise on descendent copies of the original inversion and reach equilibrium under mutation-selection balance. At this point, the inversion and wild-type karyotype will be equally fit and the inversion will subsequently evolve neutrally. The approximate fixation probability for an initially mutation-free inversion spanning the SLR is

\begin{subequations}\label{eq:NeutralPfix}
	\begin{align*}
		\Pr(\text{fix} \mid x, k = 0) &\approx N^{-1}_Y \, \exp \bigg[ U_d x \bigg( \frac{1}{1 - e^{-s_d} } - \frac{2}{s_d} \bigg) \bigg] e^{\frac{-U_d x}{s_d}} \approx N^{-1}_Y \numberthis,
%									  &\approx N^{-1}_Y \numberthis
	\end{align*}
	\text{when the inversion is on the Y chromosome, and}
	\begin{align*}
		\Pr(\text{fix} \mid x, k = 0)&\approx N^{-1}_X \, e^{\frac{U_d x}{s_d}} e^{\frac{-U_d x}{s_d}} = N^{-1}_X \numberthis, 
	\end{align*}
\end{subequations}

\noindent when the inversion is on the X chromosome (see Appendix A). Eq(\ref{eq:NeutralPfix}a) and Eq(\ref{eq:NeutralPfix}b) reduce to the same form as the autosomal case (see \citealt{Nei1967, ConnallonOlito2021}) due to our assumption that inversion fixation occurs on a shorter timescale than gene degeneration and loss within the inverted chromosomal segment (see \hyperref[sec:assumptions]{Assumptions}). When functional homologs exist on the X and Y chromosomes, the dynamics of deleterious mutations prior to the inversion, and the subsequent evolution of initially mutation-free neutral inversions, are nearly identical whether the inversion arises on an Y, X, or autosome \citet{ConnallonOlito2021}. This surprisingly simple result emerges from the rather complicated time-dependent dynamics because the greater fitness advantage to larger inversions of being initially free of deleterious mutations is approximately counterbalanced by the dwindling chance that they will in fact be initially free of deleterious alleles. \vspace{12pt}

\noindent {\itshape Key result: When inversions restricting recombination between sex chromosomes are selectively neutral, the overall fixation probability after taking deleterious mutations into account is equal to the initial frequency of the inversion (Fig.~\ref{fig:fixProbFig}A)}.


%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Unconditionally beneficial inversions}

The specific location of new inversion breakpoints may give inverted chromosomes a selective advantage over wild-type chromosomes. For example, an inversion may bring a protein coding sequence into closer proximity to a promoter region, thereby altering expression without disrupting other genes, or by disrupting synteny near 'sensitive sites' \citep{KrimbasPowell1992, CorbettDetig2016}. Under weak selection, and momentarily neglecting deleterious mutations, the fixation probability of a beneficial inversion can be approximated by $\Pr(\text{fix}) \approx 2 s_{I}$ \citep{Haldane1927} (i.e., there is no relation between the length of the inversion and the fixation probability). For beneficial inversions capturing the SLR on a Y chromosome, $s_I = h s_{I}^{m}$ represents the heterozygous selective advantage of the inversion in males (where $h$ is the dominance coefficient associated with the inversion). For a new inversion capturing the SLR on an X-chromosome

\begin{equation} \label{eq:benXlinkednoDel}
	s_{I} \approx \frac{h \big( s_{I}^{f} + s_{I}^{m} \big)}{2},
\end{equation}

\noindent where $s_{I}^{\text{sex}}$ is the sex-specific selection coefficient ($\text{sex} \in \{m,f\}$). Both approximations work well when $1/N \ll s_I \ll 1$.

Taking deleterious mutations into account is mathematically similar to the haploid autosomal case (see Eqs.[9 \& 10] in \citealt{ConnallonOlito2021}, and our Appendix A). A new beneficial inversion that is also free of deleterious mutations will have a temporary selective advantage that will decline over time from $(1 + s_I)e^{U_d x}$, eventually leaving only the intrinsic advantage, $(1 + s_I)$ \citep{Nei1967}. The resulting fixation probability can be approximated using a time-dependent branching process \citep{PeischlKirkpatrick2012, KirkpatrickPeischl2013}, which can be expressed in terms of a time-averaged {\itshape effective selection coefficient} for the inversion:

\begin{equation} \label{eq:benSe}
	s_{e} = s_t \sum_{t=0}^{\infty} (1 - s_I)^t = s_I \Bigg[1 + \frac{U_d x}{1 - (1-s_I)e^{-s_d}} \Bigg],
\end{equation}

\noindent where $s_I = s_{I}^{m}$ for inversions capturing the SLR on the Y chromosome, while $s_I$ is given by Eq.(\ref{eq:benXlinkednoDel}) for those on the X-chromosome. Incorporating the probability that the inversion is initially mutation free, we have

\begin{equation} \label{eq:benPrFix}
	\Pr(\text{fix} \mid x, k = 0) \approx 2 s_I \Bigg[ 1+ \frac{U_d x}{1 - (1-s_I)e^{-s_d}} \Bigg] e^{\frac{-U_d x}{s_d}},
\end{equation}

\noindent and $s_I$ is defined as above for Y- and X-linked inversions respectively. The overall effect of deleterious mutations is to make the fixation probability decline with inversion length, with a maximum of $\approx 2 s_I$ as $x$ approaches $0$ (Fig.~\ref{fig:fixProbFig}B). 
\vspace{12pt}

\noindent {\itshape Key result: When inversions spanning the SLR are intrinsically beneficial, smaller inversions are always favoured because they are less likely to capture deleterious mutations.}



 \begin{figure}[htbp]
 \centering
 \includegraphics[width=\linewidth]{./PrFix_combined_delMut}
 \caption[\captionsize]{Fixation probabilities for inversions of different lengths capturing the SLR on the Y-chromosome estimated from Wright-Fisher simulations. Point shapes indicate different chromosome-arm wide mutation rates relative to selection (i.e., different values of $U$), which influence the average numbers of deleterious mutations carried by a standard-arrangement chromosome ($U/hs$). Dashed lines correspond to approximations of $\Pr(\text{fix} \mid x)$ when ignoring the effects of partially recessive deleterious mutation pressure for each selection scenario (see text for approximations for each scenario). Parameters for deleterious mutations are the same for all panels: $h = 0.25$, $s = 0.01$, $n_{tot} = 10^4$. The selection coefficient for unconditionally beneficial inversions (B) is $s_b = 0.02$. For the SA selection scenario (C--F) we explored several magnitudes of prior genetic linkage between the SLR and SA locus ($r_{SA} = \{ 0.002, 0.008, 0.032, 0.5\}$), with $s_f = s_m = 0.05$, $h_f = h_m = 0.5$, $A = 1$, $P = 0.05$. All results condition on the inversion spanning the SLR.}
 \label{fig:PrFixFig}
 \end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Indirect selection -- Sexual antagonism}\label{sec:SexAntag}

It is well established that sexually antagonistic (SA) variation can theoretically drive selection for recombination modifiers coupling selected alleles with specific sex chromosomes \cite[e.g.][]{Fisher1931,Nei1969, Charlesworth1978a, Charlesworth1980, Bull1983,Lenormand2003, Otto2019}. However, the role of pre-existing linkage disequilibrium between the SLR and SA loci in this process is complicated. The idea that SA polymorphisms initially linked to the SLR can promote the accumulation of more linked SA polymorphisms, and lead to stronger selection for recombination suppression is seductively intuitive \citep{Rice1984, Rice1996, Charlesworth2017, Otto2019}. Yet, the conditions for the spread of SA polymorphisms to multiple loci in linkage disequilibrium with the SLR are in fact quite restrictive \citep{Otto2019}. When recombination is suppressed by an inversion, the scenario is more complicated still because multiple SA loci that may or may not be initially linked with the SLR can contribute to its overall fitness effect. The ancestral recombination rate will influence both the fixation probability by altering the equilibrium frequency of female- and male-beneficial alleles at captured SA loci, and the selective advantage of reducing recombination further.

We start with a simplified scenario to begin disentangling the effects of linkage on the fixation probability of new inversions. Suppose the average number of SA loci on the sex chromosomes is equal to $A$, that they are uniformly distributed along the chromosomes, are biallelic with standard SA fitness expressions {\itshape sensu} \citet{Kidwell1977} (each allele is beneficial when expressed in one sex, but deleterious when expressed in the other; see Table \ref{tab:fitness}), and are initially at equilibrium. Under our assumption that inversion breakpoints are randomly distributed along the chromosome arm, the number of SA loci spanned by a new inversion, $n$, is a Poisson distributed random variable with mean and variance $xA$. For now, we assume that $A$ is sufficiently small to ignore the possibility that $n$ is greater than about $1$ (the approximation breaks down when $A > 1$; we consider the case with multiple SA loci below). We focus on two idealized scenarios: the SLR and SA locus ($1$) recombine freely at a rate $r = 1/2$ per meiosis (i.e., the SA locus is located in the {\itshape a}-PAR); and ($2$) the SLR and SA locus are partially linked, and recombine at a rate $0 \leq r < 1/2$ (i.e., the SA locus is located in the {\itshape sl}-PAR). 

{\bf \itshape Effect of linkage between the SLR and SA locus} -- Considering, for the moment, inversions that already span the SLR, the fixation probability for a new inversion of size $x$ that also spans a single unlinked SA locus on the Y chromosome is the product of three probabilities: ($1$) that the inversion captures the SA locus, $\Pr(n = 1) = xA e^{-xA}$; ($2$) that it captures a male-beneficial allele at the SA locus, $\Pr(\text{male~ben.}) = \hat{q}$, where $\hat{q}$ is the equilibrium frequency of the male-beneficial allele; and ($3$) that it escapes stochastic loss due to genetic drift and fixes in the population, $\Pr(\text{fix}) \approx 2 s_I$ \cite{Haldane1927}. We can approximate the expected rate of increase of a rare inversion as $s_I \approx (\lambda_I - 1)$, where $\lambda_I$ is the eigenvalue associated with invasion of the inversion genotype into a population inititally at equilibrium in a deterministic two-locus model involving the SLR and SA locus ($\lambda_I$ is also the leading eigenvalue under these conditions). When the SA locus is unlinked with the SLR ($r = 1/2$), the selection coefficient for the rare inversion is

\begin{equation}\label{eq:SApFix2LocUnlinked}
	s_I \approx s_m (1 - \hat{q}) \big( 1 - \hat{q} - h_m(1 - 2\hat{q}) \big) + O(s_{m}^{2}),
\end{equation}

\noindent where $s_m$ is the selection coefficient of the male-deleterious/female-beneficial allele in males. With additive SA fitness ($h_f = h_m = 1/2$), the fixation probability reduces to

\begin{equation}\label{eq:SApFix2LocUnlinkedYAdd}
	\Pr(\text{fix} \mid x,n=1) = s_m \hat{q}(1 - \hat{q}) xA e^{-xA}.
\end{equation}

\noindent Eq(\ref{eq:SApFix2LocUnlinkedYAdd}) is convex sigmoidal increasing function of inversion size over $0 < x \leq 1$, with a maximum at $\tilde{x} = 1/A $, implying that larger inversions are always favoured (recall that $A \leq 1 $). Intuitively, larger inversions are more likely to capture rare SA loci distributed uniformly along the chromosome arm.

How does linkage between the SLR and SA locus alter the fixation probability? We now make two additional simplifying assumptions: the SA locus falls within the {\itshape sl}-PAR, which makes up a fraction, $P$, of the total chromosome arm length, and that $P \ll x$. Hence, any inversion that spans the SLR will also span the the {\itshape sl}-PAR. Put simply, we assume that genetic linkage requires relatively tight physical linkage between the SLR and the SA locus. The probability of spanning the SA locus is now $\Pr(n = 1) = AP e^{-AP}$. Relaxing this assumption results in predictions that are intermediate with the unlinked scenario. We can approximate $s_I \approx (\lambda_I - 1)$ from the deterministic two-locus model as before, but the expression now involves the equilibrium frequency of the male-beneficial allele on Y chromosomes ($\hat{Y}$) and X chromosomes in females ($\hat{X}_f$) before the inversion occurs:

\begin{equation}\label{eq:SAsI2LocLinkedY}
	s_I \approx \frac{ s_m(1 - \hat{Y}) \big( 1 - \hat{X}_f - h_m(1 - 2\hat{X}_f) \big)} { 1 - s_m \big(1 - \hat{X}_f - \hat{Y}(1 - h_m - \hat{X}_f) + h_m \hat{X}_f(1 - 2 \hat{Y}) \big) }.
\end{equation}

\noindent When expressed in terms of the equilibrium allele frequencies on the three chromosome types, the ancestral recombination rate ($r$) drops out of Eq(\ref{eq:SAsI2LocLinkedY}). Prior linkage between the SLR and SA loci influences the strength of indirect selection for the inversion by altering the equilibrium frequencies of the male-beneficial allele on Y chromosomes, and X chromosomes in females. Interestingly, the effect of $r$ on the overall selection coefficient for the inversion can take different forms, depending on the relative strength of selection on the SA alleles in males and females Fig(\ref{fig:recombEffect}). In this way, the SA selection coefficients can influence whether inversions capturing loosely linked (e.g., located in the {\itshape a}-PAR) or tightly linked (e.g., located physically close to the SLR in the {\itshape sl}-PAR) are more strongly favoured.

Under additive SA selection ($h_f = h_m = 1/2$), the fixation probability simplifies to

\begin{equation}\label{eq:SAsIpFix2LocLinked}
	\Pr(\text{fix} \mid x,n=1,\text{{\itshape sl}-PAR}) = \frac{ 2 s_m \hat{Y} (1 - \hat{Y}) A P e^{-A P}}{ 2 - s_m (2 - \hat{X}_f - \hat{Y}) },
\end{equation}

\noindent which is independent of $x$. \vspace{12pt}

\noindent {\itshape Key result: The overall effect of physical and genetic linkage between the SLR and SA locus is to shift the fixation probability towards smaller inversions. This is because large inversions no longer have an increased probability of spanning the SA locus. In the limiting case where $P \ll x$, the fixation probability is independent of inversion size. Relaxing this assumption will weaken the linkage-induced bias towards smaller inversions. Sex-biases in selection can alter how tightly linked the SLR and SA locus must be to maximize the fixation probability.}
\vspace{12pt}

 \begin{figure}[!htbp]
 \centering
 \includegraphics[scale=0.5]{./recombEffect}
 \caption{Overall selection coefficient ($s_I$) for an inversion linking the SLR and a male-beneficial allele at a SA locus within the {\itshape sl}-PAR (as defined by Eq[\ref{eq:SAsI2LocLinkedY}]) as a function of the ancestral recombination rate between the two loci ($r$). Panel A shows $s_I$ when there is equal selection on female- and male-beneficial alleles ($s_f = s_m$) and additive SA fitness effects ($h_f = h_m = 1/2$). Panel B shows the same for female biased selection ($s_f < s_m$; recall from table \ref{tab:fitness} that SA selection coefficients represent the decrease in relative fitness of either SA allele in males and females); specifically, for the special case where $s_f$ is equal to the single-locus invasion condition for the male-beneficial allele ($s_f = s_m / (1 - s_m)$).}
 \label{fig:recombEffect}
 \end{figure}


{\bf \itshape Effect of deleterious mutations} -- Once an inversion capturing the SLR and a male-beneficial allele at the SA locus successfully establishes, it will behave much like an unconditionally beneficial inversion, and the effects of deleterious mutations can be taken into account as in Eq(\ref{eq:benPrFix}). Under weak selection and additive SA fitness, the overall fixation probability for an inversion spanning the SLR and an SA locus falling within the {\itshape a}-PAR or {\itshape sl}-PAR will be 


\begin{subequations}\label{eq:SAPfixWithDel}
	\begin{align*}
	\Pr (\text{fix} \mid x,\,a\text{-PAR},\, k=0) &\approx 2 s_I x A e^{-xA} \Bigg[1 + \frac{U_d x}{1 - \big(1 - s_I \big) e^{-s_d}} \Bigg]e^{\frac{-Ud x}{s_d}} \numberthis
	\end{align*}
	\text{and}
	\begin{align*}
	\Pr (\text{fix} \mid x,\,sl\text{-PAR}) &\approx 2 s_I A P e^{-A P} \Bigg[1 + \frac{U_d x}{1 - (1 - s_I) e^{-s_d}} \Bigg]e^{\frac{-Ud x}{s_d}} \numberthis
	\end{align*}
\end{subequations}

\noindent respectively. Intermediately sized inversions have the greatest fixation probability when the SA locus is initially unlinked with the SLR, but smaller inversions are always favoured when the SA locus falls within the {\itshape sl}-PAR (figure \ref{fig:fixProbFig}C). \vspace{12pt}

\noindent {\itshape Key result: When the SA locus is initially unlinked with the SLR, intermediately sized inversions have the highest fixation probability because they balance the countervailing effects of inversion size on the likelihood of successfully capturing the SLR and SA loci (larger is better), and minimizing the chance of capturing deleterious mutations (smaller is better). When the SA locus falls within the sl-PAR, inversion size no longer influences the probability of capturing the SA locus, but smaller inversions still minimize the chance of captursubling deleterious mutations, and so they are always favoured.} \vspace{12pt}

{\bf \itshape Multiple SA loci} -- When inversions can span more than one SA locus (i.e., when $A > 1$), the effect of prior linkage between the SLR and SA loci on the fixation probability will depend on the size of the {\itshape sl}-PAR, and satisfying analytic approximations become elusive. However, under our stated assumption that the {\itshape sl}-PAR is small ($P \ll 1$), the effect of linkage will generally weaken because SA loci distributed randomly along the chromosome arm are more likely to fall within the {\itshape a}-PAR. Analogous to previous models of inversions capturing locally adaptive alleles \citep{KirkpatrickBarton2003, Connallon2018}, a new Y-linked inversion may capture male-beneficial alleles at a subset $M$ of the $n$ SA loci it spans, where $M \sim \text{Bin}(n \mid \overline{q})$, and $\overline{q}$ is the average equilibrium frequency of male beneficial alleles across the $n$ loci. With no epistasis, weak selection, and loose linkage among SA loci, the fixation probability of new inversions is

\begin{equation}\label{eq:SApFixMultiLoc}
	\Pr(\text{fix} \mid x) = \Pr(\text{fix} \mid n) \Pr(n \mid x) \approx 2 s_I e^{-x A} \frac{(xA)^n}{n!},
\end{equation}

\noindent where

\begin{equation}\label{eq:SASIMultiLoc}
	s_I \approx \sum_{i \in n} s_{m,i} (1 - \hat{q}_{i}) \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) - \sum_{i \in (n-M)} s_{m,i} \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) + O(s_{i,m}^2),
\end{equation}

\noindent and $0$ for $s_I < 0$. More detailed assumptions are necessary to model the possibility of linkage between the SLR and a subset of captured SA loci (e.g., a quantitative description of the recombination rate within the {\itshape sl}-PAR). However, when selection is weak and SA loci are not tightly linked with the SLR, higher-order linkage effects between SA loci within the {\itshape sl}-PAR can be ignored (\citealt{Otto2019}). In this case, the fixation probability is well approximated by substituting 

\begin{align*}\label{eq:SASIMultiLocLinked}
		s_I \approx \sum_{i \in (n - L)} s_{m,i} (1 - \hat{q}_{i}) \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) &+ \sum_{i \in L} s_{m,i}(1 - \hat{Y}) \big( 1 - \hat{X}_{f,i} - h_{m,i}(1 - 2 \hat{X}_{f,i}) \big)~- \numberthis\\
		\sum_{i \in (n-L-M)} s_{m,i} \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) &+ \sum_{i \in (L-M)} \big( 1 - \hat{X}_{f,i} - h_{m,i}(1 - 2 \hat{X}_{f,i}) \big),
\end{align*}

\noindent into Eq(\ref{eq:SApFixMultiLoc}), where $L$ denotes the set of SA loci falling within the {\itshape sl}-PAR ($\text{E} [L] = AP$). With deleterious mutations, the multilocus fixation probability becomes

\begin{equation} \label{eq:SApFixMultiLocDelMut}
	\Pr(\text{fix} \mid x, k = 0) \approx 2 s_I e^{-xA} \frac{(xA)^n}{n!} \Bigg[ 1+ \frac{U_d x}{1 - (1-s_I)e^{-s_d}} \Bigg] e^{\frac{-					U_d x}{s_d}}.
\end{equation}

\noindent where $s_I$ is defined as in Eq(\ref{eq:SASIMultiLoc}) and Eq(\ref{eq:SASIMultiLocLinked}).
\vspace{12pt}

\noindent {\itshape Key result: The effect of prior linkage between the SLR and SA loci on the fixation probability of different sized inversions will generally weaken when multiple SA loci are distributed along the sex chromosomes. However, this effect will ultimately depend on the size of the sl-PAR, which is assumed to be small in our models.}
\vspace{12pt}

{\bf \itshape Inversions on the X} -- Results for inversions on X chromosomes can be derived by similar steps. However, because they are exposed to selection in both males and females, X-linked inversions can invade over a smaller fraction of parameter space than Y-linked inversions, and are generally maintained as balanced polymorphisms when they do (Figure~\ref{fig:detInvFreqSA}). 
\vspace{12pt}

\noindent {\itshape Key result: X-linked inversions may contribute to reduced recombination between sex chromosomes as segregating polymorphisms, but are far less likely to cause permanent recombination suppression than Y-linked inversion.}

%-- for example, by drifting to fixation whilst segregating at high frequencies -- but probably do so less often than Y-linked inversions.


 \begin{figure}[!htbp]
 \centering
 \includegraphics[scale=0.4]{./detEqInvFreqFig}
 \caption{Equilibrium frequency of new inversions capturing the SLR and a single sexually antagonistic locus on the Y ($\hat{Y}$, panels A and B) and the X chromosomes ($\hat{X}$, panels C and D), under loose (panels A and C) and tight (panels B and D) linkage between the two loci, and additive SA fitness effects ($h_f = h_m = 1/2$). Initial equilibrium genotypic frequencies were calculated by iterating the 2-locus deterministic recursions in the absence of an inversion. Once this initial equilibrium was reached, an (heterozygote) inversion genotype was introduced at low frequency ($10^{-6}$), and the recursions were again iterated until all genotypic frequencies remained unchanged. Note the different color scale for Y and X inversions. Recursions are presented in the Supplementary Materials.}
 \label{fig:detInvFreqSA}
 \end{figure}




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% ORIGINAL SUPPLEMENT TEXT


%%%%%%%%%%%%
\subsection{Fixation Probability}

As described in the the main article, our models make several key assumptions, including ({\itshape i}) that deleterious mutations are at mutation-selection balance at loci on the sex chromosomes that are unlinked with the sex determining region (SDR) and one another; ({\itshape ii}) that new inversions are unlikely to fix unless they are initially free of deleterious mutations; and ({\itshape ii}) that the timescale for inversion fixation is shorter than that of gene loss/degeneration or dosage compensation due to recombination suppression between sex chromosomes (i.e., both inversion and non-inversion X and Y chromosomes have functional homologs for the same genes). Under these assumptions, a mutation-free inversion will enjoy a temporary fitness advantage over the average non-inverted chromosome in the population. Over time, however, mutations will accumulate on the inversion chromosomes until they too are at equilibrium, equalizing the fitness of inverted and non-inverted chromosomes. Any approximation of the fixation probability for neutral inversions that are initially free of deleterious mutations must therefore take into account how frequent inversion descendants would become in the population before they become selectivley neutral.

Prior to any recombination suppression between sex chromosomes, the dynamics of deleterious mutations at loci unlinked with the SDR are identical to those at autosomal loci. Hence, the equilibrium frequency of deleterious alleles at each locus will be $\hat{p} \approx \frac{\mu}{h_d s_d}$, where $\mu$ is the single-locus deleterious mutation rate. Following \citet{Nei1967} (their Eq[7] and Eq[8]), the general solution for the approximate frequency of deleterious mutations over time at a given locus is

\begin{equation}\label{eq:delMutAccum}
	p_t = \hat{p}(1 - e^{-h_d s_d}).
\end{equation}

\noindent The overall mutation rate in the chromosomal segment spanned by an inversion is equal to $U_d x = n \mu$, where $n$ is the number of loci spanned by the inversion. 


%%%%%%%%%%%%
\subsubsection*{Inversions spanning the SDR on the Y chromosome}

From Eq(\ref{eq:delMutAccum}), we can write the fitness of individuals carrying inversions spanning the SDR on a Y chromosome as follows:

 \begin{table}[htbp]\label{tab:NeutralYinvFitTab}
 \centering
 \caption{\bf Fitness of inversion and non-inversion genotypes (Y chromosome).}
 \begin{tabu}to 11cm {X[1,l] X[1,l] X[1,l]} \hline
 Generation & $X \mid Y_I$ & $X \mid Y$ \\
 \hline 
 $0$ & $e^{-U_d x}$ & $e^{-2 U_d x}$ \\
 $t$ & $e^{-U_d x(2 - e^{-h_d s_d t})}$ & $e^{-2 U_d x}$ \\
 $t$ (relative to $X \mid Y_I$) & 1 & $e^{- U_d x e^{-h_d s_d t}}$ \\
 \hline
 \multicolumn{3}{p{3.3\tabucolX}}{{\footnotesize Note: Fitness expressions are given for the $0^{\text{th}}$ and $t^{\text{th}}$ generation when an inversion capturing the SDR on a Y chromosome is initially free of deleterious mutations. Note also that these fitness expressions are approximate for $h_d \approx 1/2$.}}
 \end{tabu}
 \end{table}
 \newpage{}

\noindent Given these time-dependent fitness expressions, we can write the discrete time recursion for the frequency ($q$) of inversion descendents at time $t + 1$ as:

\begin{equation}\label{eq:NeutralYinvRec}
	q_{t+1} = \frac{ q_t e^{-U_d x(2 - e^{-h_d s_d t})} }{(1 - q_t) e^{-2 U_d x} + q_t e^{-U_d x(2 - e^{-h_d s_d t})}},
\end{equation} 

\noindent or more simply:

\begin{equation}\label{eq:NeutralYinvRecRatio}
	\frac{q_{t+1}}{1- q_{t+1}} = \frac{q_{t}}{1- q_{t}} \text{Exp}[U_d x e^{-h_d s_d t}].
\end{equation} 

\noindent The general solution to Eq(\ref{eq:NeutralYinvRecRatio}) is 

\begin{equation}\label{eq:NeutralYinvGenSol}
	q_{t} = \frac{q_{0} \, \text{Exp}\Bigg[ U_d x \frac{(1 - e^{-h_d s_d t})}{(1 - e^{-h_d s_d})} \Bigg]} {1 - q_{0} + q_0 \, \text{Exp}\Bigg[ U_d x \frac{(1 - e^{-h_d s_d t})}{(1 - e^{-h_d s_d})} \Bigg]}.
\end{equation} 

\noindent \citep[see][]{ConnallonOlito2021}. In the limit of $t \rightarrow \infty$, the frequency of inversion descendents will converge to 

\begin{equation}\label{eq:NeutralYinvGenSolLimit}
	q_{t} = \frac{q_{0} \, \text{Exp}\Big[ \frac{U_d x}{1 - e^{-h_d s_d} } \Big]} {1 - q_{0} + q_0 \, \text{Exp}\Big[ \frac{U_d x}{1 - e^{-h_d s_d} } \Big]}.
\end{equation} 

\noindent When $N_Y$ is large, a single-copy of an initially deleterious mutation-free neutral inversion will reach an 'effective frequency' of

\begin{align*}\label{eq:NeutralYinvQEff}
	q^Y_{\text{eff}} & \approx N^{-1}_Y \, \text{Exp}\Big[ \frac{U_d x}{1 - e^{-h_d s_d} } \Big] \\
	 &\approx N^{-1}_Y \, \text{Exp}\Big[ \frac{U_d x}{h_d s_d} \Big] \numberthis,
\end{align*} 

\noindent after which it will evolve neutrally. 

Multiplying $q_{\text{eff}}$ by the probability that the inversion is initially free of deleterious mutations, $\Pr(k = 0 \mid x) = e^{-\frac{U_d x}{s_d}}$, gives the overall fixation probability presented in Eq(\hl{2}a) of the main text:

\begin{align*}\label{eq:NeutralYinvPFix}
		\Pr(\text{fix} \mid x, k = 0) &\approx N^{-1}_Y \, \text{Exp}\Big[ \frac{U_d x}{h_d s_d} \Big] e^{\frac{-U_d x}{h_d s_d}} \\
		 &= N^{-1}_Y , \numberthis
\end{align*}

Overall, the fact that larger inversions are less likely to be initially free of deleterious mutations offsets the greater temporary fitness benefit of actually being mutation-free. The fixation probability for neutral inversions remains approximately unchanged in the presence of deleterious mutation pressure that is approximately equal to $q_0 = N^{-1}_Y$. increasing risk of catching overall fixation probability of an initially mutation-free neutral inversion spanning the SDR on a Y chromosome is approximately equal to $q^Y_{\text{eff}}$, as presented in Eq(\hl{2}a) of the main text.



%%%%%%%%%%%%%%%
\subsubsection*{Inversions spanning the SDR on an X chromosome}

The frequency dynamics of initially mutation-free inversions spanning the SDR on the X chromosome turn out to be nearly identical to both autosomal inversions and those on the Y chromsome. Here we briefly outline the one key difference between the results provided for inversions on the X vs.~Y chromosomes. 

We can write the fitness of individuals carrying inversions spanning the SDR on an X chromosome from Eq(\ref{eq:delMutAccum}) as follows:

 \begin{table}[htbp]\label{tab:NeutralXinvFitTab}
 \centering
 \caption{\bf Fitness of inversion and non-inversion genotypes (X chromosome).}
 \begin{tabu}to 14cm {X[1,l] X[1,l] X[1,l] X[1,l]} \hline
 \multicolumn{4}{l}{\bf{ Females}} \\
 Generation & $X_I \mid X_I$ & $X \mid X_I$ & $X \mid X$ \\
 \hline 
 $0$ & $1$ & $e^{-U_d x}$ & $e^{-2 U_d x}$ \\
 $t$ & $e^{-2 U_d x(1 - e^{-h_d s_d t})}$ & $e^{-U_d x(2 - e^{-h_d s_d t})}$ & $e^{-2 U_d x}$ \\
 $t$ (relative to $X_I \mid X_I$) & 1 & $e^{-U_d x e^{-h_d s_d t}}$ & $e^{-2 U_d x e^{-h_d s_d t}}$ \\
 $t$ (approx.) & 1 & $1 - U_d x e^{-h_d s_d t}$ & $1 - 2 U_d x e^{-h_d s_d t}$ \\
 \hline
 \multicolumn{4}{l}{\bf{Males}} \\
 Generation & $X_I \mid Y$ & $X \mid Y$ & \\
 \hline 
 $0$ & $e^{-U_d x}$ & $e^{-2 U_d x}$ & \\
 $t$ & $e^{-U_d x(2 - e^{-h_d s_d t})}$ & $e^{-2 U_d x}$ & \\
 $t$ (relative to $X \mid Y_I$) & 1 & $e^{-2 U_d x e^{-h_d s_d t}}$ & \\
 $t$ (approx.) & 1 & $1 - U_d x e^{-h_d s_d t}$ & \\
 \hline
 \multicolumn{4}{p{4.4\tabucolX}}{{\footnotesize Note: Fitness expressions are given for the $0^{\text{th}}$ and $t^{\text{th}}$ generation when an inversion capturing the SDR on an X chromosome is initially free of deleterious mutations. Note also that these fitness expressions are approximate for $h_d \approx 1/2$.}}
 \end{tabu}
 \end{table}

\noindent Unlike the model for inversions on the Y chromosome, there is no general solution to the discrete time recursion for the frequency of inversion descendents when the inversion spans the SDR on an X chromosome. Nevertheless, we can use a continuous time approximation to the general solution for the discrete time model. Under weak selection and mutation, the rate of change in the frequency of inversion descendents is roughly 

\begin{equation}\label{eq:NeutralXinvDiffEq}
	\frac{\partial q_t}{\partial t} = q_t(1 - q_t)U_d x \, e^{-h_d s_d t} + O(U_d^2).
\end{equation}

\noindent The general solution to Eq(\ref{eq:NeutralXinvDiffEq}) is

\begin{equation}\label{eq:NeutralXinvGenSol}
	q_{t} = \frac{q_{0} \, \text{Exp}\big[ \frac{U_d x(1 - e^{-h_d s_d t})}{h_d s_d } \big]} {1 - q_{0} + q_0 \, \text{Exp} \big[ \frac{U_d x(1 - e^{-h_d s_d t})}{h_d s_d } \big]}.
\end{equation} 

\noindent In the long term, the frequency of inversion descendents converges deterministically on 

\begin{equation}\label{eq:NeutralXinvqEff}
	q_{t} = \frac{q_{0} \, e^{\frac{U_d x}{h_d s_d }}} {1 - q_{0} + q_0 \, e^{\frac{U_d x}{h_d s_d }}}.
\end{equation} 

\noindent Finally, when $N_X$ is large, a single-copy of an initially deleterious mutation-free neutral inversion will reach an 'effective frequency' of

\begin{equation}\label{eq:NeutralXinvQEff}
	q^X_{\text{eff}} \approx N^{-1}_X \, e^{\frac{U_d x}{h_d s_d }},
\end{equation} 

\noindent after which it will evolve neutrally. $q^X_{\text{eff}}$ therefore provides an approximation for the fixation probability of an initially mutation-free neutral inversion spanning the SDR on an X chromosome. 

Multiplying $q_eff$ by the probability that the inversion is initially free of deleterious mutations, $\Pr(k = 0 \mid x) = e^{-\frac{U_d x}{s_d}}$, gives the overall fixation probability presented in Eq(\hl{2}b) of the main text:

\begin{align*}\label{eq:NeutralXinvPFix}
		\Pr(\text{fix} \mid x, k = 0) &\approx N^{-1}_X \, e^{\frac{U_d x}{h_d s_d}} e^{\frac{-U_d x}{h_d s_d}} \\
		&= N^{-1}_X, \numberthis
\end{align*}

Comparing Eq(\ref{eq:NeutralYinvPFix}) with Eq(\ref{eq:NeutralXinvPFix}) for equivalent effective population sizes shows the general solution for the continuous time model provides a good approximation to the discrete-time model, but works best when $U_d$, $s_d$, and $x$ are all small. 

\newpage{}





%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Sexual Antagonism}\label{AppB}
 \renewcommand{\theequation}{B\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{B\arabic{figure}}
 \setcounter{figure}{0}

In the main text we present analyses of simplified two-locus models of sexually antagonistic selection. These results represent modest extensions of earlier population genetic models of the PAR including \citet{Clark1987}, \citet{Otto2011}, and \citet{Otto2014, Otto2019}. We refer readers to these papers for comprehensive theoretical background on two-locus PAR models. 

We use the two-locus models to examine the invasion conditions for rare inversions spanning the SDR and other selected loci, and to approximate the overall selection coefficient for inversions, taking into account the possible effects of partial linkage between the SDR and the selected locus prior to the inversion mutation. The recursions for each of these models are structurally alike, and the evolutionary invasion analyses follow similar steps. Here, we develop the recursions for scenario ({\itshape iii}) sexual antagonism, and a detailed description of the invasion analysis. Details for the other models are presented in the Mathematica notebook files (.nb) available in the Online Supplementary Materials.


%%%%%%%%%%%%
\subsection{Recursions (Y chromosome inversion)}

Consider the two-locus genetic system described in the main text: one locus determines whether a chromosome is considered X or Y, with XX individuals being female, and XY individuals being male (YY are considered lethal); and a second locus having alleles $A$ and $a$ that may be linked to the sex-determining region and is subject to natural selection. Recombination between the two loci occurs at a rate $r$ per meiosis. Generations are discrete, and the population size is assumed to be large enough that drift is negligible. The life cycle proceeds: haploid selection $\rightarrow$ random mating $\rightarrow$ diploid selection.

When studying the invasion of an inversion capturing the SDR and one allele at a selected locus on the Y chromosome, there are three relevant female genotypes: $AA$, $Aa$, and $aa$, with frequencies denoted $x_1$, $x_2$, and $x_3$, and general fitness expressions at selection denoted $w_{f,1}$, $w_{f,2}$, $w_{f,3}$. However, there are six relevant genotypes for males: $AA$, $Aa$ (cis-), $Aa^I$ (cis-), $aA$ (trans-), $aa$, and $aa^I$, with frequencies $y_{1}$, $y_{2c}$, $y^I_{2c}$, $y_{2t}$, $y_{3}$, $y_{3I}$, but fitness expressions $w_{m,1}$, $w_{m,2}$, $w_{m,3}$. Note that male heterozygote genotype labels indicate whether the $A$ allele is located on the X chromosome ($y_{2c}$), or on the Y chromosome ($y_{2t}$), and "$I$" superscripts denote inverted haplotypes. It is assumed that recombination is suppressed between inverted and non-inverted chromosomes.

\begin{equation*}
	\text{Female genotypes}:\left( \begin{array}{cc|c}
		x_1: & XA & XA \\
		x_2: & XA & Xa \\
		x_3: & Xa & Xa 
	\end{array} \right)
\end{equation*}

\begin{equation*}
	\text{Male genotypes}:\left( \begin{array}{cc|c}
		y_1:     & XA & YA \\
		y_{2c}:   & XA & Ya \\
		y^I_{2c}: & XA & Ya^I \\
		y_{2t}:   & Xa & YA \\
		y_{3}:    & Xa & Ya \\
		y^I_{3}:  & Xa & Ya^I 
	\end{array} \right)
\end{equation*}

\noindent The genotypic frequencies among females after recombination and random mating are:
\begin{align*}
	x^m_{1} &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^m_{2} &= \Big( x_1 + \frac{x_2}{2} \Big) \big( y_{2c} \cdot r + y_{2t}(1 - r) + y_3 + y^I_3 \big)~+ \\
	&~~~~\Big( x_3 + \frac{x_2}{2} \Big) \big( y_{1} + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big)      \\
	x^m_{3} &= \Big( x_3 + \frac{x_2}{2} \Big) \big(y_{2c} \cdot r + y{2t}(1 - r) + y_{3} + y^I_3  \big) \numberthis
\end{align*}

\noindent and among males:
\begin{align*}
	y^m_{1}      &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y_1 + y_{2c} \cdot r + y_{2t}(1 - r)  \big) \\
	y^m_{2c}     &= \Big( x_1 + \frac{x_2}{2} \Big) \big( y_{2c} (1 - r) + y_{2t} \cdot r + y_3 \big) \\
	y^{I,m}_{2c} &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y^I_{2c} + y^I_{3} \big)  \\
	y^m_{2t}     &= \Big( x_3 + \frac{x_2}{2} \Big) \big( y_{2c} \cdot r + y_{2t}(1 - r) + y_1 \\
	y^m_{3}      &= \Big( x_3 + \frac{x_2}{2} \Big) \big( y_{2c}(1 - r) + y_{2t} \cdot r + y_3 \\
	y^{I,m}_{3}  &= \Big( x_3 + \frac{x_2}{2} \Big) \big(y^I_{2c} + y^I_{3} \big)  \numberthis
\end{align*}

\noindent After selection, the genotypic frequencies among females will be:
\begin{align*}
	x'_{1} &= \frac{x^m_1 w_{f,1}}{\overline{w}_{f}}\\
	x'_{2} &= \frac{x^m_2 w_{f,2}}{\overline{w}_{f}}\\
	x'_{3} &= \frac{x^m_3 w_{f,3}}{\overline{w}_{f}} \numberthis
\end{align*}

\noindent where $\overline{w}_{f} = x^m_1 w_{f,1} + x^m_2 w_{f,2} + x^m_3 w_{f,3}$. For males, the genotypic frequencies after selection are:
\begin{align*}
	y'_{1}       &= \frac{y^m_1 w_{m,1}}{\overline{w}_{m}} \\
	y'_{2c}      &= \frac{y^m_{2c} w_{m,2}}{\overline{w}_{m}} \\
	y'^{I}_{2c}  &= \frac{y^{I,m}_{2c} w_{m,2}}{\overline{w}_{m}}  \\
	y'_{2t}      &= \frac{y^m_{2t} w_{m,2}}{\overline{w}_{m}} \\
	y'_{3}       &= \frac{y^m_3 w_{m,3}}{\overline{w}_{m}} \\
	y'^{I}_{3}   &= \frac{y^{I,m}_3 w_{m,3}}{\overline{w}_{m}} \numberthis
\end{align*}

\noindent where $\overline{w}_{m} = y^m_1 w_{m,1} + y^m_{2c} w_{m,2} + y^{I,m}_{2c} w_{m,2} + y^m_{2t} w_{m,2} + y^m_3 w_{m,3} + y^{I,m}_3 w_{m,3}$.



%%%%%%%%%%%%
\subsection{Eigenvalues (Y chromosome inversion)}

Under the assumptions stated in the main article, a new inversion must capture the SDR and a male-beneficial allele at the SA locus in order to invade. Accordingly, we use SA fitness expressions {\itshape sensu} \citealt{Kidwell1977,ConnallonClark2012,Otto2011}), where $w_{f,1} = 1$, $w_{f,2} = 1 - h_f s_f$, and $w_{f,3} = 1 - s_f$, and $w_{m,1} = 1 - s_m$, $w_{m,2} = 1 - h_m s_m$, and $w_{m,3} = 1$. We are interested in the evolutionary fate of rare inversion genotypes capturing the $a$ allele at the SA locus.

Substituting $x_3 = 1 - x_1 - x_2$ and $y_3 = 1 - y_1 - y_{2c} - y^I_{2c} - y_{2t} - y^I_3$, yields a system of $7$ genotypic frequency recursions. A key assumption in the analysis is that all non-inversion genotypes are initially at equilibrium when a new inversion mutation occurs. Specifically, we assume that $x_i = \hat{x}_i$ for $i \in \{1,\,2 \}$ and $y_j = \hat{y}_j$ where $j \in \{ 1,\,2c,\,2t \}$. To solve for the conditions under which a rare inversion capturing the SDR and the male-beneficial allele at the SA locus, we evaluate the Jacobian at the boundary equilibrium where the initial frequency of inverted genotypes are $y^{I}_{2c} = y^{I}_{3} = 0$:

\begin{equation*}
	\mathbb{J} = \left( \begin{array}{ccccccc}

					\frac{\partial x'_1}{\partial x_1} &
					\frac{\partial x'_2}{\partial x_1} &
					\frac{\partial y'_1}{\partial x_{1}} &
					\frac{\partial y'_{2c}}{\partial x_1} &
					\frac{\partial y'^I_{2c}}{\partial x_1} &
					\frac{\partial y'_{2t}}{\partial x_1} &
					\frac{\partial y'^I_{3}}{\partial x_1} \\[1ex]

					\frac{\partial x'_1}{\partial x_2} &
					\frac{\partial x'_2}{\partial x_2} &
					\frac{\partial y'_1}{\partial x_{2}} &
					\frac{\partial y'_{2c}}{\partial x_2} &
					\frac{\partial y'^I_{2c}}{\partial x_2} &
					\frac{\partial y'_{2t}}{\partial x_2} &
					\frac{\partial y'^I_{3}}{\partial x_2} \\[1ex]

					\frac{\partial x'_1}{\partial y_1} &
					\frac{\partial x'_2}{\partial y_1} &
					\frac{\partial y'_1}{\partial y_{1}} &
					\frac{\partial y'_{2c}}{\partial y_1} &
					\frac{\partial y'^I_{2c}}{\partial y_1} &
					\frac{\partial y'_{2t}}{\partial y_1} &
					\frac{\partial y'^I_{3}}{\partial y_1} \\[1ex]

					\frac{\partial x'_1}{\partial y_{2c}} &
					\frac{\partial x'_2}{\partial y_{2c}} &
					\frac{\partial y'_1}{\partial y_{2c}} &
					\frac{\partial y'_{2c}}{\partial y_{2c}} &
					\frac{\partial y'^I_{2c}}{\partial y_{2c}} &
					\frac{\partial y'_{2t}}{\partial y_{2c}} &
					\frac{\partial y'^I_{3}}{\partial y_{2c}} \\[1ex]

					\frac{\partial x'_1}{\partial y^I_{2c}} &
					\frac{\partial x'_2}{\partial y^I_{2c}} &
					\frac{\partial y'_1}{\partial y^I_{2c}} &
					\frac{\partial y'_{2c}}{\partial y^I_{2c}} &
					\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
					\frac{\partial y'_{2t}}{\partial y^I_{2c}} &
					\frac{\partial y'^I_{3}}{\partial y^I_{2c}} \\[1ex]

					\frac{\partial x'_1}{\partial y_{2t}} &
					\frac{\partial x'_2}{\partial y_{2t}} &
					\frac{\partial y'_1}{\partial y_{2t}} &
					\frac{\partial y'_{2c}}{\partial y_{2t}} &
					\frac{\partial y'^I_{2c}}{\partial y_{2t}} &
					\frac{\partial y'_{2t}}{\partial y_{2t}} &
					\frac{\partial y'^I_{3}}{\partial y_{2t}} \\[1ex]

					\frac{\partial x'_1}{\partial y^I_{3}} &
					\frac{\partial x'_2}{\partial y^I_{3}} &
					\frac{\partial y'_1}{\partial y^I_{3}} &
					\frac{\partial y'_{2c}}{\partial y^I_{3}} &
					\frac{\partial y'^I_{2c}}{\partial y^I_{3}} &
					\frac{\partial y'_{2t}}{\partial y^I_{3}} &
					\frac{\partial y'^I_{3}}{\partial y^I_{3}} \\[1ex]

				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										y'^I_{2c} = y^I_{3} = 0}} \numberthis
\end{equation*}

\noindent We can isolate the candidate leading eigenvalues associated with the spread of inversion genotypes by first rearranging the Jacobian using elementary row and column operations to give a block triangular matrix \cite[Supplement~to~Primer~2]{OttoDay2007}:
\begin{equation}
	\mathbb{J}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{A} = \left( \begin{array}{ccc} 
										\frac{\partial x'_1}{\partial x_1} & \cdots & \frac{\partial y'_{2t}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'_1}{\partial y_{2t}} & \cdots & \frac{\partial y'_{2t}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) &
				\mathbf{B} = \left( \begin{array}{cc} 
										\frac{\partial y'^I_{2c}}{\partial x_1} & \frac{\partial y'^I_{3}}{\partial x_1} \\[1ex]
										\vdots & \vdots \\[1ex]
										\frac{\partial y'^I_{2c}}{\partial y_{2t}} & \frac{\partial y'^I_{3}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{C} = \left( \begin{array}{ccc} 
								0 & \cdots & 0\\[1ex]
								0 & \cdots & 0\\[1ex]
							\end{array} \right) &
		\mathbf{D} = \left( \begin{array}{cc} 
								\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} & \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} \\[1ex]
								\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} & \frac{\partial y'^I_{3}}{\partial y^I_{3}} \\[1ex]
							\end{array} \right) \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										y'^I_{2c} = y^I_{3} = 0}} \\[1ex]
\end{equation}

\noindent The eigenvalues of a block triangular matrix are the eigenvalues of the submatrices along the diagonal, which conveniently isolates the candidate leading eigenvalues for non-inversion genotypes (submatrix $\mathbf{A}$) and inversion genotypes (submatrix $\mathbf{D}$). When evaluated at $y'^I_{2c} = y^I_{3} = 0$, the two elements in the first row of submatrix $\mathbf{D}$ are identical ($ \alpha = \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} = \frac{\partial y'^I_{2c}}{\partial y^I_{3}}$), as are the two elements in the second row ($\beta = \frac{\partial y'^I_{3}}{\partial y^I_{2c}} = \frac{\partial y'^I_{3}}{\partial y^I_{3}}$). The candidate leading eigenvalue of submatrix $\mathbf{D}$ describing the spread of the rare inversion is:
\begin{equation}
	\lambda_{I} = \big( \alpha + \beta \big)
\end{equation}

\noindent $\lambda_I$ is cumbersome when expressed in terms of the adult genotypic frequencies. However, $\lambda_I$ can be transformed onto a new coordinate system of allele frequencies in the three chromosome classes among gametes: X chromosomes in ovules ($X_f$), X chromosomes in sperm ($X_m$), and Y chromosomes in sperm ($Y$) \citep{Clark1987,Otto2011,Otto2014}. $\lambda_I$ simplifies considerably when expressed on this new coordinate system:

\begin{equation}
	\lambda_I = \frac{w_{m,2}(1 - \hat{X}_f) + w_{m,3} \hat{X}_f} {w_{m,1}(1 - \hat{X}_f)(1 - \hat{Y}) + w_{m,3} \hat{X}_f \hat{Y} + w_{m,2} (\hat{X}_f + \hat{Y} - 2 \hat{X}_f \hat{Y})}
\end{equation}

\noindent If $\lambda_I > 1$, and all non-inversion genotypes are initially at equilibrium ($x_i = \hat{x}_i$, $y_j = \hat{y}_j$), as we have assumed, $\lambda_I$ will necessarily be the leading eigenvalue of the system of recursions. This is a strong assumption that does not necessarily hold once the system has been perturbed. We have therefore checked our use of $\lambda_I$ in subsequent approximations using both deterministic iteration of the recursions and Wright-Fisher simulations (see fig.~2 and fig.~5 in the main text). For inversions spanning the SDR on a Y chromosome, however, there are no internal equilibria; if the inversion can invade (i.e., if $\lambda_I > 1$), it will go to fixation. 

Substituting in the SA fitness expessions described above, and calculating $s_I \approx \lambda - 1$ provides an estimate of the overall selection coefficient for the rare inversion \cite{OttoYong2002}, which appears in the main text as \hl{Eq(8)}. Calculating $\lambda_I$ for $r = 1/2$ yields the approximation given in \hl{Eq(6)}.



%%%%%%%%%%%%
\subsection{Recursions (X chromosome inversion)}

The assumptions and development of the recursions for an inversion spanning the SDR and a single SA locus on an X chromosome largely parallel those for Y chromome inversions. We use the same SA fitness expressions described in the previous section. Now, our interest is focused on a rare inversion that captures the female-beneficial $A$ allele at the SA locus, and we have $6$ relevant genotypes for both males and females:

\begin{equation*}
	\text{Female genotypes}:\left( \begin{array}{cc|c}
		x_1:        & XA   & YA \\
		x^I_{1}:    & XA   & XA^I \\
		x^{II}_{1}: & XA^I & XA^I \\
		x_{2}:      & XA   & Xa \\
		x^I_{2}:    & XA^I & Xa \\
		x_{3}:      & Xa   & Xa 
	\end{array} \right)
\end{equation*}

\begin{equation*}
	\text{Male genotypes}:\left( \begin{array}{cc|c}
		y_1:      & XA   & YA \\
		y^I_{1}:  & XA^I & YA \\
		y_{2c}:   & XA   & Ya \\
		y^I_{2c}: & XA^I & Ya \\
		y_{2t}:   & Xa & YA \\
		y_{3}:    & Xa & Ya 
	\end{array} \right)
\end{equation*}


\noindent The genotypic frequencies among females after recombination and random mating are:
\begin{align*}
	x^m_{1}      &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{I,m}_{1}  &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big( y^I_{2c} + y^I_{2c} \big)~+ \\
				 &~~~~\Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{II,m}_{1} &= \Big( \frac{x^I_1}{2} + x^{II}_1 + \frac{x^I_2}{2} \Big) \big( y^I_{2c} + y^I_{2c} \big) \\
	x^m_2        &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big)~+ \\
				 &~~~~\Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{I,m}_2    &= \Big( \frac{x^I_1}{2} + x^{II}_{1} + \frac{x^I_2}{2} \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big)~+ \\
				 &~~~~\Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big( y^I_{2c} + y^I_{2c} \big) \\
	x^m_3        &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big) \numberthis
\end{align*}

\noindent and among males:
\begin{align*}
	y^m_{1}      &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{I,m}_{1}  &= \Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big)  \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{m}_{2c}   &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big)  \\
	y^{I,m}_{2c} &= \Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big) \big( y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big) \\
	y^m_{2t}     &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{m}_{3}    &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big( y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big)  \numberthis
\end{align*}

\noindent After selection, the genotypic frequencies among females will be:
\begin{align*}
	x'_{1}     &= \frac{x^m_1 w_{f,1}     }{\overline{w}_{f}} \\
	x'^{I}_{1}  &= \frac{x^{I,m}_1 w_{f,1} }{\overline{w}_{f}} \\
	x'^{II}_{1} &= \frac{x^{II,m}_1 w_{f,1}}{\overline{w}_{f}} \\
	x'_{2}      &= \frac{x^m_2 w_{f,2}     }{\overline{w}_{f}} \\
	x'^{I}_{2}  &= \frac{x^{I,m}_2 w_{f,2} }{\overline{w}_{f}} \\
	x'_{3}      &= \frac{x^m_3 w_{f,3}     }{\overline{w}_{f}} \\
\end{align*}
    
\noindent where $\overline{w}_{f} = x^m_1 w_{f,1} + x^{I,m}_1 w_{f,1} + x^{II,m}_1 w_{f,1} + x^m_2 w_{f,2} + x^{I,m}_2 w_{f,2} + x^m_3 w_{f,3} $. The genotypic frequencies in males after selection are:
\begin{align*}
	y'_{1}       &= \frac{y^m_1 w_{m,1}       }{\overline{w}_{m}} \\
	y'^I_{1}     &= \frac{y^{I,m}_{1} w_{m,1} }{\overline{w}_{m}} \\
	y'_{2c}      &= \frac{y^{m}_{2c} w_{m,2}  }{\overline{w}_{m}}  \\
	y'^I_{2c}    &= \frac{y^{I,m}_{2c} w_{m,2}}{\overline{w}_{m}} \\
	y'_{2t}      &= \frac{y^m_{2t} w_{m,2}    }{\overline{w}_{m}} \\
	y'_{3}       &= \frac{y^{m}_3 w_{m,3}   }{\overline{w}_{m}} \numberthis
\end{align*}
  
\noindent where $\overline{w}_{m} = y^m_1 w_{f,1} + y^{I,m}_{1} w_{f,1} + y^{m}_{2c} w_{f,2} + y^{I,m}_{2c} w_{f,2} + y^m_{2t} w_{f,2} + y^{I,m}_3 w_{f,3} $.



%%%%%%%%%%%%
\subsection{Eigenvalues (X chromosome inversion)}

Again, our analysis for the model of X chromosome inversions largely paralleles that for the Y.

Substituting $x_3 = 1 - x_{1} - x^I_{1} - x^{II}_{1} - x_{2} - x^I_{2}$ and $y_3 = 1 - y_1 - y^I_1 - y_{2c} - y^I_{2c} - y_{2t}$, yields a system of $10$ genotypic frequency recursions. As before, we make the key assumption that all non-inversion genotypes are initially at equilibrium when a new inversion mutation occurs: $x_i = \hat{x}_i$ for $i \in \{1,\,2 \}$ and $y_j = \hat{y}_j$ where $j \in \{ 1,\,2c,\,2t \}$. To solve for the conditions under which a rare inversion capturing the SDR and the male-beneficial allele at the SA locus, we evaluate the Jacobian at the boundary equilibrium where the initial frequency of inverted genotypes are $x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0$:


\begin{equation*}
	\mathbb{J} = \left( \begin{array}{cccccccccc}

					\frac{\partial x'_1      }{\partial x_1} &
					 \frac{\partial x'^I_1   }{\partial x_1} &
					 \frac{\partial x'^{II}_1}{\partial x_1} &
					 \frac{\partial x'_{2}   }{\partial x_1} &
					 \frac{\partial x'^I_{2} }{\partial x_1} &
					 \frac{\partial y'_1     }{\partial x_1} &
					 \frac{\partial y'^I_1   }{\partial x_1} &
					 \frac{\partial y'_{2c}  }{\partial x_1} &
					 \frac{\partial y'^I_{2c}}{\partial x_1} &
					 \frac{\partial y'_{2t}  }{\partial x_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x^I_1} &
					 \frac{\partial x'^I_1   }{\partial x^I_1} &
					 \frac{\partial x'^{II}_1}{\partial x^I_1} &
					 \frac{\partial x'_{2}   }{\partial x^I_1} &
					 \frac{\partial x'^I_{2} }{\partial x^I_1} &
					 \frac{\partial y'_1     }{\partial x^I_1} &
					 \frac{\partial y'^I_1   }{\partial x^I_1} &
					 \frac{\partial y'_{2c}  }{\partial x^I_1} &
					 \frac{\partial y'^I_{2c}}{\partial x^I_1} &
					 \frac{\partial y'_{2t}  }{\partial x^I_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x^{II}_1} &
					 \frac{\partial x'^I_1   }{\partial x^{II}_1} &
					 \frac{\partial x'^{II}_1}{\partial x^{II}_1} &
					 \frac{\partial x'_{2}   }{\partial x^{II}_1} &
					 \frac{\partial x'^I_{2} }{\partial x^{II}_1} &
					 \frac{\partial y'_1     }{\partial x^{II}_1} &
					 \frac{\partial y'^I_1   }{\partial x^{II}_1} &
					 \frac{\partial y'_{2c}  }{\partial x^{II}_1} &
					 \frac{\partial y'^I_{2c}}{\partial x^{II}_1} &
					 \frac{\partial y'_{2t}  }{\partial x^{II}_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x_2} &
					 \frac{\partial x'^I_1   }{\partial x_2} &
					 \frac{\partial x'^{II}_1}{\partial x_2} &
					 \frac{\partial x'_{2}   }{\partial x_2} &
					 \frac{\partial x'^I_{2} }{\partial x_2} &
					 \frac{\partial y'_1     }{\partial x_2} &
					 \frac{\partial y'^I_1   }{\partial x_2} &
					 \frac{\partial y'_{2c}  }{\partial x_2} &
					 \frac{\partial y'^I_{2c}}{\partial x_2} &
					 \frac{\partial y'_{2t}  }{\partial x_2} \\[1ex]

					\frac{\partial x'_1      }{\partial x'^I_{2}} &
					 \frac{\partial x'^I_1   }{\partial x'^I_{2}} &
					 \frac{\partial x'^{II}_1}{\partial x'^I_{2}} &
					 \frac{\partial x'_{2}   }{\partial x'^I_{2}} &
					 \frac{\partial x'^I_{2} }{\partial x'^I_{2}} &
					 \frac{\partial y'_1     }{\partial x'^I_{2}} &
					 \frac{\partial y'^I_1   }{\partial x'^I_{2}} &
					 \frac{\partial y'_{2c}  }{\partial x'^I_{2}} &
					 \frac{\partial y'^I_{2c}}{\partial x'^I_{2}} &
					 \frac{\partial y'_{2t}  }{\partial x'^I_{2}} \\[1ex]

					\frac{\partial x'_1      }{\partial y_1} &
					 \frac{\partial x'^I_1   }{\partial y_1} &
					 \frac{\partial x'^{II}_1}{\partial y_1} &
					 \frac{\partial x'_{2}   }{\partial y_1} &
					 \frac{\partial x'^I_{2} }{\partial y_1} &
					 \frac{\partial y'_1     }{\partial y_1} &
					 \frac{\partial y'^I_1   }{\partial y_1} &
					 \frac{\partial y'_{2c}  }{\partial y_1} &
					 \frac{\partial y'^I_{2c}}{\partial y_1} &
					 \frac{\partial y'_{2t}  }{\partial y_1} \\[1ex]

					\frac{\partial x'_1      }{\partial y^I_1} &
					 \frac{\partial x'^I_1   }{\partial y^I_1} &
					 \frac{\partial x'^{II}_1}{\partial y^I_1} &
					 \frac{\partial x'_{2}   }{\partial y^I_1} &
					 \frac{\partial x'^I_{2} }{\partial y^I_1} &
					 \frac{\partial y'_1     }{\partial y^I_1} &
					 \frac{\partial y'^I_1   }{\partial y^I_1} &
					 \frac{\partial y'_{2c}  }{\partial y^I_1} &
					 \frac{\partial y'^I_{2c}}{\partial y^I_1} &
					 \frac{\partial y'_{2t}  }{\partial y^I_1} \\[1ex]

					\frac{\partial x'_1      }{\partial y_{2c}} &
					 \frac{\partial x'^I_1   }{\partial y_{2c}} &
					 \frac{\partial x'^{II}_1}{\partial y_{2c}} &
					 \frac{\partial x'_{2}   }{\partial y_{2c}} &
					 \frac{\partial x'^I_{2} }{\partial y_{2c}} &
					 \frac{\partial y'_1     }{\partial y_{2c}} &
					 \frac{\partial y'^I_1   }{\partial y_{2c}} &
					 \frac{\partial y'_{2c}  }{\partial y_{2c}} &
					 \frac{\partial y'^I_{2c}}{\partial y_{2c}} &
					 \frac{\partial y'_{2t}  }{\partial y_{2c}} \\[1ex]

					\frac{\partial x'_1      }{\partial y^I_{2c}} &
					 \frac{\partial x'^I_1   }{\partial y^I_{2c}} &
					 \frac{\partial x'^{II}_1}{\partial y^I_{2c}} &
					 \frac{\partial x'_{2}   }{\partial y^I_{2c}} &
					 \frac{\partial x'^I_{2} }{\partial y^I_{2c}} &
					 \frac{\partial y'_1     }{\partial y^I_{2c}} &
					 \frac{\partial y'^I_1   }{\partial y^I_{2c}} &
					 \frac{\partial y'_{2c}  }{\partial y^I_{2c}} &
					 \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
					 \frac{\partial y'_{2t}  }{\partial y^I_{2c}} \\[1ex]

					\frac{\partial x'_1      }{\partial y_{2t}} &
					 \frac{\partial x'^I_1   }{\partial y_{2t}} &
					 \frac{\partial x'^{II}_1}{\partial y_{2t}} &
					 \frac{\partial x'_{2}   }{\partial y_{2t}} &
					 \frac{\partial x'^I_{2} }{\partial y_{2t}} &
					 \frac{\partial y'_1     }{\partial y_{2t}} &
					 \frac{\partial y'^I_1   }{\partial y_{2t}} &
					 \frac{\partial y'_{2c}  }{\partial y_{2t}} &
					 \frac{\partial y'^I_{2c}}{\partial y_{2t}} &
					 \frac{\partial y'_{2t}  }{\partial y_{2t}} \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation*}

\noindent As before, we can isolate the candidate leading eigenvalues associated with the spread of the inversion by rearranging the Jacobian to give the following block triangular matrix \cite[Supplement~to~Primer~2]{OttoDay2007}:

\begin{equation}
	\mathbb{J}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{A} = \left( \begin{array}{ccc} 
										\frac{\partial x'_1}{\partial x_1} & \cdots & \frac{\partial y'_{2t}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'_1}{\partial y_{2t}} & \cdots & \frac{\partial y'_{2t}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) &
				\mathbf{B} = \left( \begin{array}{ccc} 
										\frac{\partial x'^I_1}{\partial x_1} & \cdots & \frac{\partial y'^I_{2c}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'^I_{1}}{\partial y_{2t}} & \cdots & \frac{\partial y'^I_{2c}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{C} = \left( \begin{array}{ccc} 
								0      & \cdots & 0     \\[1ex]
								\vdots & \ddots & \vdots\\[1ex]
								0      & \cdots & 0     \\[1ex]
							\end{array} \right) &
				\mathbf{D} = \left( \begin{array}{ccc} 
										\frac{\partial x'^I_1}{\partial x^I_1} & \cdots & \frac{\partial y'^I_{2c}}{\partial x^I_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'^I_{1}}{\partial y^I_{2c}} & \cdots & \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} \\[1ex]
									\end{array} \right) \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation}

\noindent The eigenvalues calculations can be further simplified by rearranging submatrix $\mathbf{D}$ so that it is also of block triangular form (this does not affect the eigenvalues associated with changes in non-inversion genotype frequencies from submatrix $\mathbf{A}$). Ultimately, we have 
\begin{equation}
	\mathbf{D}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{W} = \left( \begin{array}{cc}
										\frac{\partial x'^I_1   }{\partial x^I_1} &
										 \frac{\partial x'^I_{2}}{\partial x^I_1} \\[1ex]
										\frac{\partial x'^I_1   }{\partial x^I_2} &
										 \frac{\partial x'^I_{2}}{\partial x^I_2} \\[1ex]
									\end{array} \right) &
				\mathbf{X} = \left( \begin{array}{ccc} 
										\frac{\partial y'^I_1    }{\partial x^I_1} &
										 \frac{\partial y'^I_{2c}}{\partial x^I_1} &
										 \frac{\partial x'^{II}_1}{\partial x^I_1} \\[1ex]
										\frac{\partial y'^I_1    }{\partial x^I_2} &
										 \frac{\partial y'^I_{2c}}{\partial x^I_2} &
										 \frac{\partial x'^{II}_1}{\partial x^I_2} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{Y} = \left( \begin{array}{cc} 
								0      & 0     \\[1ex]
								0      & 0     \\[1ex]
								0      & 0     \\[1ex]
							\end{array} \right) &
				\mathbf{Z} = \left( \begin{array}{ccc} 
										\frac{\partial y'^I_1    }{\partial y^I_1} &
										 \frac{\partial y'^I_{2c}}{\partial y^I_1} &
										 \frac{\partial x'^{II}_1}{\partial y^I_1} \\[1ex]
										\frac{\partial y'^I_1    }{\partial y^I_{2c}} &
										 \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
										 \frac{\partial x'^{II}_1}{\partial y^I_{2c}} \\[1ex]
										\frac{\partial y'^I_1    }{\partial x^{II}_1} &
										 \frac{\partial y'^I_{2c}}{\partial x^{II}_1} &
										 \frac{\partial x'^{II}_1}{\partial x^{II}_1} \\[1ex]
									\end{array} \right) \\
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation}

\noindent The relevant candidate leading eigenvalue comes from submatrix $\mathbf{W}$:

\begin{equation}
	\lambda_{I} = \big( \alpha + \beta \big)
\end{equation}

where $\alpha = \frac{\partial x'^I_1}{\partial x^I_1} = \frac{\partial x'^I_{2}}{\partial x^I_1}$ and $\beta = \frac{\partial x'^I_1}{\partial x^I_2} = \frac{\partial x'^I_{2}}{\partial x^I_2}$. Transforming $\lambda_I$ onto a coordinate system of allele frequencies on the three chromosome classes, $X_f$, $X_m$, and $Y$, yields:


\begin{equation}
	\lambda_I = \frac{w_{f,2} + \hat{X}_f(w_{f,1} - w_{f,2})} {w_{f,3}(1 - \hat{X}_f)(1 - \hat{X}_m) + w_{f,1} \hat{X}_f \hat{Y} + w_{f,2} (\hat{X}_f + \hat{X}_m - 2 \hat{X}_f \hat{X}_m)}
\end{equation}

\noindent If $\lambda_I > 1$, and all non-inversion genotypes are initially at equilibrium ($x_i = \hat{x}_i$, $y_j = \hat{y}_j$), as we have assumed, $\lambda_I$ will necessarily be the leading eigenvalue of the system of recursions. However, for inversions on the X chromosome there exist internal equilibria for the inversion genotypes which we have confirmed with deterministic iteration of the recursions (\hl{e.g., see fig.~4C,D in the main text}). 

Our approximation of $s_I \approx \lambda - 1$, must be interpreted cautiously for the model of X chromosome inversions. In particular, $2 s_I$ does not provide a valid approximation of the fixation probability for the inversion, but rather an optimistic approximation of the probability that the inversion escapes stochastic loss when rare to approach an internal deterministic equilibrium frequency. In finite populations, $2 s_I$ will overestimate the probability that a new inversion is maintained as a polymorphism in the long-term due to chance extinction of inversions maintained at low equilibrium frequencies. Hence, inversions on the X chromosome are less likely to be maintained as stable polymorphisms than suggested by the deterministic simulations (\hl{ figure 4C,D})

\newpage



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Expected length distributions of evolutionary strata} \label{AppC:DistFixedInv}
 \renewcommand{\theequation}{C\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{C\arabic{figure}}
 \setcounter{figure}{0}
%%%%%%%%%%%%%%%%%%%%%%%%

With expressions for the fixation probability of new inversions under different evolutionary scenarios in hand, it is possible to derive the corresponding expected distributions of fixed inversion sizes. Following \citet{vanValenLevins1968, Santos1986}, and \citet{ConnallonOlito2021}, the proportion of fixed inversions of length $x$ is given by 

\begin{equation} \label{eq:generalInvSizeModel}
	g(x) = \frac{\Pr(\text{fix} \mid x) f(x)} {\int \Pr(\text{fix} \mid x) f(x)\,dx},
\end{equation}

\noindent where $f(x)$ is the probability of a new inversion of length $x$, and $\Pr(\text{fix} \mid x)$ is the fixation probability given in Eq(\ref{eq:generalPrFix}) with appropriate substitutions made for each selection scenario. $x\int \Pr(\text{fix} \mid x) f(x)\,dx$ gives the mean length of fixed inversions. Little is known about how the mutational process for new inversions shapes $f(x)$, and we therefore examine two scenarios representing plausible extremes to illustrate the spectrum of possible outcomes.

On one hand, if inversion breakpoints are distributed uniformly across the chromosome arm containing the SLR, then $f(x) = 2(1 - x)$, an extreme scenario we refer to as the "random breakpoint" model \citep{vanValenLevins1968}. On the other hand, if inversion breakpoints tend to be clustered, for example in chromosomal regions with repetitive sequences, the resulting enrichment of smaller new inversions can be modeled phenomenologically using a truncated exponential distribution:

\begin{equation} \label{eq:truncExp}
	f(x) = \frac{ \lambda e^{-\lambda x}} {1 - e^{-\lambda}},
\end{equation}

\noindent where $\lambda$ is the exponential rate parameter \citep{PevznerTesler2003, PengPevznerTesler2006, ChengKirkpatrick2019,ConnallonOlito2021}. For strongly skewed distributions (e.g., $\lambda > 10$, as we assume here), the truncation effect is negligible, and $f(x)$ is approximately equal to the numerator of Eq(\ref{eq:truncExp}). We refer to this other extreme as the "exponential model". Two key results emerge from the expected distributions of evolutionary strata length.


 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.65]{./expDistFig}
 \caption{Probability density functions for fixed inversions expanding the SLR on Y chromosomes ($g(x)$ from Eq[\ref{eq:generalInvSizeModel}]). For clarity, we show results for the model of Sexual Anatagonism with an initially unlinked SA locus ($r = 1/2$). Results are shown for the same parameter values as in Fig.~\ref{fig:fixProbFig} and Fig.~\ref{fig:probCatchFixFig}: $U_d = 0.2$, $s_d = 0.02$, and $s_I = 0.02$ for Neutral and Beneficial inversion scenarios, and $s_f = s_m = 0.05$, $U_d = 0.1$, $s_d = 0.01$, $A = 1$ for the Sex Antagonism scenario.}
 \label{fig:ExpectedDistFig}
 \end{figure}


First, the results are again strongly influenced by the location of the SLR. When the SLR is located in the center of the chromosome arm, neutral inversions are expected to give rise to a triangular distribution of evolutionary strata lengths, with a mean at $x = 1/2$ (Fig.\ref{fig:ExpectedDistFig}A). Both beneficial inversions and those capturing SA loci yield largely overlapping distributions of smaller inversions, although the distribution for sex antagonism has a distinctly heavier tail under the random breakpoint model. The differences between the distributions for neutral and selected inversions becomes exaggerated when the SLR is located near one end of the chromosome arm (Fig.\ref{fig:ExpectedDistFig}C). The distribution for neutral inversions now has three distinct regions, yielding a plateau shape, while those for beneficial and sex antagonistic inversions become increasingly skewed and overlapping. The unusual form of the length distributions for neutral inversions under the random breakpoint model results from the appearance of $(1 - x)$ terms in both $f(x)$ and $\Pr(\text{SLR} \mid x)$, which cancel in different ranges of $x$ depending on the location of the SLR.  

Second, the expected length distributions of evolutionary strata are sensitive to the form of $f(x)$. In contrast to the random breakpoints model, when inversion breakpoints are clustered the predicted distributions of strata length are highly overlapping for all three selection scenarios, and are practically indistinguishable when the SLR is located near the end of the chromosome arm (Fig.~\ref{fig:ExpectedDistFig}B,D). \vspace{12pt}

\noindent {\itshape Key result: Two dominant factors influence the expected length distribution of fixed inversions expanding the SLR: the location of the ancestral SLR on the chromosome arm, and the length distribution of new inversions. While different selection scenarios are expected to result in distinct distributions under a random breakpoint model, the length distributions become practically indistinguishable under an exponential model of new inversion lengths.} 

% \begin{figure}[h!]
% \includegraphics[scale=0.7]{./recSimFig_add}
% \caption{Haplotype recursions evaluated at QE, and the resulting invasion conditions, approximate the evolutionary trajectory of the full genotypic recursions very well under additive allelic effects, even under strong selection. Predicted regions of SA polymorphism based on deterministic simulations using the genotypic recursions Eq(2) are compared against the outcome of the invasion analysis for the haplotype recursions using the QE approximation (Eq 1) across a gradient of selfing ($C$) and recombination ($r$) rates. Green points indicate parameter conditions where deterministic simulations of the genotypic recursions (Sim.) and the invasion analysis based on eigenvalues for the QE haplotype approximations (Eig.) both predicted polymorphism. Red points indicate regions where the Sim.~predicted polymorphism but the Eig.~did not; and blue points indicate the opposite. The proportion of each outcome is shown in the upper left corner of each panel. Black solid lines show the invasion conditions based on the haplotype recursions using the QE approximation for the given values of $C$ and $r$, with lines drawn for both the % two-locus and single-locus invasion conditions when $r > 0$.}
% \label{fig:addSim}
% \end{figure}
% \newpage{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Supplementary Figures} \label{SuppFigs}
 \renewcommand{\theequation}{S\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{S\arabic{figure}}
 \setcounter{figure}{0}

\begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.56]{./SchematicFig-all}
 \caption{The spread of inversions capturing the SDR on a Y chromosome under three of the main scenarios described in the main text: \textbf{(A)} beutral inversions, \textbf{(B)} directly beneficial inversions (e.g., beneficial break-point effects), and \textbf{(C)} indirectly beneficial (sexual antagonism). From left to right, each illustration depicts a sample of Y chromosomes at three time points during the spread of an inversion, highlighting several key features of the theoretical models. The left-hand panels show key outcomes when new inversions first arise, and emphasize our assumption that inversions capturing deleterious mutations (panels A-C) or female-beneficial alleles (panel C) are unlikely to spread (indicated by skull and crossbones). The center panels illustrate the spread of the inversion and highlight that the fitness advantage enjoyed by mutation-free inversions decays over time as they accumulate new deleterious mutations. Finally, the right-hand panels illustrate the eventual fixation of the inversion and the resulting expansion of the SDR region. In all panels, the chromosomal region where $0 \leq r < 1/2$ (the sl-PAR) is indicated by blue shading.}
 \label{fig:diagramFig}
 \end{figure}

\newpage


\begin{figure}[htbp]
 \centering
 \includegraphics[width=\linewidth]{./SuppFigPrCatchSDR}
 \caption{Probability that a new inversion spans the SDR as a function of inversion size, $x$, and the location of the SDR on the chromosome arm (see Eq(16) and corresponding assumptions in the main text). We illustrate the same scenarios analysed in the main article: \textbf{(A)} the SDR is located at the exact middle of the chromosome arm, $\text{SDR}_\text{loc} = 1/2$, and \textbf{(B)} the SDR is located closer to the centromere $\text{SDR}_\text{loc} = 1/10$ (this choice is arbitrary, and results are identical if the SDR is located equally close to the telomere, $\text{SDR}_\text{loc} = 9/10$). To aid comparison, an illustration of a Y chromosome arm with the corresponding SDR locations are drawn above each plot. The form of $\Pr (\text{SDR} \mid x)$ changes as $x$ increases, and shaded regions indicate the relevant regions of parameter space where the piecewise function changes form. Darker grey indicates where $x \leq y_1,y_3$, light grey indicates $y_1 < x < y_3$, and no shading indicates $x > y_1, y_3$. Note that the region corresponding to $y_1 > x > y_3$ is not applicable for the values of $\text{SDR}_\text{loc}$ examined here. The corresponding piecewise function for each region is shown in boxes.}
 \label{fig:PrCatchFixFig}
 \end{figure}


 \begin{figure}[!htbp]
 \centering
 \includegraphics[scale=0.5]{./recombEffect}
 \caption{Overall selection coefficient ($s_I$) for an inversion linking the SLR and a male-beneficial allele at a SA locus within the {\itshape sl}-PAR (as defined by Eq[\ref{eq:SAsI2LocLinkedY}]) as a function of the ancestral recombination rate between the two loci ($r$). Panel A shows $s_I$ when there is equal selection on female- and male-beneficial alleles ($s_f = s_m$) and additive SA fitness effects ($h_f = h_m = 1/2$). Panel B shows the same for female biased selection ($s_f < s_m$; recall from table \ref{tab:fitness} that SA selection coefficients represent the decrease in relative fitness of either SA allele in males and females); specifically, for the special case where $s_f$ is equal to the single-locus invasion condition for the male-beneficial allele ($s_f = s_m / (1 - s_m)$).}
 \label{fig:recombEffect}
 \end{figure}




 \begin{figure}[!htbp]
 \centering
 \includegraphics[scale=0.4]{./detEqInvFreqFig}
 \caption{Equilibrium frequency of new inversions capturing the SLR and a single sexually antagonistic locus on the Y ($\hat{Y}$, panels A and B) and the X chromosomes ($\hat{X}$, panels C and D), under loose (panels A and C) and tight (panels B and D) linkage between the two loci, and additive SA fitness effects ($h_f = h_m = 1/2$). Initial equilibrium genotypic frequencies were calculated by iterating the 2-locus deterministic recursions in the absence of an inversion. Once this initial equilibrium was reached, an (heterozygote) inversion genotype was introduced at low frequency ($10^{-6}$), and the recursions were again iterated until all genotypic frequencies remained unchanged. Note the different color scale for Y and X inversions. Recursions are presented in the Supplementary Materials.}
 \label{fig:detInvFreqSA}
 \end{figure}





%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%
% Any references in this section should also be copied into the main 
% template, under the heading
% \section*{References Cited Only in the Online Enhancements}
% except when they are already cited in the main text.
\newpage
\bibliography{../bibliography-inversionSize-ProtoSexChrom}


\end{document}
