%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Preamble
\documentclass{article}

%Dependencies
\usepackage[left]{lineno}
\usepackage{titlesec}
\usepackage{xcolor}

\newcommand\hl[1]{%
  \bgroup
  \hskip0pt\color{blue!80!black}%
  #1%
  \egroup
}
\usepackage{ogonek}
\usepackage{float}
%\usepackage{charter}
%\usepackage[charter]{mathdesign}
\usepackage{amsmath}
\usepackage{old-arrows}
\usepackage{enumitem}
\usepackage{wasysym}

% Other Packages
%\usepackage{times}
\RequirePackage{fullpage}
\linespread{1.5}
\RequirePackage[colorlinks=true, allcolors=black]{hyperref}
\RequirePackage[english]{babel}
\RequirePackage{amsmath,amsfonts,amssymb}
\RequirePackage[sc]{mathpazo}
\RequirePackage[T1]{fontenc}
\RequirePackage{url}
\usepackage{tabu}
\usepackage{booktabs}

% Bibliography
%\usepackage[authoryear,sectionbib,sort]{natbib}
\usepackage{natbib} \bibpunct{(}{)}{;}{author-year}{}{,}
\bibliographystyle{genetics}
\addto{\captionsenglish}{\renewcommand{\refname}{Literature Cited}}
\setlength{\bibsep}{0.0pt}

% Graphics package
\usepackage{graphicx}
\graphicspath{{../output/figures/}.pdf}

% New commands: fonts
%\newcommand{\code}{\fontfamily{pcr}\selectfont}
%\newcommand*\chem[1]{\ensuremath{\mathrm{#1}}}
\newcommand\numberthis{\addtocounter{equation}{1}\tag{\theequation}}
%\titleformat{\subsubsection}[runin]{\bfseries\itshape}{\thesubsubsection.}{0.5em}{}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Title Page

\title{The evolution of suppressed recombination between sex chromosomes by chromosomal inversions}
\author{Colin Olito$^{\ast}$ \& Jessica K.~Abbott}
\date{}

\begin{document}
\maketitle

\noindent{} Department of Biology, Section for Evolutionary Ecology, Lund University, Lund 223 62, Sweden.

\noindent{} $^{\ast}$ Corresponding author e-mail: \url{colin.olito@gmail.com}

\bigskip

\noindent{} \textit{Manuscript elements}: Figure~1, Figure~2, Figure~3, Figure~4, Figure~5, Table~1, Table~2; {\itshape Supplementary Material}: Appendix A -- Neutural inversions; Appendix B -- Sexual Antagonism; Appendix C -- Haploid \& Diploid Selection; Appendix D -- Supplementary Figures; Appendix E -- Mathematica code (.nb file \& PDF) to reproduce analytic results

\bigskip
\noindent{} \textit{Running Title}: Inversions on sex chromosomes

\bigskip

\noindent{} \textit{Keywords}: Sex chromosomes; Recombination; Chromosomal inversions; Sexual antagonism; Evolutionary strata.

\bigskip

\noindent{} \textit{Manuscript type}: Investigation

\bigskip

\begin{center} 
	Submitted to {\itshape Genetics} \date{\today}
\end{center}

% Set line number options
\linenumbers
\modulolinenumbers[1]
\renewcommand\linenumberfont{\normalfont\small}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Main Text

\newpage{}
\section*{Abstract}% ($\leq 250$ words)}

\noindent{} The idea that sex-differences in selection drive the evolution of suppressed recombination between sex chromosomes is well-developed in population genetics. Yet, despite a now classic body of theory, empirical evidence that sexual antagonism drives the evolution of recombination suppression remains meagre and alternative hypotheses underdeveloped. We investigate whether the length of 'evolutionary strata' formed by chromosomal inversions that expand the non-recombining sex determining region (SDR) on recombining sex chromosomes can offer an informative signature of whether, and how, selection influenced their fixation. We develop population genetic models that determine how the length of a chromosomal inversion that expands the SDR, and the location of the SDR on the sex chromosomes, affect its fixation probability for three categories of inversions: ({\itshape i}) neutral, ({\itshape ii}) directly beneficial (i.e., due to breakpoint or position effects), and ({\itshape iii}) indirectly beneficial (especially those capturing sexually antagonistic loci). Our models predict that neutral inversions should leave behind a unique signature of large evolutionary strata, but distinguishing between strata formed by direct or indirect selection on inversions will be extremely difficult. An interesting prediction of our models is that the physical location of the ancestral SDR on the sex chromosomes is the most important factor influencing the relation between inversion size and the probability of expanding the SDR. Our findings raise a suite of new questions about how physical as well as selective processes influence the evolution of recombination suppression between sex chromosomes.
\newpage{}


%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Introduction} \label{sec:Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%

Two characteristic features of sex chromosomes give them a unique role in evolutionary biology: ({\itshape i}) the presence of one or more genes providing a mechanism for sex-determination, and ({\itshape ii}) suppressed recombination in the vicinity of the sex-determining loci, possibly extending to entire chromosomes. Recombination suppression is a critical early step in sex chromosome evolution because it enables subsequent divergence between the X and Y (or Z and W) chromosomes through the accumulation of point mutations, insertions, deletions, duplications, and rearrangements. In the long term, loss of recombination leads to several familiar defining features of heteromorphic sex chromosomes such as differences in effective population size between X-linked, Y-linked, and autosomal genes, hemizygosity, and dosage compensation \citep{CharlesworthMarais2005, BergeroCharlesworth2009,BeukeboomPerrin2014}.
	
Classic population genetics theory proposes that heteromorphic sex chromosomes evolve from ancestral autosomes in several steps: a new sex-determination gene (or linked gene cluster) originates on an ancestral pair of autosomes, followed by the accumulation of sexually antagonistic variation in linkage with the sex-determining alleles -- with male-beneficial alleles associated with the proto-Y (or proto-Z) and female-beneficial alleles with the proto-X (or proto-W) chromosomes -- resulting in indirect selection for reduced recombination between these and the sex-determining gene \citep{Fisher1931, Nei1969, Charlesworth1980, Bull1983, Rice1987, Lenormand2003, CharlesworthMarais2005}. Sex-differences in selection, and especially sexually antagonistic selection, are central in this theory. Indeed, sexually antagonistic selection also plays a key role in theories for the initial evolution of separate sexes from hermaphroditism by means of genetic sex-determination \citep{Charlesworth1978a, Charlesworth1978b, Bull1983, Olito2019}, sex-chromosome turnovers \citep{vanDoornKirkpatrick2007,vanDoornKirkpatrick2010,OttoScottOsmond2018}, and even transitions from environmental to genetic sex determination \citep{MuralidharVeller2018}. 

Despite this well developed body of theory, empirical evidence that sexual antagonism drives the evolution of recombination suppression between sex chromosomes remains weak. On one hand, influential sex-limited selection experiments and population genomic analyses of heteromorphic sex chromosomes demonstrate that sexually antagonistic variation can accumulate on sex chromosomes, apparently supporting the above theory \cite[e.g.,][]{Rice1992,Chippindale2001,Gibson2002, ZhouBachtrog2012,QiuBergeroCharlesworth2013}. On the other hand, it is often difficult or impossible to determine whether the accumulation of sexually antagonistic variation in fact preceded the evolution of suppressed recombination \citep{Charlesworth1980, Rice1984, Ironside2010, Ponnikas2018}. Recent studies identifying sexually antagonistic variation within sex linked regions on established sex chromosomes provide meagre support for the above theory for the same reason \cite[e.g.][]{BergeroCharlesworth2009,QiuBergeroCharlesworth2013,KirkpatrickGuerrero2014, Wright2017, BergeroCharlesworth2019}.

However, several other processes besides sexual antagonism have beeen proposed that could cause the evolution of suppressed recombination between sex chromosomes, including: (1) genetic drift -- e.g., neutral or nearly-neutral chromsomal rearrangements or accumulated sequence dissimilarities drifting to fixation \citep{CharlesworthMarais2005}; (2) positive selection -- e.g., of a beneficial chromsomal rearrangement supressing recombination \citep{Haldane1957} or due to breakpoint location effects \citep{CorbettDetig2016}; and (3) meiotic drive -- e.g., establishment of a meiotic drive element in tight linkage with a sex-determining factor \citep{UbedaPatten2010}. Compared to sexual antagonism these alternative hypotheses are theoretically and empirically underdeveloped (reviewed in \citealt{Ironside2010, Ponnikas2018}). If unique genomic signatures could be ascribed to each process empiricists could descriminate between different models of recombination suppression using genome sequence data. 

One potentially informative signature to differentiate between different drivers of recombination suppression is the length of 'evolutionary strata' (discrete sex-linked regions with different levels of sequence differentiation). Evolutionary strata can form when the non-recombining sex-determining region (SDR) is expanded by fixation of inversions inhibiting crossovers between the X and Y (Z and W) chromosomes (or other large-effect recombination modifiers). They also appear to be relatively common: fixation of multiple inversions has generated evolutionary strata on both ancient heteromorphic and younger homomorphic sex chromosomes in both plants and animals \citep{LahnPage1999,Handley2004, Wang2012}, and are becoming increasingly feasible to identify from long-read genome sequence data \citep{WellenreutherBernatchez2018}. Importantly, the length of new inversions is thought to influence both the form and strength of selection they experience, and therefore their fixation probabilty \citep{vanValenLevins1968, KrimbasPowell1992}. The size of fixed inversions that expand the SDR may therefore shed light on the evolutionary processes underlying recombination suppression between sex chromosomes. 

Linking inversion size with fixation probability is difficult, however, particularly for inversions expanding the SDR. The successful establishment of new inversions depends upon the balance of opposing size-dependent processes: larger inversions are more likley to capture beneficial mutations or combinations of coadapted alleles, but also deleterious mutations, which could outweigh any beneficial effects (\citealt{Nei1967,vanValenLevins1968, Santos1986, ChengKirkpatrick2019}). Recently, \citet{ConnallonOlito2020} extended this theoretical framework to address various selection scenarios for autosomal inversions. The situation is more complicated for still-recombining sex chromosomes. For example, partial linkage between sexually antagonistic loci and the SDR builds stronger associations between male-beneficial alleles and the Y chromosome, but also reduces the benefit of suppressing recombination further \citep{Nei1969,Otto2019}. Another obvious complication is that a new inversion must both span the SDR and subsequently fix in the population in order for it to expand the non-recombining region and establish a new evolutionary stratum. 

Here, we extend the theoretical framework developed by \citet{vanValenLevins1968, Santos1986}, and \citet{ConnallonOlito2020} to determine how the size of a chromosomal inversion suppressing recombination between sex-chromosomes affect its fixation probability. Simply put, we ask: does the length of chromosomal inversions creating evolutionary strata reflect the evolutionary processes driving their fixation? We examine three main evolutionary scenarios: ({\itshape i}) genetic drift of neutral inversions, ({\itshape ii}) unconditionally beneficial inversions (e.g., due to breakpoint effects \citealt{CorbettDetig2016}), and ({\itshape iii}) indirect selection (due to sexually antagonistic selection, or differing selection across life-history stages). We do not consider a recent meiotic drive hypothesis \citep{UbedaPatten2010} even though it involves the evolution of restricted recombination because it deals with the origination of genetic sex-determination rather than expansion of an existing SDR (this scenario deserves future theoretical attention). We also do not address 'sheltering hypotheses', which propose that recessive deleterious alleles can be masked as heterozygotes on the heteromorphic sex chromosome (reviewed in \citealt{Ironside2010, Ponnikas2018, Charlesworth2017}) because previous theory indicates this is probably not a major evolutionary pathway towards the initial evolution of suppressed recombination between sex chromosomes \citep{Fisher1935, Olito2020}. We derive probabilities of fixation as a function of inversion size under each idealized scenario, first ignoring, and then taking into account the effects of deleterious mutations. We then use these fixation probabilities to illustrate the expected length distribution of fixed inversions for each scenario \citep[after][]{vanValenLevins1968,Santos1986}. 

Our theoretical predictions suggest that evolutionary strata formed by the fixation of neutral inversions should be distinctly larger than those fixed under the other selection scenarios. However, except under certain conditions, it will be difficult to distinguish evolutionary strata formed by the fixation of inversions under direct or indirect selection (i.e., sexually antagonistic) from their lengths. An interesting prediction of our models is that the physical location of the SDR on the sex chromosomes is the single most influential factor determining the relation between inversion size and the probability of expanding the SDR. We conclude by briefly reviewing available data for sex-linked inversions on recombining sex chromosomes, discussing how our predictions might be used to help distinguish between different processes potentially driving the evolution of suppressed recombination between sex chromosomes. We propose a suite of new questions about how the genomic location of the ancestral SDR potentially affects the process of recombination suppression between sex chromosomes.



%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Models and Results} \label{sec:Models}
%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Key Assumptions} \label{sec:assumptions}
We make several important simplifying assumptions in our models. First, sex is determined genetically, with a dominant male-determining factor (i.e., an X-Y system with heterozygous males). Our results are equally applicable to female heterogametic Z-W systems if male- and female-specific parameters are reversed. Second, the gene(s) involved in sex determination are located in a sufficiently small non-recombining SDR that they can effectively be treated as a single locus. Hence, our models are most applicable to the early stages of recombination suppression, when the SDR is still small relative to the chromosome arm on which it resides and the length of inversions expanding it. Outside of the SDR, in the pseudoautosomal region (PAR), the sex chromosomes still recombine at rate $r$ per meiosis. For ease of comparison in our models, we further distinguish two regions within the PAR based on the mode of inheritance and 'behavior' of genes located therein: ({\itshape i}) the sex-linked PAR ({\itshape sl}-PAR) where $0 \leq r < 1/2$; and ({\itshape ii}) the autosomal PAR ({\itshape a}-PAR) PAR where $r = 1/2$ (see figure \ref{fig:diagramFig}A). Third, we assume that inversions are equally likely to occur at any point along the chromosome arm on which the SDR resides. Fourth, we assume new inversions occur rarely enough that all inverted chromosomes segregating in a population are descendent copies of a single inversion mutation. The evolutionary fate of a new inversion is therefore effectively independent of any others (i.e., we assume weak mutation; \citealt{Gillespie1991}). Fifth, recombination is completely suppressed between heterokaryotypes, although in reality genetic exchange may rarely occur via double crossovers or gene conversion \citep{KrimbasPowell1992, KorunesNoor2019}. Finally, we assume that the timescale for fixation of a new inversion is much shorter than that for the evolution of genetic degeneration and dosage compensation in the chromosomal region spanned by the inversion. This assumption is justified by the relative rates of fixation for beneficial mutations compared to that for multiple 'clicks' of Muller's ratchet or the fixation of weakly deleterious mutations due to background selection (see \citealt{Charlesworth2000, Bachtrog2008}).

We focus on the evoutionary fate of inversions spanning the SDR on a Y chromosome. As we outline below, inversions spanning the SDR on an X chromosome may also suppress recombination if they go to fixation in a population, but inversions on the Y are more likely to do so because they have a smaller effective population size than X-linked inversions ($N_Y < N_X$), experience selection exclusively in males, and are less likely to be maintained as balanced polymorphisms. We therefore highlight only essential differences between model predictions for inversions on the Y and X chromosomes in each evolutionary scenario. Full details for each model are provided in the Supporting Information, and simulation code is available at \url{https://github.com/colin-olito/inversionSize-ProtoSexChrom}.


 \begin{figure}[htbp]
 \centering
 \includegraphics[width=0.95\linewidth]{./SchematicFig-cropped}
 \caption{\textbf{(A)} Simplified diagram of recombining sex chromosomes in the models illustrating the three main chromosomal regions with distinct evolutionary dynamics: ({\itshape i}) the non-recombining sex-determining region (SDR; orange and purple bars), containing the sex-determining gene(s); ({\itshape ii}) the autosomal-PAR, or {\itshape a}-PAR, region, in which there is free recombination between the sex chromosomes ($r = 0.5$; white). Genes located in the a-PAR are physically sex-linked, yet exhibit evolutionary dynamics that are identical to autosomal genes because they recombine freely; and ({\itshape iii}) The sex-linked pseudo-autosomal region ({\itshape sl}-PAR), which is physically adjacent to the SDR, and in which the recombination rate between sex chromosomes is $0 \leq r < 0.5$ (indicated by blue shading). Due to partial linkage with the SDR, genes contained within this region exhibit evolutionary dynamics that are distinct from the other two regions, particularly those with sexually antagonistic effects (reviewed in \citealt{Otto2011}). \textbf{(B)} Illustration of new chromosomal inversions capturing the SDR and a single SA locus (see fitness expressions in Table \ref{tab:fitness}) on the Y chromosome highlighting several key features of the theoretical models, with reference to the fixation probability provided in the main text. From top to bottom, the diagrams illustrate: ({\itshape i}) new inversions capturing a deleterious mutation will not spread, and this is more likely for larger inversions; ({\itshape ii}) a mutation-free inversion on the proto-Y capturing a female-beneficial allele will not spread; ({\itshape iii} -- {\itshape iv}) a mutation-free inversion on the proto-Y capturing a male-beneficial allele can spread, and will have a fixation probability equal to Eq(\ref{eq:SApFix2LocUnlinked}) if the SA locus is located in the {\itshape a}-PAR, and Eq(\ref{eq:SAsIpFix2LocLinked}) if it is located in the {\itshape sl}-PAR. Note that inversions completely suppress recombination between the sex chromosomes ($r = 0$ inside 'new inversion' brackets). See supplementary figure S1 in Appendix D of the Online Supplementary Material for illustrative figures for other scenarios explored in this paper.}
 \label{fig:diagramFig}
 \end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Linking selection to fixation probabilities}

Following \citet{vanValenLevins1968}, \citet{Santos1986}, and \citet{ConnallonOlito2020}, we define the length of an inversion, $x$, as the proportion of the chromosome arm spanned by the inversion ($0 < x < 1$). 
We use this scale of inversion length because it is convenient, intuitive, and has been the conventional scale for reporting inversion length data in the past (e.g., Drosophila polytene chromosome data; \citealt{KrimbasPowell1992}). Conversion to other scales of inversion length, such as the number of bases or loci, is trivial (e.g., an inversion of length $x$ on a chromosome arm of $L$ nucleotides is $Lx$ nucleotides long), so our results can be easily applied to data using other length scales. Note, this scale is applicable only to paracentric inversions (those not spanning the centromere), which appear to be more common than pericentric inversions \citep{WellenreutherBernatchez2018}.

New inversions of different lengths will vary systematically in the average number of mutations they capture when they first arise. The fixation probability of an inversion of length $x$ will depend both upon the selection scenario (i.e., scenarios ({\itshape i}) -- ({\itshape iii}) above), and the number of deleterious alleles that it carries when it first arises in the population (represented by $k$, where $0 \leq k$). We assume that deleterious mutations segregate independently at different loci, and are at mutation-selection balance prior to the origin of a given inversion. 

Following previous models of inversion evolution (\citealt{Nei1967, Santos1986, Connallon2018}; see also \citealt{OrrKim1998}), we assume that new inversions are unlikely to successfully establish unless they are initially free of deleterious mutations. Moreover, for a new inversion to expand the non-recombing SDR region, it must span the ancestral SDR. Specifically, the ancestral SDR must fall between the two breakpoints of the new inversion. We can therefore express the overall fixation probability of an inversion of length $x$ as

\begin{equation}\label{eq:generalPrFix}
	\Pr(\text{fix} \mid x) = \Pr(\text{fix} \mid x, k=0) \cdot \Pr(k = 0 \mid x) \cdot \Pr(\text{SDR} \mid x),
\end{equation}

\noindent where $\Pr(k = 0 \mid x) = \exp \big[\frac{-U_d x}{s_d} \big]$ is the probability that the inversion is initially free of deleterious mutations \cite[e.g.][]{Nei1967,OrrKim1998}, $\Pr(\text{fix} \mid x, k=0)$ is the probability that the inversion fixes in the population given that it is initially mutation-free, $\Pr(\text{SDR} \mid x)$ is the probability that the inversion spans the ancestral SDR, $s_d$ is the heterozygous fitness effect of each deleterious allele an individual inherits, and $U_d$ is the deleterious mutation rate for the chromosome arm on which the SDR resides. The overall effect of deleterious mutations (i.e., the terms $\Pr(\text{fix} \mid x, k=0)$ and $\Pr(k = 0 \mid x)$) is time-dependent. Deleterious mutation-free inversions will initially be favoured relative to wild-type chromosomes, which will, on average, carry some deleterious alleles \citep{Nei1967,OhtaKojima1968, KimuraOhta1970}. However, this selective advantage will decay over time, eventually equalizing the relative fitnesses of wild-type and inversion-bearing Y chromosomes, as loci captured by the inversion approach equilibrium under mutation-selection balance \citep{Nei1967}. 

To illustrate the link between selection and the fixation probability, we first present results that condition on the inversions spanning the SDR (i.e., we temporarily assume $\Pr(\text{SDR} \mid x) = 1$). For each scenario, we derive simple expressions for $\Pr(\text{fix} \mid x)$ in the absence of deleterious mutational variation (i.e., setting $U_d = 0$). We then use a time-dependent branching process approximation to derive an expression for the fixation probability $\Pr(\text{fix} \mid x)$, which takes into account the effects of segregating deleterious mutations (i.e., $\Pr(\text{fix} \mid x, k=0)$ and $\Pr(k=0 \mid x)$). Finally, we relax the assumption that new inversions span the SDR by defining simple expressions for $\Pr(\text{SDR} \mid x)$, and then illustrate the interaction between inversion size and the location of the SDR on the fixation probability of inversions expanding the SDR.


%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Wright-Fisher Simulations}

To validate our analytic results, we ran complementary stochastic Wright-Fisher simulations in R \citep{RSoftware}. In each replicate simulation, a single-copy deleterious mutation-free inversion was introduced into a population with $N$ individuals initially at deterministic mutation-selection equilibrium. In the absence of epistasis and linkage disequilibrium between deleterious mutations (as we have assumed throughout) the average fitness of wild type chromosomes is $e^{-2 U_d}$, the standard multilocus deleterious mutation load \citep{Haldane1937,AgrawalWhitlock2012}. We used deterministic allele frequency recursions to predict the per-generation change in frequency of the inversion, with time-dependent selection modeled after \citet{Nei1967}. Realized frequencies in each generation were calculated by multinomial sampling using the predicted deterministic genotype frequencies to determine the probability of sampling a given genotype. Simulation computer code is provided in the Supplementary Material, and is freely available at  \url{https://github.com/colin-olito/inversionSize-ProtoSexChrom}.

%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Neutral inversions}

The fixation probability of neutral inversions expanding the SDR on recombining sex chromsomes is very similar to that of autosomal inversions \citep{ConnallonOlito2020}, but must take into account the appropriate effective population size. For an inversion spanning the SDR on the Y chromosome in a population with an equal sex ratio, the effective population size is $N_Y = N_m = N/2$, where $N_m$ is the number of breeding males in the population and $N$ is the total breeding population size. In the absence of deleterious mutations the fixation probability for a neutral inversion is equal to the initial frequency of the inversion: $\Pr(\text{fix}) = 1/N_Y = 2/N$ for a single copy inversion mutation \citep{Kimura1962, CrowKimura1970}. Under the same assumptions, inversions spanning the SDR on the X chromosome will have an effective population size of $N_X = 3 N_f/2 = 3 N/4 $, and $\Pr(\text{fix}) = 1/N_X = 4/3N$. 

Under deleterious mutation pressure, the evolutionary fate of neutral inversions is analogous to beneficial alleles under time-dependent selection. Unfortunately, there is no simple analytic solution for the fixation probability under this scenario (\citealt{OhtaKojima1968, KimuraOhta1970, UeckerHermisson2011, Waxman2011}). However, it is possible to approximate the fixation probability for large populations under weak selection \citep{ConnallonOlito2020}. In large populations ($0 < N_Y^{-1},\, N_X^{-1} \ll 1$) an initially deleterious mutation-free inversion will have an initial fitness advantage over non-inverted chromosomes, and will increase in frequency pseudo-deterministically until new deleterious mutations arise on descendent copies of the original inversion and reach equilibrium under mutation-selection balance. At this point, the inversion and wild-type karyotype will be equally fit and the inversion will subsequently evolve neutrally. The approximate fixation probability for an initially mutation-free inversion spanning the SDR is

\begin{subequations}\label{eq:NeutralPfix}
	\begin{align*}
		\Pr(\text{fix} \mid x, k = 0) &\approx N^{-1}_Y \, \exp \bigg[ U_d x \bigg( \frac{1}{1 - e^{-s_d} } - \frac{2}{s_d} \bigg) \bigg] e^{\frac{-U_d x}{s_d}} \approx N^{-1}_Y \numberthis,
%									  &\approx N^{-1}_Y \numberthis
	\end{align*}
	\text{when the inversion is on the Y chromosome, and}
	\begin{align*}
		\Pr(\text{fix} \mid x, k = 0)&\approx N^{-1}_X \, e^{\frac{U_d x}{s_d}} e^{\frac{-U_d x}{s_d}} = N^{-1}_X \numberthis, 
	\end{align*}
\end{subequations}

\noindent when the inversion is on the X chromosome (see Appendix A). Eq(\ref{eq:NeutralPfix}a) and Eq(\ref{eq:NeutralPfix}b) reduce to the same form as the autosomal case (see \citealt{Nei1967, ConnallonOlito2020}) due to our assumption that inversion fixation occurs on a shorter timescale than gene degeneration and loss within the inverted chromosomal segment (see \hyperref[sec:assumptions]{Assumptions}). When functional homologs exist on the X and Y chromosomes, the dynamics of deleterious mutations prior to the inversion, and the subsequent evolution of initially mutation-free neutral inversions, are nearly identical whether the inversion arises on an Y, X, or autosome \citet{ConnallonOlito2020}. This surprisingly simple result emerges from the rather complicated time-dependent dynamics because the greater fitness advantage to larger inversions of being initially free of deleterious mutations is approximately counterbalanced by the dwindling chance that they will in fact be initially free of deleterious alleles. \vspace{12pt}

\noindent {\itshape Key result: When inversions restricting recombination between sex chromosomes are selectively neutral, the overall fixation probability after taking deleterious mutations into account is equal to the initial frequency of the inversion (Fig.~\ref{fig:fixProbFig}A)}.


%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Unconditionally beneficial inversions}

The specific location of new inversion breakpoints may give inverted chromosomes a selective advantage over wild-type chromosomes. For example, an inversion may bring a protein coding sequence into closer proximity to a promoter region, thereby altering expression without disrupting other genes, or by disrupting synteny near 'sensitive sites' \citep{KrimbasPowell1992, CorbettDetig2016}. Under weak selection, and momentarily neglecting deleterious mutations, the fixation probability of a beneficial inversion can be approximated by $\Pr(\text{fix}) \approx 2 s_{I}$ \citep{Haldane1927} (i.e., there is no relation between the length of the inversion and the fixation probability). For beneficial inversions capturing the SDR on a Y chromosome, $s_I = h s_{I}^{m}$ represents the heterozygous selective advantage of the inversion in males (where $h$ is the dominance coefficient associated with the inversion). For a new inversion capturing the SDR on an X-chromosome

\begin{equation} \label{eq:benXlinkednoDel}
	s_{I} \approx \frac{h \big( s_{I}^{f} + s_{I}^{m} \big)}{2},
\end{equation}

\noindent where $s_{I}^{\text{sex}}$ is the sex-specific selection coefficient ($\text{sex} \in \{m,f\}$). Both approximations work well when $1/N \ll s_I \ll 1$.

Taking deleterious mutations into account is mathematically similar to the haploid autosomal case (see Eqs.[9 \& 10] in \citealt{ConnallonOlito2020}, and our Appendix A). A new beneficial inversion that is also free of deleterious mutations will have a temporarily heightened selective advantage. Specifically, the relative fitness of the inversion chromosome will decline over time from $(1 + s_I)e^{U_d x}$ to $(1 + s_I)$ as it accumulates deleterious mutations \citep{Nei1967}. That is, the advantages of being mutation-free and intrinsically beneficial are both present initially, but the advantage of being mutation-free decays and eventually disappears, leaving only the intrinsic advantage. The resulting fixation probability can be approximated using a time-dependent branching process \citep{PeischlKirkpatrick2012, KirkpatrickPeischl2013}, which can be expressed in terms of a time-averaged {\itshape effective selection coefficient} for the inversion:

\begin{equation} \label{eq:benSe}
	s_{e} = s_t \sum_{t=0}^{\infty} (1 - s_I)^t = s_I \Bigg[1 + \frac{U_d x}{1 - (1-s_I)e^{-s_d}} \Bigg],
\end{equation}

\noindent where $s_I = s_{I}^{m}$ for inversions capturing the SDR on the Y chromosome, while $s_I$ is given by Eq.(\ref{eq:benXlinkednoDel}) for those on the X-chromosome. Incorporating the probability that the inversion is initially mutation free, we have

\begin{equation} \label{eq:benPrFix}
	\Pr(\text{fix} \mid x, k = 0) \approx 2 s_I \Bigg[ 1+ \frac{U_d x}{1 - (1-s_I)e^{-s_d}} \Bigg] e^{\frac{-U_d x}{s_d}},
\end{equation}

\noindent and $s_I$ is defined as above for Y- and X-linked inversions respectively. The overall effect of deleterious mutations is to make the fixation probabilty decline with inversion length, with a maximum of $\approx 2 s_I$ as $x$ approaches $0$ (Fig.~\ref{fig:fixProbFig}B). 
\vspace{12pt}

\noindent {\itshape Key result: When inversions spanning the SDR are intrinsically beneficial, smaller inversions are always favoured because they are less likely to capture deleterious mutations.}



 \begin{figure}[htbp]
 \centering
 \includegraphics[width=\linewidth]{./Fig1}
 \caption{Fixation probability for inversions of different lengths capturing the SDR on the Y-chromosome under: (A) Neutral inversions; (B) Unconditionally beneficial inversions; and (C) Sexually antagonistic selection. Lines show analytic approximations of $\Pr(\text{fix} \mid x)$ (Eq(\ref{eq:NeutralPfix}), Eq(\ref{eq:benPrFix}), and Eq(\ref{eq:SAPfixWithDel})a in panels A, B, and C respectively), points show results for corresponding Wright-Fisher simulations. Note that analytic approximations for all three effective populations sizes overlap in panel A. Results are shown for the following parameter values: (A) $U_d = 0.2$ and $s_d = 0.02$; (B) $s_I = 0.02$, $s_d = 0.02$; (C) $s_f = s_m = 0.05$, $U_d = 0.1$, $s_d = 0.01$, $A = 1$, $P = 0.05$. All results condition on the inversion spanning the SDR.}
 \label{fig:fixProbFig}
 \end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Indirect selection -- Sexual antagonism}\label{sec:SexAntag}

It is well established that sexually antagonistic (SA) variation can theoretically drive selection for recombination modifiers coupling selected alleles with specific sex chromosomes \cite[e.g.][]{Fisher1931,Nei1969, Charlesworth1978a, Charlesworth1980, Bull1983,Lenormand2003, Otto2019}. However, the role of pre-existing linkage disequilibrium between the SDR and SA loci in this process is complicated. The idea that SA polymorphisms initially linked to the SDR can promote the accumulation of more linked SA polymorphisms, and lead to stronger selection for recombination suppression is seductively intuitive \citep{Rice1984, Rice1996, Charlesworth2017, Otto2019}. Yet, the conditions for the spread of SA polymorphisms to multiple loci in linkage disequilibrium with the SDR are in fact quite restrictive \citep{Otto2019}. When recombination is suppressed by an inversion, the scenario is more complicated still because multiple SA loci that may or may not be initially linked with the SDR can contribute to its overall fitness effect. The ancestral recombination rate will influence both the fixation probability by altering the equilibrium frequency of female- and male-beneficial alleles at captured SA loci, and the selective advantage of reducing recombination further.

We start with a simplified scenario to begin disentangling the effects of linkage on the fixation probability of new inversions. Suppose the average number of SA loci on the sex chromosomes is equal to $A$, that they are uniformly distributed along the chromosomes, are biallelic with standard SA fitness expressions {\itshape sensu} \citet{Kidwell1977} (each allele is beneficial when expressed in one sex, but deleterious when expressed in the other; see Table \ref{tab:fitness}), and are initially at equilibrium. Under our assumption that inversion breakpoints are randomly distributed along the chromosome arm, the number of SA loci spanned by a new inversion, $n$, is a Poisson distributed random variable with mean and variance $xA$. For now, we assume that $A$ is sufficiently small to ignore the possibility that $n$ is greater than about $1$ (the approximation breaks down when $A > 1$; we consider the case with multiple SA loci below). We focus on two idealized scenarios: the SDR and SA locus ($1$) recombine freely at a rate $r = 1/2$ per meiosis (i.e., the SA locus is located in the {\itshape a}-PAR); and ($2$) the SDR and SA locus are partially linked, and recombine at a rate $0 \leq r < 1/2$ (i.e., the SA locus is located in the {\itshape sl}-PAR). 

{\bf \itshape Effect of linkage between the SDR and SA locus} -- Considering, for the moment, inversions that already span the SDR, the fixation probability for a new inversion of size $x$ that also spans a single unlinked SA locus on the Y chromosome is the product of three probabilities: ($1$) that the inversion captures the SA locus, $\Pr(n = 1) = xA e^{-xA}$; ($2$) that it captures a male-beneficial allele at the SA locus, $\Pr(\text{male~ben.}) = \hat{q}$, where $\hat{q}$ is the equilibrium frequency of the male-beneficial allele; and ($3$) that it escapes stochastic loss due to genetic drift and fixes in the population, $\Pr(\text{fix}) \approx 2 s_I$ \cite{Haldane1927}. We can approximate the expected rate of increase of a rare inversion as $s_I \approx (\lambda_I - 1)$, where $\lambda_I$ is the eigenvalue associated with invasion of the inversion genotype into a population inititally at equilibrium in a deterministic two-locus model involving the SDR and SA locus ($\lambda_I$ is also the leading eigenvalue under these conditions). When the SA locus is unlinked with the SDR ($r = 1/2$), the selection coefficient for the rare inversion is

\begin{equation}\label{eq:SApFix2LocUnlinked}
	s_I \approx s_m (1 - \hat{q}) \big( 1 - \hat{q} - h_m(1 - 2\hat{q}) \big) + O(s_{m}^{2}),
\end{equation}

\noindent where $s_m$ is the selection coefficient of the male-deleterious/female-beneficial allele in males. With additive SA fitness ($h_f = h_m = 1/2$), the fixation probability reduces to

\begin{equation}\label{eq:SApFix2LocUnlinkedYAdd}
	\Pr(\text{fix} \mid x,n=1) = s_m \hat{q}(1 - \hat{q}) xA e^{-xA}.
\end{equation}

\noindent Eq(\ref{eq:SApFix2LocUnlinkedYAdd}) is convex sigmoidal increasing function of inversion size over $0 < x \leq 1$, with a maximum at $\tilde{x} = 1/A $, implying that larger inversions are always favoured (recall that $A \leq 1 $). Intuitively, larger inversions are more likely to capture rare SA loci distributed uniformly along the chromosome arm.

How does linkage between the SDR and SA locus alter the fixation probability? We now make two additional simplifying assumptions: the SA locus falls within the {\itshape sl}-PAR, which makes up a fraction, $P$, of the total chromosome arm length, and that $P \ll x$. Hence, any inversion that spans the SDR will also span the the {\itshape sl}-PAR. Put simply, we assume that genetic linkage requires relatively tight physical linkage between the SDR and the SA locus. The probability of spanning the SA locus is now $\Pr(n = 1) = AP e^{-AP}$. Relaxing this assumption results in predictions that are intermediate with the unlinked scenario. We can approximate $s_I \approx (\lambda_I - 1)$ from the deterministic two-locus model as before, but the expression now involves the equilibrium frequency of the male-beneficial allele on Y chromosomes ($\hat{Y}$) and X chromosomes in females ($\hat{X}_f$) before the inversion occurs:

\begin{equation}\label{eq:SAsI2LocLinkedY}
	s_I \approx \frac{ s_m(1 - \hat{Y}) \big( 1 - \hat{X}_f - h_m(1 - 2\hat{X}_f) \big)} { 1 - s_m \big(1 - \hat{X}_f - \hat{Y}(1 - h_m - \hat{X}_f) + h_m \hat{X}_f(1 - 2 \hat{Y}) \big) }.
\end{equation}

\noindent When expressed in terms of the equilibrium allele frequencies on the three chromosome types, the ancestral recombination rate ($r$) drops out of Eq(\ref{eq:SAsI2LocLinkedY}). Prior linkage between the SDR and SA loci influences the strength of indirect selection for the inversion by altering the equilibrium frequencies of the male-beneficial allele on Y chromosomes, and X chromosomes in females. Interestingly, the effect of $r$ on the overall selection coefficient for the inversion can take different forms, depending on the relative strength of selection on the SA alleles in males and females Fig(\ref{fig:recombEffect}). In this way, the SA selection coefficients can influence whether inversions capturing loosely linked (e.g., located in the {\itshape a}-PAR) or tightly linked (e.g., located physically close to the SDR in the {\itshape sl}-PAR) are more strongly favoured.

Under additive SA selection ($h_f = h_m = 1/2$), the fixation probability simplifies to

\begin{equation}\label{eq:SAsIpFix2LocLinked}
	\Pr(\text{fix} \mid x,n=1,\text{{\itshape sl}-PAR}) = \frac{ 2 s_m \hat{Y} (1 - \hat{Y}) A P e^{-A P}}{ 2 - s_m (2 - \hat{X}_f - \hat{Y}) },
\end{equation}

\noindent which is independent of $x$. \vspace{12pt}

\noindent {\itshape Key result: The overall effect of physical and genetic linkage between the SDR and SA locus is to shift the fixation probability towards smaller inversions. This is because large inversions no longer have an increased probability of spanning the SA locus. In the limiting case where $P \ll x$, the fixation probability is independent of inversion size. Relaxing this assumption will weaken the linkage-induced bias towards smaller inversions. Sex-biases in selection can alter how tightly linked the SDR and SA locus must be to maximize the fixation probability.}
\vspace{12pt}

 \begin{figure}[!htbp]
 \centering
 \includegraphics[scale=0.5]{./recombEffect}
 \caption{Overall selection coefficient ($s_I$) for an inversion linking the SDR and a male-beneficial allele at a SA locus within the {\itshape sl}-PAR (as defined by Eq[\ref{eq:SAsI2LocLinkedY}]) as a function of the ancestral recombination rate between the two loci ($r$). Panel A shows $s_I$ when there is equal selection on female- and male-beneficial alleles ($s_f = s_m$) and additive SA fitness effects ($h_f = h_m = 1/2$). Panel B shows the same for female biased selection ($s_f < s_m$; recall from table \ref{tab:fitness} that SA selection coefficients represent the decrease in relative fitness of either SA allele in males and females); specifically, for the special case where $s_f$ is equal to the single-locus invasion condition for the male-beneficial allele ($s_f = s_m / (1 - s_m)$).}
 \label{fig:recombEffect}
 \end{figure}


{\bf \itshape Effect of deleterious mutations} -- Once an inversion capturing the SDR and a male-beneficial allele at the SA locus successfully establishes, it will behave much like an unconditionally beneficial inversion, and the effects of deleterious mutations can be taken into account as in Eq(\ref{eq:benPrFix}). Under weak selection and additive SA fitness, the overall fixation probability for an inversion spanning the SDR and an SA locus falling within the {\itshape a}-PAR or {\itshape sl}-PAR will be 


\begin{subequations}\label{eq:SAPfixWithDel}
	\begin{align*}
	\Pr (\text{fix} \mid x,\,a\text{-PAR},\, k=0) &\approx 2 s_I x A e^{-xA} \Bigg[1 + \frac{U_d x}{1 - \big(1 - s_I \big) e^{-s_d}} \Bigg]e^{\frac{-Ud x}{s_d}} \numberthis
	\end{align*}
	\text{and}
	\begin{align*}
	\Pr (\text{fix} \mid x,\,sl\text{-PAR}) &\approx 2 s_I A P e^{-A P} \Bigg[1 + \frac{U_d x}{1 - (1 - s_I) e^{-s_d}} \Bigg]e^{\frac{-Ud x}{s_d}} \numberthis
	\end{align*}
\end{subequations}

\noindent respectively. Intermediately sized inversions have the greatest fixation probability when the SA locus is initially unlinked with the SDR, but smaller inversions are always favoured when the SA locus falls within the {\itshape sl}-PAR (figure \ref{fig:fixProbFig}C). \vspace{12pt}

\noindent {\itshape Key result: When the SA locus is initially unlinked with the SDR, intermediately sized inversions have the highest fixation probability because they balance the countervailing effects of inversion size on the likelihood of successfully capturing the SDR and SA loci (larger is better), and minimizing the chance of capturing deleterious mutations (smaller is better). When the SA locus falls within the sl-PAR, inversion size no longer influences the probability of capturing the SA locus, but smaller inversions still minimize the chance of capurting deleterious mutations, and so they are always favoured.} \vspace{12pt}

{\bf \itshape Multiple SA loci} -- When inversions can span more than one SA locus (i.e., when $A > 1$), the effect of prior linkage between the SDR and SA loci on the fixation probability will depend on the size of the {\itshape sl}-PAR, and satisfying analytic approximations become elusive. However, under our stated assumption that the {\itshape sl}-PAR is small ($P \ll 1$), the effect of linkage will generally weaken because SA loci distributed randomly along the chromosome arm are more likely to fall within the {\itshape a}-PAR. Analogous to previous models of inversions capturing locally adaptive alleles \citep{KirkpatrickBarton2003, Connallon2018}, a new Y-linked inversion may capture male-beneficial alleles at a subset $M$ of the $n$ SA loci it spans, where $M \sim \text{Bin}(n \mid \overline{q})$, and $\overline{q}$ is the average equilibrium frequency of male beneficial alleles across the $n$ loci. With no epistasis, weak selection, and loose linkage among SA loci, the fixation probability of new inversions is

\begin{equation}\label{eq:SApFixMultiLoc}
	\Pr(\text{fix} \mid x) = \Pr(\text{fix} \mid n) \Pr(n \mid x) \approx 2 s_I e^{-x A} \frac{(xA)^n}{n!},
\end{equation}

\noindent where

\begin{equation}\label{eq:SASIMultiLoc}
	s_I \approx \sum_{i \in n} s_{m,i} (1 - \hat{q}_{i}) \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) - \sum_{i \in (n-M)} s_{m,i} \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) + O(s_{i,m}^2),
\end{equation}

\noindent and $0$ for $s_I < 0$. More detailed assumptions are necessary to model the possibility of linkage between the SDR and a subset of captured SA loci (e.g., a quantitative description of the recombination rate within the {\itshape sl}-PAR). However, when selection is weak and SA loci are not tightly linked with the SDR, higher-order linkage effects between SA loci within the {\itshape sl}-PAR can be ignored (\citealt{Otto2019}). In this case, the fixation probability is well approximated by substituting 

\begin{align*}\label{eq:SASIMultiLocLinked}
		s_I \approx \sum_{i \in (n - L)} s_{m,i} (1 - \hat{q}_{i}) \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) &+ \sum_{i \in L} s_{m,i}(1 - \hat{Y}) \big( 1 - \hat{X}_{f,i} - h_{m,i}(1 - 2 \hat{X}_{f,i}) \big)~- \numberthis\\
		\sum_{i \in (n-L-M)} s_{m,i} \big( 1 - \hat{q}_i - h_{m,i} (1 - 2 \hat{q}_i) \big) &+ \sum_{i \in (L-M)} \big( 1 - \hat{X}_{f,i} - h_{m,i}(1 - 2 \hat{X}_{f,i}) \big),
\end{align*}

\noindent into Eq(\ref{eq:SApFixMultiLoc}), where $L$ denotes the set of SA loci falling within the {\itshape sl}-PAR ($\text{E} [L] = AP$). With deleterious mutations, the multilocus fixation probability becomes

\begin{equation} \label{eq:SApFixMultiLocDelMut}
	\Pr(\text{fix} \mid x, k = 0) \approx 2 s_I e^{-xA} \frac{(xA)^n}{n!} \Bigg[ 1+ \frac{U_d x}{1 - (1-s_I)e^{-s_d}} \Bigg] e^{\frac{-					U_d x}{s_d}}.
\end{equation}

\noindent where $s_I$ is defined as in Eq(\ref{eq:SASIMultiLoc}) and Eq(\ref{eq:SASIMultiLocLinked}).
\vspace{12pt}

\noindent {\itshape Key result: The effect of prior linkage between the SDR and SA loci on the fixation probability of different sized inversions will generally weaken when multiple SA loci are distributed along the sex chromosomes. However, this effect will ultimately depend on the size of the sl-PAR, which is assumed to be small in our models.}
\vspace{12pt}

{\bf \itshape Inversions on the X} -- Results for inversions on X chromosomes can be derived by similar steps. However, because they are exposed to selection in both males and females, X-linked inversions can invade over a smaller fraction of parameter space than Y-linked inversions, and are generally maintained as balanced polymorphisms when they do (Figure~\ref{fig:detInvFreqSA}). 
\vspace{12pt}

\noindent {\itshape Key result: X-linked inversions may contribute to reduced recombination between sex chromosomes as segregating polymorphisms, but are far less likely to cause permanent recombination suppression than Y-linked inversion.}

%-- for example, by drifting to fixation whilst segregating at high frequencies -- but probably do so less often than Y-linked inversions.


 \begin{figure}[!htbp]
 \centering
 \includegraphics[scale=0.4]{./detEqInvFreqFig}
 \caption{Equilibrium frequency of new inversions capturing the SDR and a single sexually antagonistic locus on the Y ($\hat{Y}$, panels A and B) and the X chromosomes ($\hat{X}$, panels C and D), under loose (panels A and C) and tight (panels B and D) linkage between the two loci, and additive SA fitness effects ($h_f = h_m = 1/2$). Initial equilibrium genotypic frequencies were calculated by iterating the 2-locus deterministic recursions in the absence of an inversion. Once this initial equilibrium was reached, an (heterozygote) inversion genotype was introduced at low frequency ($10^{-6}$), and the recursions were again iterated until all genotypic frequencies remained unchanged. Note the different color scale for Y and X inversions. Recursions are presented in the Supplementary Materials.}
 \label{fig:detInvFreqSA}
 \end{figure}



%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Indirect selection -- Haploid \& diploid selection}

We have so far considered selection in the diploid phase only. However, sexually reproducing eukaryotes have alternating life-cycles with a reduced (e.g., haploid) and doubled (e.g., diploid) phase \citep{Strasburger1894, Roe1975}. Moreover, haploid selection can play an important role in maintaining genetic polymorphisms \citep{ImmlerOtto2011}, as well as facilitating sex chromosome turnovers \citep{OttoScottOsmond2018} and transitions between sex determination systems \citep{MuralidharVeller2018}. The models summarized above for sexually antagonistic selection can be easily extended to incorporate haploid selection, although this opens up many new possible selection scenarios \citep{ImmlerOtto2011, OttoScottOsmond2018}. For simplicity and brevity, we briefly consider a general model of haploid and diploid selection, and illustrate the model predictions with a single representative case of ploidally antagonistic selection. The critical difference between this model and those of SA selection above with respect to the fixation probability of inversions is that $s_I$ is now a function of fitness during both haploid and diploid phases. 

Consider the simple case of a rare inversion capturing the SDR and a single selected locus on the Y chromsome. To keep the model general, and relatively simple, we retain arbitrary fitness expressions for the haploid ($v^{f}_{1}$, $v^{f}_{2}$ for female, and $v^{m}_{1}$, $v^{m}_{2}$ for male gametes, respectively) and diploid genotypes ($w^{f}_{11}$, $w^{f}_{12}$, $w^{f}_{22}$ in females, and $w^{m}_{11}$, $w^{m}_{12}$, $w^{m}_{22}$ in males). For Y-linked inversions capturing a single selected locus, the approximate selection coefficient for the rare inversion under arbitrary linkage ($0 \leq r \leq 1/2$) is:

\begin{equation}\label{eq:sI-HapDip-2LocLinked}
	s_I \approx \frac{\big( v^{f}_{2} \hat{X}_f (v^{m}_{2} w^{m}_{22} -v^{m}_{1} w^{m}_{12} ) - v^{f}_{1} (1 - \hat{X}_f) (v^{m}_{1} w^{m}_{11} - v^{m}_{2} w^{m}_{12}) \big) (1 - \hat{Y})} {v^{f}_{1} (1 - \hat{X}_f) \big( v^{m}_{1} w^{m}_{11} (1 - \hat{Y}) + v^{m}_{2} w^{m}_{12} \hat{Y} \big) + v^{f}_{2} \hat{X}_f \big( v^{m}_{2} w^{m}_{22} \hat{Y} + v^{m}_{1} (w^{m}_{12} - w^{m}_{12} \hat{Y}) \big)},
\end{equation}

\noindent where $\hat{Y}$ and $\hat{X_f}$ represent the equilibrium frequency of the $2^{\text{nd}}$ allele at the selected locus in Y chromosomes and X chromosomes in females when the inversion originates. For a rare inversion to invade, $s_I > 0$ must be satisfied for Eq(\ref{eq:sI-HapDip-2LocLinked}), which requires that the net fitness effect of the inversion across haploid and diploid phases is male-beneficial, or there is sufficient linkage disequilibrium to offset a female-bias in selection. For example, under weak ploidally antagonistic selection with additive fitness in the diploid phase (see Table.~\ref{tab:fitness}), an inversion capturing the SDR and the $2^{\text{nd}}$ allele at the selected locus can invade when $s > 2 t + O(s^2,t^2)$.  

Calculation of the fixation probability, and the effects of deleterious mutations are the same as for the sexually antagonistic model described above, and result in qualitatively similar predictions. Selection, whether during the haploid, diploid, or both phases, influences the fixation probability of differently sized inversions similarly, and should favour small to intermediately sized inversions.

For X-linked inversions, the addition of selection during the haploid phase expands the conditions under which an inversion can be maintained as a balanced polymorphism. The overall result parallels that for sexually antagonistic selection: while X-linked inversions can contribute to reduced recombination between sex chromosomes, they are far less likely to fix and thereby form evolutionary strata than Y-linked inversions. \vspace{12pt}

\noindent {\itshape Key result: The fixation probability of different sized inversions is a similar function for selection occuring during the haploid, diploid, or both phases. Inversion length will therefore provide little insight into when during the life cycle indirect selection for suppressed recombination occurs.} 



%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Probability of expanding the SDR}\label{sec:ProbExpSDR}

So far, we have presented results that are conditioned on inversions spanning the SDR to clarify the relation between selection and inversion size for each scenario (i.e., we have assumed $\Pr(\text{SDR} \mid x) = 1$). Under this assumption, the models suggest that the length of fixed inversions expanding the SDR will reflect the selective process underlying their fixation: neutral, directly beneficial, and indirectly beneficial inversions will leave distinct footprints of different sized evolutionary strata. We now relax this assumption and examine the effects of explicitly modeling the probability that new inversions span the ancestral SDR.

Assuming, as we have throughout, that inversions are equally likely to occur at any point along the chromosome arm on which the SDR resides, the probability that a given inversion will span the SDR depends on two factors: the length of the inversion ($x$) and the location of the SDR on the chromosome arm in question (denoted $\text{SDR}_{\text{loc}}$). The total length of the chromosome arm can be subdivided into three regions: from the centromere and the SDR ($y_1$), the SDR itself ($y_2$), and from the SDR to the telomere ($y_3$), where $y_1 + y_2 + y_3 = 1$. If the SDR is small relative to the length of new inversions (as we have assumed throughout), $y_2 \approx 0$ and $y_1 + y_3 \approx 1$. From these assumptions, the probability that a new inversion of length $x$ spans the SDR is a piecewise function of $x$ which follows

\begin{equation}\label{eq:PrSpanSDR}
	\Pr(\text{SDR} \mid x) = \left\{ 
		\begin{array}{ccl} 
			x  /(1 - x) & \mbox{for} & x \leq y_1,\,y_3 \\
			y_1/(1 - x) & \mbox{for} & y_1 < x < y_3 \\ 
			y_3/(1 - x) & \mbox{for} & y_1 > x > y_3 \\ 
			1 & \mbox{for} & x > y_1,\,y_3 \numberthis			
		\end{array}\right.
\end{equation}

\noindent where $y_1 = \text{SDR}_{\text{loc}}$ and $y_3 = (1 - \text{SDR}_{\text{loc}})$. Critically, the form of Eq(\ref{eq:PrSpanSDR}) depends upon the location of the SDR. 

Taking into account the probability that a new inversion spans the SDR by substituting Eq(\ref{eq:PrSpanSDR}) into Eq(\ref{eq:generalPrFix}) has an immediate and strong effect on our model predictions. For simplicity, we examine the fixation probability of new inversions in each selection scenario under two limiting cases for $\Pr(\text{SDR} \mid x)$: ($1$) the SDR is located at the exact center of the chromosome arm ($\text{SDR}_{\text{loc}} = 1/2$), and ($2$) the SDR is located near either the centromere or telomere ($\text{SDR}_{\text{loc}} = 1/10$; results are identical if $\text{SDR}_{\text{loc}} = 9/10$). An illustration of Eq(\ref{eq:PrSpanSDR}) for these scenarios is provided in supplementary figure S2 in Appendix D of the Online Supplementary Material. Intermediate values of $\text{SDR}_{\text{loc}}$ yield predictions that fall between these extremes. 

The effect of $\Pr(\text{SDR} \mid x)$ on the relation between inversion size and the probability of expanding the SDR is most dramatic for neutral inversions (Fig.~\ref{fig:probCatchFixFig}A,D). When the SDR is located in the middle of the chromosome arm ($\text{SDR}_{\text{loc}} = 1/2$) the probability of expanding the SDR increases until $x = 1/2$, after which it plateaus at $1/N_Y$ (figure \ref{fig:probCatchFixFig}A). Intuitively, the probability that a new inversion spans the SDR increases until $x > y_1,\,y_3$, above which any inversion will necessarily span the SDR. A similar, but more exaggerated pattern favouring large inversions emerges when the SDR is located near one end of the chromosome arm (figure \ref{fig:probCatchFixFig}D). The prediction that larger inversions are always more likely to expand the SDR is unique to neutral inversions. However, when the effective population size is small, the weakened benefit for new inversion of being initially free of deleterious mutations can result in a peak fixation probability for intermediately sized inversions (Fig.~\ref{fig:probCatchFixFig}A, simulation results for $N_Y = 10^2$).

For unconditionally beneficial inversions, taking $\Pr(\text{SDR} \mid x)$ into account results in intermediately sized inversions having the highest fixation probability (Fig.~\ref{fig:probCatchFixFig}B,E). When the SDR is located closer to either the centromere or telomere, smaller inversions have the highest fixation probability, although a second peak appears for very large inversions under lower deleterious mutation rates for  (Fig.~\ref{fig:probCatchFixFig}E, grey points).

For indirectly beneficial inversions capturing a single SA locus, the relation between inversion size and fixation probability are robust to the location of the SDR (Fig.~\ref{fig:probCatchFixFig}C,F). The only qualitative difference arises when the SA locus is initially linked with the SDR, where the fixation probability now has an intermediate peak associated with slighly smaller inversions than when the SA locus is initially unlinked with the SDR. Notably, when the SDR is located in the middle of the chromosome arm, the relation between inversion size and fixation probability is very similar for beneficial inversions and those capturing an SA locus (compare Fig.~\ref{fig:probCatchFixFig}B with C,F). The two scenarios differ most when the SDR is near the end of the chromosome arm and the deleterious mutation rate is high, but otherwise it will likely be difficult to distinguish between these two selection scenarios from the length of evolutionary strata.  \vspace{12pt}

\noindent {\itshape Key result: When explicitly taking into account the probability that new inversions span the ancestral SDR, the physical location of the SDR strongly influences the resulting fixation probabilities of different length inversions. Large inversions are only favoured under the neutral scenario, while small to intermediate length inversions are favoured when inversions are either beneficial, or if they capture sexually antagonistic loci.} 



 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.53]{./PrCatchFixFig}
 \caption{Taking into account the location of the ancestral SDR, and its effect on the fixation probability of inversions of different length. Each panel shows the overall fixation probability of new inversions of length $x$, reevaluated with Eq(\ref{eq:PrSpanSDR}) substituted into Eq(\ref{eq:generalPrFix}) for each selection scenario. Panels A--C show results when the SDR is located in the exact middle of the chromosome arm ($\text{SDR}_\text{loc} = 1/2$) for neutral inversions, beneficial inversions, and inversions capturing a sexually antagonistic locus; panels D--E show the same when the SDR is located near either the centromere or telomere ($\text{SDR}_\text{loc} = 1/10$). Solid and dashed lines show the relevant analytic approximations of $\Pr(\text{fix} \mid x,\, \text{SDR})$, while points show results for Wright-Fisher simulations. Note that analytic approximations for all three effective populations sizes overlap in panel A. Results are shown for the same parameter values as in Fig(\ref{fig:fixProbFig}): (A,D) $U_d = 0.2$ and $s_d = 0.02$; (B,E) $s_I = 0.02$, $s_d = 0.02$; (C,F) $s_f = s_m = 0.05$, $U_d = 0.1$, $s_d = 0.01$, $A = 1$, $P = 0.05$.}
 \label{fig:probCatchFixFig}
 \end{figure}



%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Expected length distributions of evolutionary strata} \label{subsec:DistFixedInv}

With expressions for the fixation probability of new inversions under different evolutionary scenarios in hand, it is possible to derive the corresponding expected distributions of fixed inversion sizes. Following \citet{vanValenLevins1968, Santos1986}, and \citet{ConnallonOlito2020}, the proportion of fixed inversions of length $x$ is given by 

\begin{equation} \label{eq:generalInvSizeModel}
	g(x) = \frac{\Pr(\text{fix} \mid x) f(x)} {\int \Pr(\text{fix} \mid x) f(x)\,dx},
\end{equation}

\noindent where $f(x)$ is the probability of a new inversion of length $x$, and $\Pr(\text{fix} \mid x)$ is the fixation probability given in Eq(\ref{eq:generalPrFix}) with appropriate substitutions made for each selection scenario. $x\int \Pr(\text{fix} \mid x) f(x)\,dx$ gives the mean length of fixed inversions. Little is known about how the mutational process for new inversions shapes $f(x)$, and we therefore examine two scenarios representing plausible extremes to illustrate the spectrum of possible outcomes.

On one hand, if inversion breakpoints are distributed uniformly across the chromosome arm containing the SDR, then $f(x) = 2(1 - x)$, an extreme scenario we refer to as the "random breakpoint" model \citep{vanValenLevins1968}. On the other hand, if inversion breakpoints tend to be clustered, for example in chromosomal regions with repetitive sequences, the resulting enrichment of smaller new inversions can be modeled phenomenologically using a truncated exponential distribution:

\begin{equation} \label{eq:truncExp}
	f(x) = \frac{ \lambda e^{-\lambda x}} {1 - e^{-\lambda}},
\end{equation}

\noindent where $\lambda$ is the exponential rate parameter \citep{PevznerTesler2003, PengPevznerTesler2006, ChengKirkpatrick2019,ConnallonOlito2020}. For strongly skewed distributions (e.g., $\lambda > 10$, as we assume here), the truncation effect is negligible, and $f(x)$ is approximately equal to the numerator of Eq(\ref{eq:truncExp}). We refer to this other extreme as the "exponential model". Two key results emerge from the expected distributions of evolutionary strata length.


 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.65]{./expDistFig}
 \caption{Probability density functions for fixed inversions expanding the SDR on Y chromosomes ($g(x)$ from Eq[\ref{eq:generalInvSizeModel}]). For clarity, we show results for the model of Sexual Anatagonism with an initially unlinked SA locus ($r = 1/2$). Results are shown for the same parameter values as in Fig.~\ref{fig:fixProbFig} and Fig.~\ref{fig:probCatchFixFig}: $U_d = 0.2$, $s_d = 0.02$, and $s_I = 0.02$ for Neutral and Beneficial inversion scenarios, and $s_f = s_m = 0.05$, $U_d = 0.1$, $s_d = 0.01$, $A = 1$ for the Sex Antagonism scenario.}
 \label{fig:ExpectedDistFig}
 \end{figure}


First, the results are again strongly influenced by the location of the SDR. When the SDR is located in the center of the chromosome arm, neutral inversions are expected to give rise to a triangular distribution of evolutionary strata lengths, with a mean at $x = 1/2$ (Fig.\ref{fig:ExpectedDistFig}A). Both beneficial inversions and those capturing SA loci yield largely overlapping distributions of smaller inversions, although the distribution for sex antagonism has a distincly heavier tail under the random breakpoint model. The differences between the distributions for neutral and selected inversions becomes exaggerated when the SDR is located near one end of the chromosome arm (Fig.\ref{fig:ExpectedDistFig}C). The distribution for neutral inversions now has three distinct regions yielding a plateau shape, while those for beneficial and sex antagonistic inversions become increasingly skewed and overlapping. The unusual form of the length distributions for neutral inversions under the random breakpoint model results from the appearance of $(1 - x)$ terms in both $f(x)$ and $\Pr(\text{SDR} \mid x)$, which cancel in different ranges of $x$ depending on the location of the SDR.  

Second, the expected length distributions of evolutionary strata are sensitive to the form of $f(x)$. In contrast to the random breakpoints model, when inversion breakpoints are clustered the predicted distributions of strata length are highly overlapping for all three selection scenarios, and are practically indistinguishable when the SDR is located near the end of the chromosome arm (Fig.~\ref{fig:ExpectedDistFig}B,D). \vspace{12pt}

\noindent {\itshape Key result: Two dominant factors influence the expected length distribution of fixed inversions expanding the SDR: the location of the ancestral SDR on the chromosome arm, and the length distribution of new inversions. While different selection scenarios are expected to result in distinct distributions under a random breakpoint model, the length distributions become practically indistinguishable under an exponential model of new inversion lengths.} 



%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Discussion} \label{sec:Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%

Our models reveal two major implications for the evolution of recombination suppression between sex chromosomes. The first is that different selection scenarios should result in unique associations between inversion length and fixation probability, suggesting that the length of evolutionary strata may reflect the selective process underlying expansion of the non-recombining SDR. Specifically, our models predict that evolutionary strata formed by the fixation of neutral inversions should be significantly larger, on average, than those formed by directly or indirectly beneficial inversions. However, the most popular hypothesis for the evolution of suppressed recombination, sexually antagonistic selection, will likely be indistinguishable from scenarios involving either direct or indirect selection based on the size of evolutionary strata.

One obvious application of our findings is to compare the lengths of early evolutionary strata (i.e., those occurring when the ancestral SDR is still quite small) identified from DNA sequence data with the expected length distributions we have derived here. The ongoing development of whole-genome sequencing technology and analyses is making the identification of genome structural variation, including fixed inversions and evolutionary strata on sex chromosomes, increasingly feasible for non-model organisms \citep[reviewed in ][]{Muyle2017, Charlesworth2018,PandayAzad2016}. Sex-linked regions have been identified in a variety of unrelated species with still- or recently-recombining sex chromosomes, including Papaya (Caricaceae) and two closely related species \citep{Wang2012, Lovene2015}, {\itshape Mercurialis annua} (\citealt{VeltsosPannell2019}), the genus {\itshape Populus}  (Salicaceae) \citep[reviewed in][]{HobzaEtAl2018}, and several fishes including African cichlids \citep{GammerdingerKocher2018} and yellowtail \citep{KoyamaEtAl2015}. Moreover, inversions appear to be involved in the evolution of sex-linked genome regions in several of these species (but see recent work on {\itshape Salix}; \citealt{AlmeidaMank2019}). 

Our findings suggest inversion lengths may inform how, or whether, selection affected the fixation of inversions (or other recombination modifiers) in  systems like these, however such comparisons will never be definitively diagnostic. Clearly it is not possible to observe a distribution of evolutionary strata lengths for single species. Moreover, subsequent sequence evolution within a newly expanded SDR, including deletions, duplications, and the accumulation of transposable elements will distort comparisons. Nevertheless, the observed length of relatively undegraded evolutionary strata should often provide different levels of support for neutral vs.~selection scenarios: large evolutionary strata are more consistent with the fixation of a neutral inversion (or other linked large-effect recombination modifier), while small strata (possibly including gene-by-gene recombination suppression or gradual expansion of the SDR; e.g., \citealt{BergeroQiuCharlesworth2013, QiuBergeroCharlesworth2015}), is more consistent with scenarios involving selection. Given recent results indicating the role of genomewide heterochiasmy in restricting recombination between sex chromosomes, coupled with within-lineage variation in the extent of recombination suppression in {\itshape Poecillid} fish and some frogs \citep[e.g.,][]{Wright2017,BergeroCharlesworth2019,DaroltiWrightMank2019,FurmanEvans2018}, it would be very interesting to critically examine parallels between heterochiasmy and the fixation of large inversions expanding the SDR theoretically.

The second major implication of our models is that physical characteristics of recombining sex chromosomes, including the location of the ancestral SDR, can have as strong or an even stronger effect than selection on the evolution of suppressed recombination. This is a crucial difference between the process of recombination suppression between sex chromosomes, and the fixation of inversions on autosomes, for which the interaction between deleterious genetic variation and the form of natural selection is critical \citep{ConnallonOlito2020}. The effect of SDR location on the likelihood of forming different sized evolutionary strata emerges directly from the geometry of a functionally two dimensional chromosome arm and our assumption that inversion breakpoints are distributed uniformly along it. These are clearly major simplifying assumptions, but the resulting predictions suggest that considering physical characteristics of recombining sex chromosomes could shed light on several outstanding questions (reviewed in \citealt{Charlesworth2016, Charlesworth2017}), such as why large sex-linked regions or heteromorphic sex chromosomes have evolved in some lineages and not others, and how many recombination suppression events are involved and why this varies among lineages? Overall, our models suggest that considering the physical processes involved in recombination suppression may offer additional insights into why and how restricted recombination does or does not evolve in different lineages than seeking evidence of past bouts of sexually antagonistic selection.

Although we have modelled the effect of SDR location explicitly, other physical characteristics of recombining sex chromosomes not included in our models also influence the process of recombination suppression. For example, it is well known that the rate of recombination at different locations along chromosomes -- the 'recombination landscape' -- can be highly variable within and among species, and that marked differences often exist between males and females \citep[reviewed in ][]{SinghalEtAl2015,SardellKirkpatrick2020}. It has also been suggested that new sex determining genes may be more likely to recruit to genome regions with already low recombination rates \citep{Charlesworth1978a,vanDoornKirkpatrick2007, vanDoornKirkpatrick2010, OttoScottOsmond2018, Charlesworth2015, Olito2019}. For example, this appears to be the case for {\itshape Rumex hastatulus} and Papaya relatives \citep{RifkinBarrettWright2020,Lovene2015}. Moreover, classical theory predicts that low recombination rates are favourable for the maintenance of sexually antagonistic polymorphism \citep{Charlesworth1978a,Olito2017,Olito2019,Charlesworth2018}. If these regions of low recombination are more likely to occur at certain locations along the chromosome arm, the possible locations of the SDR may be constrained, thereby influencing whether further recombination suppression will involve small vs.~large evolutionary strata. Given that recombination is often lower in genome regions surrounding the centromere \citep[e.g.,][]{MahtaniWillard1998,SardellKirkpatrick2020}, it would be interesting to examine how our predictions, which are limited to paracentric inversions, might change when inversions suppressing recombination are pericentric.

There is perhaps a parallel between the evolution of divergence between sex chromosomes parallels the genomics of speciation. Early genomic analysis of hybrid species pairs suggested the existence of "genomic islands of speciation" -- restricted regions with high genetic differentiation between species -- which were speculated to contribute to adaptation and reproductive isolation \citep[e.g.,][]{Ellegren2012}. Although apparent genomic islands of divergence have been identified \citep{TavaresEtAl2018}, a number of early analyses were later shown to provide inadequate control for confounding factors such as variable levels of genetic diversity across the genome or variation in recombination rate \citep{NoorBennett2009, WolfEllegren2017}. Consequently, regions of high divergence were often erroneously ascribed to selection rather than neutral or structural factors. Both this example and the results of our models suggest that caution is warranted when inferring causation with respect to genomic differentiation, and that selective explanations, although intuitively appealing, may not always be the most parsimonious.

Finally, our results show that the shape of the distribution of new inversion lengths (e.g., random breakpoint vs.~exponential) can weaken or exaggerate differences between selection scenarios in the expected length distributions of evolutionary strata. Although little is known about the distribution of new inversion lengths (limited data from {\itshape Drosophila} mutagenesis experiments are roughly consistent with a random breakpoint model; \citealt{KrimbasPowell1992}), it will be determined, at least in part, by other physical aspects of proto sex chromosome structure, such as the density and physical location of gene duplications, chromatin structure, transposable elements (TEs) and other repetitive sequences, which create hotspots for inversion breakpoints and DNA replication errors \citep[e.g.,][]{Charlesworth1994, PevznerTesler2003, PengPevznerTesler2006, LeeBatzer2008}. Indeed, the spatial distribution of these structural features of sex chromosomes will contribute jointly to determine the whether and how expanded non-recombining regions on sex chromosomes evolve. The interaction between physical and seletive processes driving the evolution of recombination suppression between sex chromosomes offers a variety of future directions for theoretical and empirical research.


%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Data Availability}
Data sharing is not applicable because no datasets were generated or analyzed by the current study. Full derivations and details for each model and all key analytic results are provided in the Supporting Information, and simulation code is available at \url{https://github.com/colin-olito/inversionSize-ProtoSexChrom}.

%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Acknowledgements}
This research was supported by a Wenner-Gren Postdoctoral Fellowship to C.O., and ERC-StG-2015-678148 to J.K.A. This manuscript benefitted greatly from many detailed discussions and constructive feedback from T.~Connallon, C.Y. Jordan, C.~Venables, H.~Papoli, the SexGen group at Lund University, the editor, and two anonymous reviewers. C.O. conceived the study, developed the models, performed the analyses. Both C.O. and J.K.A. wrote the manuscript.



%%%%%%%%%%%%%%%%%%%%%%%%
%\subsection*{Supplementary Materials}
%Requests for supplementary material and correspondence can be directed to C.O. (\url{colin.olito@gmail.com}).


%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%
\bibliography{bibliography-inversionSize-ProtoSexChrom}

\newpage


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%  Tables 

\begin{table}[htbp]
\caption{\bf Definition of terms and parameters.}
\begin{tabu}to \linewidth{l X}
\toprule
\multicolumn{2}{l}{{\itshape Key terms}} \\
\midrule
$x$ & Inversion size, expressed as fraction of the chromosome that it spans ($0 < x < 1$). \\
$N$, $N_f$, $N_m$ & Census, and breeding male and female population sizes, respectively. \\
$N_Y$, $N_X$, & Effective population size for Y- and X-linked genes, respectively. \\
$s_I$ & Overall fitness effect for a new inversion ($0 < s \ll 1$). \\
$s_{i}$ & Sex specific selection coefficient for selected loci in diploid phase ($i \in \{s,f\}$). \\
$h_{i}$ & Sex specific dominance coefficient for selected loci in diploid phase ($i \in \{s,f\}$). \\
$t_{i}$ & Sex specific selection coefficient selected loci in haploid phase ($i \in \{s,f\}$). \\
$n$ & Number of selected loci captured by new inversion \\
$k$ & Number of deleterious mutations captured by new inversion \\
$A$ & Expected number of sexually antagonistic loci on sex chromosomes \\
$P$ & Length of {\itshape sl}-PAR, expressed as fraction of total chromosome length \\
$\lambda$ & Rate parameter for the exponential model of new inversion lengths; $\lambda^{-1}$ is the average length of a new inversion under the exponential model. \\
$U_d$ & Chromosome-wide deleterious mutation rate ($0 < U_d$) \\
$s_{d}$ & Selection coefficient for deleterious mutations ($0 < s_d \ll 1$). \\
$h_{d}$ & Dominance coefficient for deleterious mutations ($0 \leq h_d \leq 1$; often approximated as $h_d \approx 1/2$). \\
\addlinespace
\multicolumn{2}{l}{{\itshape Deterministic 2-locus models}} \\
\midrule
$w^{f}_{ii}$, $w^{m}_{ii}$ & diploid fitness terms for each genotype in females and males \\
$v^{f}_{i}$, $v^{m}_{i}$ & haploid fitness terms for each genotype in female and male gametes \\
$r$ & Recombination rate between SDR and selected locus \\
$\lambda_I$ & Leading eigenvalue associated with invasion of rare inversion genotype \\
$\hat{q}$ & Equilibrium frequency of male-beneficial sexually antagonistic allele (when $r = 1/2$) \\
$X_f$, $X_m$, $Y$ & Equilibrium frequency of male-beneficial sexually antagonistic allele on X chromosomes in males and females, and Y chromosomes, respectively. \\
\addlinespace
\multicolumn{2}{l}{{\itshape Probability inversion spans SDR}} \\
\midrule
$\text{SDR}_{\text{loc}}$ & Location of the SDR on the chromosome arm, expressed as a proportion of the distance between the centromere and telomere ($0 \leq \text{SDR}_{\text{loc}} \leq 1$). \\
$y_1$, $y_2$, $y_3$  & Proportion of total length of the chromosome arm that falls between the centromere and SDR, between the SDR and the telomere, and spanned by the ancestral SDR, respectively. \\
\addlinespace
\bottomrule
\end{tabu}
\label{tab:Parameters}\\
\end{table}
\newpage{}


 \begin{table}[htbp]
 \centering
 \caption{\bf Fitness expressions for models of Indirect Selection.}
 \begin{tabu}to 10.5cm {X[1,l] X[2,l] X[2,l] X[2,l]}
 \toprule
 \multicolumn{4}{l}{{\textit{Sexually antagonistic selection}}} \\
 \midrule
	Females: & $w^{f}_{11} = 1$ & $w^{f}_{12} = 1 - h_f s_f$ & $w^{f}_{22} = 1 - s_f$ \\
	Males: & $w^{m}_{11} = 1 - s_m$ & $w^{m}_{12} = 1 - h_m s_m$ & $w^{m}_{22} = 1$ \\
 \addlinespace
 \multicolumn{3}{l}{{\textit{Ploidally-antagonistic selection}}} \\
 \midrule
 	Diploid: & $w^{\text{sex}}_{11} = 1 - s$ & $w^{\text{sex}}_{12} = 1 - s/2$ & $w^{\text{sex}}_{22} = 1$ \\
%	Females: & $w^{f}_{11} = 1 - s$ & $w^{f}_{12} = 1 - s/2$ & $w^{f}_{22} = 1$ \\
%	Males:   & $w^{m}_{11} = 1 - s$ & $w^{m}_{12} = 1 - s/2$ & $w^{m}_{22} = 1$ \\
 \addlinespace
 \end{tabu}
 \begin{tabu}to 10.5cm {X[1,l] X[2,l] X[2,c] X[2,l]}
 	Haploid: & $v^{\text{sex}}_{1} = 1$ & -- & $v^{\text{sex}}_{2} = 1 - t$ \\
%	Female gametes: & $v^{f}_{1} = 1$ & $v^{f}_{2} = 1 - t$ \\
%	Male gametes:   & $v^{m}_{1} = 1$ & $v^{m}_{2} = 1$ \\
 \bottomrule
 \end{tabu}
 \begin{tabu}to 10.5cm {X[1,l]}
 {\footnotesize Where $\text{sex} \in \{m,f\}$.}
 \end{tabu}
 \label{tab:fitness}\\
 \end{table}
 \newpage{}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%  Figures 




\end{document}
