%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Preamble
\documentclass{article}[12pt]

%Dependencies
\usepackage[left]{lineno}
\usepackage{titlesec}
\usepackage[table]{xcolor}

\newcommand\hl[1]{%
  \bgroup
  \hskip0pt\color{blue!80!black}%
  #1%
  \egroup
}
\usepackage{ogonek}
\usepackage{float}
%\usepackage{charter}
\usepackage{amsmath}
\usepackage{old-arrows}
\usepackage{enumitem}
\usepackage{wasysym}
\usepackage{nameref}
\usepackage{booktabs}
\usepackage{tabu}

% Other Packages
\usepackage{txfonts}
\RequirePackage{fullpage}
\linespread{1.7}
\RequirePackage[colorlinks=true, allcolors=black]{hyperref}
\RequirePackage[english]{babel}
\RequirePackage{amsmath,amsfonts,amssymb}
\RequirePackage[sc]{mathpazo}
\RequirePackage[T1]{fontenc}
\RequirePackage{url}
\RequirePackage{nameref}

% Bibliography
\usepackage{natbib} \bibpunct{(}{)}{;}{author-year}{}{,}
\bibliographystyle{evolution}
\addto{\captionsenglish}{\renewcommand{\refname}{Literature Cited}}
\setlength{\bibsep}{0.0pt}
% Graphics package
\usepackage{graphicx}
\graphicspath{{../figures/}.pdf}


% Graphics package
\usepackage{graphicx}
\graphicspath{{../output/figures/}.pdf}
\usepackage[font={small}]{caption}

% New commands: fonts
%\newcommand{\code}{\fontfamily{pcr}\selectfont}
%\newcommand*\chem[1]{\ensuremath{\mathrm{#1}}}
\newcommand\numberthis{\addtocounter{equation}{1}\tag{\theequation}}
%\titleformat{\subsubsection}[runin]{\bfseries\itshape}{\thesubsubsection.}{0.5em}{}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Title Page

\title{The evolution of suppressed recombination between sex chromosomes by chromosomal inversions}
%\author{Colin Olito$^{\ast}$ \& Jessica K.~Abbott}
\date{}

\begin{document}
\maketitle

%\noindent{} Department of Biology, Section for Evolutionary Ecology, Lund University, Lund 223 62, Sweden.

%\noindent{} $^{\ast}$ Corresponding author e-mail: \url{colin.olito@gmail.com}

\bigskip

\noindent \textit{Manuscript elements}: Figure~1, Figure~2, Figure~3, Figure~4, Figure~5, Table~1, Table~2;

\noindent {\itshape Online Supplementary Material}: Appendix A -- Neutural inversions; Appendix B -- Sexual Antagonism; Appendix C -- Haploid \& Diploid Selection; Appendix D -- Supplementary Figures; Appendix E -- Mathematica code (.nb file \& PDF) to reproduce analytic results

\bigskip
\noindent{} \textit{Running Title}: Inversions on sex chromosomes

\bigskip

\noindent{} \textit{Keywords}: Sex chromosomes; Recombination; Chromosomal inversions; Sexual antagonism; Evolutionary strata.

\bigskip

\noindent{} \textit{Manuscript type}: Investigation

\bigskip

\begin{center} 
	Submitted to {\itshape Evolution} \date{\today}
\end{center}

% Set line number options
\linenumbers
\modulolinenumbers[1]
\renewcommand\linenumberfont{\normalfont\small}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Main Text

\newpage{}
\section*{Abstract}% ($\leq 250$ words)}

\noindent The idea that sex-differences in selection drive the evolution of suppressed recombination between sex chromosomes is well-developed in population genetics. Yet, despite a now classic body of theory, empirical evidence that sexual antagonism drives the evolution of recombination suppression remains meagre and alternative hypotheses underdeveloped. Here, we investigate whether the length of 'evolutionary strata' formed by chromosomal inversions expanding the non-recombining sex-linked region (SLR) on proto sex chromosomes can be informative of how selection influenced their fixation. We develop population genetic models to show how the length of an SLR-expanding inversion, and the presence of partially recessive deleterious mutational variation, affect the fixation probability of three different classes of inversions: (i) neutral, (ii) directly beneficial (i.e., due to breakpoint or positional effects), and (iii) those capturing sexually antagonistic (SA) loci. Our models indicate that neutral inversions, and those capturing an SA locus in linkage disequilibrium with the ancestral SLR, will exhibit a strong fixation bias towards small inversions; while unconditionally beneficial inversions, and those capturing a previously unlinked SA locus, will generally favour fixation of larger inversions. Hence, different selection regimes may leave behind unique signatures of evolutionary strata size. 

%The expected inversion length distributions are sensitive to parameters influencing the deleterious mutation load, the physical position of the ancestral SLR, and the shape of the length distribution of new inversions (about which we know very little).

\newpage{}


%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Introduction} \label{sec:Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%

Two characteristic features of sex chromosomes give them a unique role in evolutionary biology: ({\itshape i}) the presence of one or more genes providing a mechanism for sex-determination, and ({\itshape ii}) suppressed recombination in the vicinity of the sex-determining loci, possibly extending to the entire chromosomes. Recombination suppression is a critical early step in sex chromosome evolution because it enables subsequent divergence between the X and Y (or Z and W) chromosomes through the accumulation of point mutations, insertions, deletions, duplications, and rearrangements. In the long term, loss of recombination leads to several familiar defining features of heteromorphic sex chromosomes such as differences in effective population size between X-linked, Y-linked, and autosomal genes, hemizygosity, and dosage compensation \citep{CharlesworthMarais2005, BergeroCharlesworth2009,BeukeboomPerrin2014,LenormandRoze2022}.
	
Classic population genetics theory proposes that heteromorphic sex chromosomes evolve from ancestral autosomes in several steps: a new sex-determination gene (or linked gene cluster) originates on an ancestral pair of autosomes, followed by the accumulation of sexually antagonistic genetic variation in physical/genetic with the sex-determining alleles -- with male-beneficial alleles associated with the proto-Y (or proto-Z) and female-beneficial alleles with the proto-X (or proto-W) chromosomes -- resulting in indirect selection for reduced recombination between these and the sex-determining gene \citep{Fisher1931, Nei1969, Charlesworth1980, Bull1983, Rice1987, Lenormand2003, CharlesworthMarais2005}. Sex-differences in selection, and especially sexually antagonistic selection, are central to this theory. %Indeed, sexually antagonistic selection also plays a key role in theories for the initial evolution of separate sexes from hermaphroditism by means of genetic sex-determination \citep{Charlesworth1978a, Charlesworth1978b, Bull1983, Olito2019}, sex-chromosome turnovers \citep{vanDoornKirkpatrick2007,vanDoornKirkpatrick2010,OttoScottOsmond2018}, and even transitions from environmental to genetic sex determination \citep{MuralidharVeller2018}. 

However, empirical evidence that sexually antagonistic selection drives the evolution of recombination suppression between sex chromosomes remains weak. On one hand, influential sex-limited selection experiments and population genomic analyses of heteromorphic sex chromosomes demonstrate that sexually antagonistic variation can accumulate on sex chromosomes, apparently supporting the above theory \cite[e.g.,][]{Rice1992,Chippindale2001,Gibson2002, ZhouBachtrog2012,QiuBergeroCharlesworth2013}. On the other hand, it is often difficult or impossible to determine whether the accumulation of sexually antagonistic variation in fact preceded the evolution of suppressed recombination \citep{Charlesworth1980, Rice1984, Ironside2010, Ponnikas2018}. Studies identifying sexually antagonistic variation within sex linked regions on established sex chromosomes provide only weak support for the above theory for the same reason \cite[e.g.][]{BergeroCharlesworth2009,QiuBergeroCharlesworth2013,KirkpatrickGuerrero2014, Wright2017, BergeroCharlesworth2019}.

However, several other hypotheses besides sexually antagonistic selection have beeen proposed that could explain the evolution of suppressed recombination between sex chromosomes, including: ({\itshape i}) "sheltering" of partially recessive deleterious mutations on sex-limited chromosomal regions \citep{Ironside2010, Ponnikas2018, Olito-etal-2022, Jay2022}, and the closely related idea of neutral chromosomal rearrangements or accumulated sequence dissimilarities drifting to fixation \citep{CharlesworthMarais2005}; ({\itshape ii}) positive selection of a beneficial chromosomal rearrangement suppressing recombination \citep{Haldane1957} due to, for example, breakpoint location effects \citep{CorbettDetig2016}; ({\itshape iii}) establishment of a meiotic drive element in tight linkage with a sex-determining factor \citep{UbedaPatten2010}; and ({\itshape iv}) regulatory degeneration in newly sex-linked regions resulting in sex-specific regulatory evolution and positive selection for recombination-suppressing inversions \citep{LenormandRoze2022}. Compared to the sexually antagonistic selection hypothesis, these alternative scenarios are theoretically and empirically underdeveloped (reviewed in \citealt{Ironside2010, Ponnikas2018,Olito-etal-2022}). If unique genomic signatures could be ascribed to different processes, empiricists could potentially discriminate between competing theories of recombination suppression using sequence data. 

One potentially informative signature of different drivers of recombination suppression is the length of 'evolutionary strata' (discrete sex-linked regions with different levels of sequence differentiation). Evolutionary strata can form when the non-recombining sex-linked region (SLR hereafter) is expanded by fixation of inversions inhibiting crossovers between the X and Y (Z and W) chromosomes (or other large-effect recombination modifiers). They also appear to be relatively common: fixation of multiple inversions has generated evolutionary strata on both ancient heteromorphic and younger homomorphic sex chromosomes in both plants and animals \citep{LahnPage1999,Handley2004, Wang2012}, and are becoming increasingly feasible to identify from long-read genome sequence data \citep{WellenreutherBernatchez2018}. 

The successful establishment of new inversions depends on the balance of opposing size-dependent processes: larger inversions are more likely to capture beneficial mutations or combinations of coadapted alleles, but also deleterious mutations that could wipe out any selective benefit (\citealt{Nei1967,vanValenLevins1968, Santos1986, ChengKirkpatrick2019}). \citet{ConnallonOlito2021} developed this theoretical framework for autosomal inversions under a variety of selection scenarios, but the situation is more complicated for still-recombining sex chromosomes, and can result in qualitatively different theoretical predictions. Recently, \citet{Olito-etal-2022} showed that the fixation probability of neutral SLR-expanding inversions differs from that of autosomal ones when the time-dependent selection effects caused by partially recessive deleterious mutations are taken into account (and that previous formulations of the 'sheltering' hypothesis in fact imply this scenario). Similar differences are likely to emerge for other selection scenarios, with non-intuitive outcomes. For instance, incomplete genetic linkage between the SLR and sexually antagonistic loci makes the conditions for maintaining sexually antagonistic polymorphisms more permissive, but also generates linkage disequilibrium between male-beneficial alleles and the male-limited SLR-region, thereby reducing any indirect fitness benefit of reducing the recombination rate further \citep{Nei1969, JordanCharlesworth2012, Otto2019}. Another complication unique to recombining sex chromosomes is that an inversion must both span the SLR and subsequently fix in the population in order to expand the non-recombining region and establish a new evolutionary stratum.

Here, we extend the theoretical framework developed by \citet{vanValenLevins1968, Santos1986}, and more recently by \citet{ConnallonOlito2021} and \citet{Olito-etal-2022} to ask: does the length of chromosomal inversions expanding the SLR reflect the evolutionary processes driving their fixation? We examine three main scenarios: ($1$) the evolution of neutral inversions (i.e., we revisit the 'sheltering' hypothesis), ($2$) intrinsically beneficial inversions (due to breakpoint effects \citealt{CorbettDetig2016}), and ($3$) inversions under indirect selection due to the capture of a sexually antagonistic locus. We do not consider the meiotic drive hypothesis \citep{UbedaPatten2010} because it deals with the origination of genetic sex-determination rather than expansion of an existing SLR, nor do we model the regulatory degeneration hypothesis \citep[see]{LenormandRoze2022}. In fact, the latter hypothesis begins with fixation of a neutral inversion under deleterious mutation pressure (as in \citealt{Olito-etal-2022,Jay2022}), which is followed by extensive gene regulatory evolution. Our goal is to describe the probability that a new inversion expanding the SLR on a proto-Y chromosome goes to fixation as a function of inversion length for each selection scenario, while explicitly taking into account the fitness effects of segregating partially recessive deleterious mutational variation on the proto sex chromosomes. We then use these fixation probabilities to illustrate the effects of each selection scenario on the length distribution of fixed SLR-expanding inversions \citep[after][]{vanValenLevins1968,Santos1986,ConnallonOlito2021}. 

%\hl{Precis of key results likely to change once we've included partially recessive del. mutations...} Our theoretical predictions suggest that evolutionary strata formed by the fixation of neutral inversions should be distinctly larger than those fixed under the other selection scenarios. However, except under certain conditions, it will be difficult to distinguish evolutionary strata formed by the fixation of inversions under direct or indirect selection (i.e., sexually antagonistic) from their lengths. An interesting prediction of our models is that the physical location of the SLR on the sex chromosomes is the single most influential factor determining the relation between inversion size and the probability of expanding the SLR. We conclude by briefly reviewing available data for sex-linked inversions on recombining sex chromosomes, discussing how our predictions might be used to help distinguish between different processes potentially driving the evolution of suppressed recombination between sex chromosomes. We propose a suite of new questions about how the genomic location of the ancestral SLR potentially affects the process of recombination suppression between sex chromosomes.



%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Models and Results} \label{sec:Models}
%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Overview and Key Assumptions} \label{sec:assumptions}
As in \citet{Olito-etal-2022}, we make several important simplifying assumptions in our models. First, sex is assumed to be genetically determined, with a dominant male-determining factor (i.e., an X-Y system with heterozygous males; reversal of male- and female-specific parameters corresponds to a model of Z-W systems). Second, the gene(s) involved in sex determination are located in a sufficiently small non-recombining SLR that they can effectively be treated as a single locus. Hence, our models are most applicable to the early stages of recombination suppression, when the SLR is still small relative to the chromosome arm on which it resides and the length of inversions expanding it. Outside of the SLR, in the pseudoautosomal region (PAR), the sex chromosomes still recombine. For SLR-expanding inversions capturing an SA locus (selection scenario $3$), the rate of recombination between the ancestral SLR and the SA locus in females and males without an inverted Y becomes a relevant parameter ($0 < r_{SA} \leq \frac{1}{2}$). %; see figure \ref{fig:diagramFig}A). For ease of comparison in our models, we further distinguish two regions within the PAR based on the mode of inheritance and 'behavior' of genes located therein: ({\itshape i}) the sex-linked PAR ({\itshape sl}-PAR) \leq r < 1/2$; and ({\itshape ii}) the autosomal PAR ({\itshape a}-PAR) where $r = 1/2$ (see figure \ref{fig:diagramFig}A). 
Third, we focus on paracentric inversions (those not spanning the centromere), which are more common than pericentric inversions \citep{WellenreutherBernatchez2018}. Fourth, we assume new inversions occur rarely enough that all inverted chromosomes segregating in a population are descendent copies of a single ancestral inversion mutation. The evolutionary fate of each new inversion mutation is therefore independent of any others (i.e., we assume weak mutation and strong selection with respect to inversions; \citealt{Gillespie1991}). Fifth, recombination is completely suppressed between heterokaryotypes, although in reality genetic exchange may rarely occur via double crossovers or gene conversion \citep{KrimbasPowell1992, KorunesNoor2019}. Finally, we assume that the timescale for fixation of a new inversion is much shorter than that for the evolution of genetic degeneration and dosage compensation in the chromosomal region spanned by the inversion. This assumption is justified by the relative rates of fixation for beneficial mutations compared to that for multiple 'clicks' of Muller's ratchet or the fixation of weakly deleterious mutations due to background selection (\citealt{Charlesworth2000, Bachtrog2008}; but see \citealt{LenormandRoze2020, LenormandRoze2022}).

We focus on the evolutionary fate of inversions expanding the male-limited SLR on a Y chromosome. Inversions spanning the SLR on an X chromosome may also suppress recombination between sex chromosomes if they fix within a population, but Y-linked inversions are more likely to do (we highlight essential differences between models of inversions on the Y and X chromosomes in \hl{Appendix X}). 
%so because they have a smaller effective population size than X-linked inversions ($N_Y < N_X$), and have no opportunity to experience sex-differences in selection 

We begin by modelling the fixation probability of an inversion conditioned on its expanding the SLR as a function of inversion length for each selection scenario. Our models explicitly take into account the effects of partially recessive deleterious mutational variation \citep[after][]{Olito-etal-2022}, which represents the most common dominance scenario for deleterious mutations (\citealt{AgrawalWhitlock2012,Manna2011,Huber2018}; reviewed in \citealt{Billiard-etal-2021}). Next, we examine how the predicted fixation probabilities change when the likelihood that new inveresions actually capture and expand the ancestral SLR are also taken into account. Finally, we illustrate how these combined effects alter the expected length distributions of fixed SLR-expanding inversions for each selection scenario \citep[as in][]{ConnallonOlito2021}. Full details for each model are provided in the Supporting Information, and simulation code is available at \url{https://github.com/colin-olito/inversionSize-ProtoSexChrom} and a version of record is archived on Zenodo (\hl{cite Zenodo}).




 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=1]{./simpleDiagramFig.pdf}
 \caption{\textbf{Do we even need a schematic figure?} I've fiddled a bit with editing our old figure here, but it doesn't quite work as well as before because now that we are dealing with partially deleterious mutations several of the cases we highlighted before don't really need highlighting anymore... We could still include a diagram of our cartoon chromosomes as above, but I'm not sure how much it helps... . }
 %\caption{\textbf{(A)} Simplified diagram of recombining sex chromosomes in the models illustrating the three main chromosomal regions with distinct evolutionary dynamics: ({\itshape i}) the non-recombining sex-determining region (SLR; orange and purple bars), containing the sex-determining gene(s); ({\itshape ii}) the autosomal-PAR, or {\itshape a}-PAR, region, in which there is free recombination between the sex chromosomes ($r = 0.5$; white). Genes located in the a-PAR are physically sex-linked, yet exhibit evolutionary dynamics that are identical to autosomal genes because they recombine freely; and ({\itshape iii}) The sex-linked pseudo-autosomal region ({\itshape sl}-PAR), which is physically adjacent to the SLR, and in which the recombination rate between sex chromosomes is $0 \leq r < 0.5$ (indicated by blue shading). Due to partial linkage with the SLR, genes contained within this region exhibit evolutionary dynamics that are distinct from the other two regions, particularly those with sexually antagonistic effects (reviewed in \citealt{Otto2011}). \textbf{(B)} Illustration of new chromosomal inversions capturing the SLR and a single SA locus (see fitness expressions in Table \ref{tab:fitness}) on the Y chromosome highlighting several key features of the theoretical models, with reference to the fixation probability provided in the main text. From top to bottom, the diagrams illustrate: ({\itshape i}) new inversions capturing a deleterious mutation will not spread, and this is more likely for larger inversions; ({\itshape ii}) a mutation-free inversion on the proto-Y capturing a female-beneficial allele will not spread; ({\itshape iii} -- {\itshape iv}) a mutation-free inversion on the proto-Y capturing a male-beneficial allele can spread, and will have a fixation probability equal to Eq(\ref{eq:SApFix2LocUnlinked}) if the SA locus is located in the {\itshape a}-PAR, and Eq(\ref{eq:SAsIpFix2LocLinked}) if it is located in the {\itshape sl}-PAR. Note that inversions completely suppress recombination between the sex chromosomes ($r = 0$ inside 'new inversion' brackets). See supplementary figure S1 in Appendix D of the Online Supplementary Material for illustrative figures for other scenarios explored in this paper.}
 \label{fig:diagramFig}
 \end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Linking selection to inversion fixation probabilities}

Under our stated assumptions, whether or not a new inversion on a proto-Y chromosome will expand the SLR depends upon three factors: (1) the selection scenario (neutral, beneficial, or SA selection); (2) the time-dependent selection process caused by deleterious allele frequency dynamics at loci spanned by the inversion; and (3) whether the inversion spans the ancestral SLR. The latter two effects vary systematically with inversion length \citep{ConnallonOlito2021, Olito-etal-2022}, and the first will when inversions capture an SA locus, but will not for neutral or beneficial inversions. We can express the overall probability that a new inversion of length $x$ (expressed as the proportion of the chromosome arm it spans) on a proto-Y chromosome ultimately expands the SLR as the product of two probabilities:
\begin{linenomath*}
\begin{equation}\label{eq:generalPrFix}
  \Pr(\text{expand SLR} \mid x) = \underbrace{\Pr(\text{fix} \mid x, n,sel.)}_{\text{Effects of selection \& del. mut.}} \times \underbrace{\Pr(\text{SLR} \mid x)}_{\text{Prob. capturing SLR}},
\end{equation}
\end{linenomath*}

\noindent where $\Pr(\text{fix} \mid x, n, sel.)$ represents the probability that the inversion fixes in the population of Y chromosomes given its' length ($x$), the number of loci it spans under deleterious mutation pressure ($n$), and the details of the selection scenario ($sel.$), and $\Pr(\text{SLR} \mid x)$ is the probability that the inversion spans the ancestral SLR.

The time-dependent effects of deleterious mutations are similar for all selection scenarios, so we begin by briefly outlining this form of indirect selection (\citealt[see][]{Olito-etal-2022} for a detailed study of this process). We define the total number of selected loci outside of the ancestral SLR, $n_{tot}$, and the length of an inversion, $x$, expressed as the proportion of the chromosome arm spanned by the inversion (where $0 < x < 1$). Assuming that the selected loci are uniformly distributed along the chromosome arm, the number of loci spanned by a new inversion is $n = n_{tot}x$. Each of the $n_{tot}$ loci are assumed to be diallelic, with a wild-type allele ($A$) which mutates to a deleterious variant ($a$) at the $i^{th}$ locus at a rate $\mu_i$ per meiosis (we ignore backmutation from $a \rightarrow A$), with conventional relative fitness expressions for each genotype: $w_{i,AA} = 1$, $w_{i,Aa} = 1 - h_i s_i$, and $w_{i,a} = 1 - s_i$. We assume that deleterious mutations segregate independently at each locus, and are at mutation-selection balance equilibrium frequencies ($\hat{q}_i = \mu_i/h_i s_i$) prior to the origin of a given inversion. 

A new inversion will capture a random sample of the standing deleterious variation at the $n$ loci it spans, which fall into two categories: loci where the inversion initially captures a deleterious allele (denoted by a superscript $D$) or a wild-type allele (denoted by a superscript $W$), respectively. Modeling the frequency dynamics of an SLR-expanding inversion requires that we track allele frequency changes for these two classes of loci within four different classes of chromosomes: X's in female gametes, X's in male gametes, non-inverted Y's, and inverted Y's, which are denoted $X_f$, $X_m$, $Y$, and $Y^I$ respectively \citep[see][]{Otto2014,Olito-etal-2022}. We denote the frequencies at time $t$ for each of the $n$ loci using the following notation: $q_{X_f,t}^{D,i}$, $q_{X_m,t}^{D,i}$, $q_{Y,t}^{D,i}$, $q_{Y^I,t}^{D,i}$, and $q_{X_f,t}^{W,i}$, $q_{X_m,t}^{W,i}$, $q_{Y,t}^{W,i}$, $q_{Y^I,t}^{W,i}$, where $q$ indicates the deleterious allele frequency at the $i^{th}$ locus. Note that $q_{Y^I,t}^{D,i} = 1$ for all $t$ due to our assumption of no back-mutation, while $q_{Y^I,t}^{W,i} = 0$ at $t=1$ but will increase over time due to the occurrence of new mutations on descendent copies of the inversion. A full development of the deterministic recursions for deleterious allele frequencies is presented in \citep{Olito-etal-2022}. 

To incorporate the effects of time-dependent indirect selection due to segregating deleterious mutational variation on the fixation probabilities of different length inversions in our models, we used Wright-Fisher simulations performed in R \citep{RSoftware}. Following \citet{Olito-etal-2022}, we made the simplifying assumption that deleterious allele frequencies at the $n$ loci spanned by an inversion change deterministically, while the inversion itself is subject to genetic drift; an approach that qualitatively captures the time-dependent selection effects of deleterious variation and is most applicable to large population sizes. In each generation, haplotype frequencies were censused among gametes prior to fertilization, following mutation and indirect selection due to segregating deleterious alleles at each locus. We then used exact deterministic recursions to predict gene frequencies at each of the $n$ loci after selection (see \citealt{Olito-etal-2022}; Appendix \hl{X}). The realized frequency of the inversion among Y chromosomes in each generation was then simulated using pseudo-random binomial sampling during the gametic phase, with the deterministic frequency representing the probability of sampling the inversion among male gametes, and the number of Y chromosomes representing the number of trials ($N/2$). For each replicate simulation, a single-copy inversion mutation was introduced into a population of $N$ individuals initially at mutation-selection equilibrium. The number of loci spanned by the new inversion was $n = n_{tot} x$ and the number of deleterious alleles initially captured by a new inversion of length $x$ was drawn from a Poisson distribution with mean and variance $U x/hs$, where $U = n_{tot} \mu$. Fixation probabilities were estimated from the outcomes of at least $100 \times N/2$ replicate simulations (this was increased to $500 \times N/2$ for larger inversions because of their low fixation probabilities).

Below, we first present results for the fixation probability of inversions of different lenghts for each selection scneario conditioned on them also spanning the SLR (i.e., we temporarily assume $\Pr(\text{SLR} \mid x) = 1$). To highlight the effects of deleterious mutational variation, we also derive simple expressions for $\Pr(\text{fix} \mid x)$ in the absence of deleterious mutational variation (i.e., setting $U = 0$). We then relax the assumption that new inversions span the SLR by deriving expressions for $\Pr(\text{SLR} \mid x)$ based on a simple geometric argument, and illustrate the interaction between inversion size and the location of the ancestral SLR on the fixation probability of inversions expanding it. Computer code for the simulations is provided in the Supplementary Material, and is available on Zenodo (\url{hhtps://Zenodo.link.com}) and at \url{https://anonymized.github.site.com}. %\url{https://github.com/colin-olito/inversionSize-ProtoSexChrom}.



%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Fixation Probabilities}

\subsubsection*{Neutral Inversions (the sheltering hypothesis)}

A neutral inversion (i.e., one without any direct fitness effects) spanning the SLR on a Y chromosome is a plausible instance of earlier sheltering hypotheses for the evolution of recombination arrest between sex chromosomes (see \citealt[][]{Olito-etal-2022} for a summary of previous formulations of this idea). Under partially recessive deleterious mutation pressure, the evolutionary fate of neutral inversions expanding the SLR emerge from time-dependent indirect selection effects caused by the number of mutations initially captured by the inversrion, and the process of mutation accumulation on descendent copies of the inversion. In brief, initially mutation-free or lightly-loaded inversions (those initially capturing far fewer than the population average over the chromosomal segment it spans) enjoy an ephemeral selective advantage that erodes as new mutations accumulate on descendent copies of the inversion. If the original inversion mutation captures no deleterious mutations, it will accumulate new mutations until it eventually approaches the population average load and becomes selectively neutral. However, if it captures even one deleterious mutation, it becomes irreversibly deleterious in the long-term because all descendent inversion copies will be fixed for those deleterious mutations. The fixation process is therefore a race against time, the outcome of which is influenced by parameters affecting the population average deleterious mutation load. Smaller inversions are systematically favoured in this scenario because they are more likely to capture very few deleterious mutations (fig.~\ref{fig:PrFixFig}A; see also \citealt{Olito-etal-2022}). 

The overall effect of indirect selection due to partially deleterious mutational variation on the fixation probabilities of different length inversions can be illustrated by a simple comparison. A neutral SLR-expanding inversion in a population with an even sex ratio will have an effective population size of $N_Y = N/2$, where $N$ is the total breeding population size. Ignoring the effects of deleterious mutations, the fixation probability becomes independent of inversion length, and is equal to its' initial frequency: $\Pr(\text{fix}) = N_Y^{-1} = 2/N$ for a single copy inversion mutation \citep{Kimura1962, CrowKimura1970}, which is benchmarked by the horizontal dashed line in fig.~\ref{fig:PrFixFig}A.




%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Beneficial inversions}

The specific location of new inversion breakpoints may give inverted chromosomes an intrinsic selective advantage over wild-type chromosomes. For example, an inversion may bring a protein coding sequence into closer proximity to a promoter region, thereby altering expression without disrupting other genes, or by disrupting synteny near 'sensitive sites' \citep{KrimbasPowell1992, CorbettDetig2016}. Under weak selection (when $1/N \ll s_b \ll 1$) and momentarily ignoring the effect of deleterious mutations, the fixation probability of a beneficial inversion can be approximated by $\Pr(\text{fix}) \approx 2 s_{b}$ \citep{Haldane1927} (where $s_b$ is the heterozygous selective advantage of the inversion), which is independent of inversion length (fig.~\ref{fig:PrFixFig}B).

The effects of deleterious mutations are essentialy the same for beneficial SLR-expanding inversions as for neutral ones. A new beneficial inversion that is free of deleterious mutations will have a temporary selective advantage caused by both it's direct fitness effects and those caused by deleterious mutational variation that will decline over time, eventually leaving only the intrinsic advantage, $(1 + s_b)$ \citep{Nei1967}. For lightly-loaded inversions, the intrinsic selective benefit will offset some of the permanent deleterious load caused by initially captured deleterious mutations, slightly increasing the fixation probability of larger inversions. Overall, the balance of these two selective processes results in a fixation bias for smaller SLR-expanding inversions, but with a larger average size compared to the netural scenario (fig.~\ref{fig:PrFixFig}). Intuitively, the magnitude of the shift in fixation bias towards larger inversions will depend on the strength of the intrinsic selective benefit of the inversiong, $s_b$. The time-dependent effects of deleterious mutations also elevates the fixation probability of small inversions so that it exceeds the standard approximation of $2s_b$.


%%%%%%%%%%%%%%%
 \begin{figure}[htbp]
 \centering
 \includegraphics[width=0.94\linewidth]{./PrFix_PrCatchSA_combined_delMut}
 \caption[\captionsize]{Fixation probabilities for inversions of different lengths capturing the SLR on the Y-chromosome estimated from Wright-Fisher simulations. Point shapes indicate different chromosome-arm wide mutation rates relative to selection (i.e., different values of $U$), which influence the average numbers of deleterious mutations carried by a standard-arrangement chromosome ($U/hs$). Dashed lines correspond to approximations of $\Pr(\text{fix} \mid x)$ when ignoring the effects of partially recessive deleterious mutation pressure for each selection scenario (see text for approximations for each scenario). Parameters for deleterious mutations are the same for all panels: $h = 0.25$, $s = 0.01$, $n_{tot} = 10^4$. The selection coefficient for unconditionally beneficial inversions (B) is $s_b = 0.02$. For the SA selection scenario (C--D) we explored scenarios of strong and weak linkage disequilibrium between the SLR and SA locus ($r_{SA} = \{0.002,\, 0.5\}$), with $s_f = s_m = 0.05$, $h_f = h_m = 0.5$, $A = 1$, $P = 0.05$. All results condition on the inversion spanning the SLR.}
 \label{fig:PrFixFig}
 \end{figure}
%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Sexually antagonistic selection}\label{sec:SexAntag}

It is well established that sexually antagonistic (SA) variation can theoretically drive selection for recombination modifiers coupling selected alleles with specific sex chromosomes \cite[e.g.][]{Fisher1931,Nei1969, Charlesworth1978a, Charlesworth1980, Bull1983,Lenormand2003, Otto2019}. However, the role of pre-existing linkage disequilibrium between the SLR and SA loci in this process is complicated. The conditions for maintaining SA polymorphisms become more permissive when there is linkage disequilibrium with a sex-determining locus \citep{JordanCharlesworth2012}. Moreover, the idea that SA polymorphisms genetically linked to the SLR will promote the accumulation of more linked SA polymorphisms, leading to stronger selection for recombination suppression seems superficially intuitive \citep{Rice1984, Rice1996, Charlesworth2017, Otto2019}. Yet, the conditions for the spread of such linked SA polymorphisms are in fact quite restrictive \citep{Otto2019}. Clearly, when a new SLR-expanding inversion captures SA loci, the extent of prior linkage disequilibrium between the SLR and those SA loci will determine its overall fitness effect, and so the ancestral recombination rate between the SLR and SA loci is a key parameter. To gain some intuition for the effects of linkage disequilibrium between the SLR and SA loci on the fixation probability of SLR-expanding inversions, we examine two idealized scenarios reperesenting corner cases of loose and tight genetic linkage bewteen the SLR and SA loci.

Suppose the average number of SA loci on the sex chromosomes is equal to $A$, that they are uniformly distributed along the chromosomes, are biallelic with standard SA fitness expressions {\itshape sensu} \citet{Kidwell1977} (i.e., $w_{11}^{f} = 1$, $w_{12}^{f} = 1 - h_f s_f$, $w_{22}^{f} = 1 - s_f$, and $w_{11}^{m} = 1 - s_m$, $w_{12}^{m} = 1 - h_m s_m$, $w_{22}^{m} = 1$; where $\{f,m\}$ index females and males respectively), and are assumed to be at equilibrium prior to the inversion mutation. If SA loci are uniformly distributed along the chromosome arm, the number of loci spanned by a new inversion, $n_{SA}$, will be a Poisson distributed random variable with mean and variance $xA$. For simplicity, we assume that $A$ is sufficiently small to ignore the possibility that $n_{SA}$ is greater than $1$. 

In this simplified scenario, the fixation probability of a new inversion of length $x$ on the Y chromosome, $\Pr(\text{fix} \mid x)$ from Eq.(\ref{eq:generalPrFix}), is itself the product of three probabilities: ($1$) that the inversion captures the SA locus, $\Pr(n_{SA} = 1) = xA e^{-xA}$; ($2$) that it captures a male-beneficial allele at the SA locus, $\Pr(\text{male~ben.}) = \hat{q}$, where $\hat{q}$ is the equilibrium frequency of the male-beneficial allele; and ($3$) that it escapes stochastic loss due to genetic drift and fixes in the population, $\Pr(\text{fix}) \approx 2 s_I$ (where $s_I$ is the overall selection coefficient for the inversion due to indirect selection for reduced recombination, and we momentarily ignore effects of deleterious mutations; \citealt{Haldane1927}). The expected rate of increase of a rare inversion can be approximated as $s_I \approx (\lambda_I - 1)$, where $\lambda_I$ is the eigenvalue associated with invasion of the inversion genotype into a population inititally at equilibrium in a deterministic two-locus model involving the SLR and SA locus ($\lambda_I$ is also the leading eigenvalue under these conditions; see Appendix \hl{XXX} for full development of the deterministic 2-locus model). 

When the SA locus is initially unlinked with the SLR ($r_{SA} = 1/2$), the selection coefficient for a rare SLR-expanding inversion capturing the SA locus is
\begin{linenomath*}
\begin{equation}\label{eq:SApFix2LocUnlinked}
	s_I \approx s_m (1 - \hat{q}) \big( 1 - \hat{q} - h_m(1 - 2\hat{q}) \big) + O(s_{m}^{2}),
\end{equation}
\end{linenomath*}

\noindent where $s_m$ is the selection coefficient of the male-deleterious/female-beneficial allele in males. With additive SA fitness ($h_f = h_m = 1/2$), the fixation probability reduces to
\begin{linenomath*}
\begin{equation}\label{eq:SApFix2LocUnlinkedYAdd}
	\Pr(\text{fix} \mid x) = s_m \hat{q} (1 - \hat{q}) xA e^{-xA}.
\end{equation}
\end{linenomath*}

\noindent Eq(\ref{eq:SApFix2LocUnlinkedYAdd}) is convex sigmoidal increasing function of inversion size over $0 < x \leq 1$, with a maximum at $\tilde{x} = 1/A $, implying that intermediate to large inversions will be favoured (recall that $A \leq 1 $) (fig.~\ref{fig:PrFixFig}C, dashed line). Intuitively, larger inversions are more likely to capture rare SA loci distributed uniformly along the chromosome arm, which is necessary for them to have any selective advantage under the SA selection scenario.

To include prior linkage disequilibrium between the SLR and SA locus, we make the additional simplifying assumption that genetic linkage requires relatively tight physical linkage. Specifically, we assume that the SA locus is located in the region corresponding to the fraction, $P$, of the chromosome arm immediately adjacent to the SLR, and that $P \ll x$. Hence, any inversion that expands the SLR will also capture the SA locus. In this case, the probability of capture the SA locus is $\Pr(n_{SA} = 1) = AP e^{-AP}$. We can approximate $s_I$ from the deterministic two-locus model as before, but the expression now involves the equilibrium frequencies of the male-beneficial allele on Y chromosomes ($\hat{Y}$) and X chromosomes in females ($\hat{X}_f$) before the inversion occurs:
\begin{linenomath*}
\begin{equation}\label{eq:SAsI2LocLinkedY}
	s_I \approx \frac{ s_m(1 - \hat{Y}) \big( 1 - \hat{X}_f - h_m(1 - 2\hat{X}_f) \big)} { 1 - s_m \big(1 - \hat{X}_f - \hat{Y}(1 - h_m - \hat{X}_f) + h_m \hat{X}_f(1 - 2 \hat{Y}) \big) }.
\end{equation}
\end{linenomath*}

\noindent From Eq.(\ref{eq:SAsI2LocLinkedY}), we see that linkage disequilibrium between the SLR and SA locus influences indirect selection for the inversion by altering the equilibrium frequencies of the male-beneficial allele on Y chromosomes, and X chromosomes in females (note that the ancestral recombination rate $r_{SA}$ drops out of Eq.~\ref{eq:SAsI2LocLinkedY}). Interestingly, the effect of $r_{SA}$ on the overall selection coefficient for the inversion can take different forms, depending on the relative strength of selection on the SA alleles in males and females (see Supplementary fig.~\hl{SX--X}). In this way, the SA selection coefficients can influence whether inversions capturing loosely or tightly linked SA-loci are more strongly favoured. 

Under additive SA selection ($h_f = h_m = 1/2$), the fixation probability simplifies to
\begin{linenomath*}
\begin{equation}\label{eq:SAsIpFix2LocLinked}
	\Pr(\text{fix}) = \frac{ 2 s_m \hat{Y} (1 - \hat{Y}) A P e^{-A P}}{ 2 - s_m (2 - \hat{X}_f - \hat{Y}) },
\end{equation}
\end{linenomath*}

\noindent which is independent of $x$ (fig.~\ref{fig:PrFixFig}D, dashed line). The key intuitions from our simple scenarios are as follows: The overall effect of physical and genetic linkage between the SLR and SA locus is to shift the fixation probability towards smaller inversions because large inversions no longer have an increased probability of capturing the SA locus. In the limiting case where $P \ll x$ the fixation probability is independent of inversion size. Less extreme mappings between genetic and physical linkage will result in fixation probabilities that are weakly dependent on $x$. Sex-biases in selection can alter how tightly linked the SLR and SA locus must be to maximize an inversion's fixation probability.

As in the selection scenarios, time-dependent effects of partially recessive deleterious mutational variation skew fixation probabilities towards smaller inversions. The combined effect of indirect selection due to SA selection and deleterious mutations, however, results in very different patterns depending on the extent of prior linkage disequilibrium between the SLR and SA locus. 

When the captured SA locus is initially unlinked with the SLR (fig.~\ref{fig:PrFixFig}D; $r_{SA} = 0.5$), intermediately sized inversions have the highest fixation probability because they balance the countervailing effects of inversion size on the likelihood of successfully capturing the SLR and SA loci (larger is better), and minimizing the chance of capturing deleterious mutations (smaller is better). In contrast, when the captured SA locus is tightly linked to the SLR (fig.~\ref{fig:PrFixFig}D; $r_{SA} = 0.002$), inversions capturing the SA locus behave similarly to neutral or (weakly) unconditionally beneficial inversions, with small inversions strongly favoured (fig.~\ref{fig:PrFixFig}D). This similarity in behaviour arises because (a) inversion size does not significantly influence the chance of capturing a tightly linked SA locus; but (b) smaller inversions still minimize the chance of capturing deleterious mutations; and (c) there is only weak indirect selection for complete recombination suppression when there is strong genetic linkage between the SLR and SA locus.



%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Probability of expanding the SLR}\label{sec:ProbExpSLR}

So far, we have presented results that are conditioned on inversions spanning the SLR to illustrate the relation between selection and inversion size for each scenario (we have assumed $\Pr(\text{SLR} \mid x) = 1$). %Under this assumption, the models suggest that the length of fixed inversions expanding the SLR will reflect the selective process underlying their fixation: neutral, directly beneficial, and indirectly beneficial inversions will leave distinct footprints of different sized evolutionary strata. 
We now relax this assumption and examine the effects of explicitly modeling the probability that new inversions span the ancestral SLR.

Under our assumption that inversions are equally likely to occur at any point along the chromosome arm on which the SLR resides, the probability that a given inversion will span the SLR depends on two factors: the length of the inversion ($x$) and the position of the ancestral SLR on the chromosome arm (denoted $\text{SLR}_{\text{pos}}$). The chromosome arm can be subdivided into three regions: from the centromere to the SLR ($y_1$), the SLR itself ($y_2$), and from the SLR to the distal end ($y_3$), where $y_1 + y_2 + y_3 = 1$. If the ancestral SLR is small relative to the length of new inversions (as we have assumed), $y_2 \approx 0$ and $y_1 + y_3 \approx 1$. A simple geometric argument suggests that the probability of a new inversion of length $x$ expanding the SLR can be described as a piecewise function of $x$:
\begin{linenomath*}
\begin{equation}\label{eq:PrSpanSLR}
	\Pr(\text{SLR} \mid x) = \left\{ 
		\begin{array}{ccl} 
			x  /(1 - x) & \mbox{for} & x \leq y_1,\,y_3 \\
			y_1/(1 - x) & \mbox{for} & y_1 < x < y_3 \\ 
			y_3/(1 - x) & \mbox{for} & y_1 > x > y_3 \\ 
			1 & \mbox{for} & x > y_1,\,y_3 \numberthis			
		\end{array}\right.
\end{equation}
\end{linenomath*}

\noindent where $y_1 = \text{SLR}_{\text{pos}}$ and $y_3 = (1 - \text{SLR}_{\text{pos}})$. 

For simplicity, we re-examine the fixation probability of new inversions in each selection scenario under two representative values of $\text{SLR}_{\text{pos}}$: ($1$) the SLR is located at the exact center of the chromosome arm ($\text{SLR}_{\text{pos}} = 0.5$), and ($2$) the SLR is located near one end of the chromosome arm ($\text{SLR}_{\text{pos}} = 0.1$; results are identical if $\text{SLR}_{\text{pos}} = 0.9$). Intermediate values of $\text{SLR}_{\text{pos}}$ yield predictions that fall between these extremes. An illustration of Eq(\ref{eq:PrSpanSLR}) is provided in \hl{Appendix D of the Online Supplementary Material}.  

The main effect of explicitly modeling $\Pr(\text{SLR} \mid x)$ on the relation between inversion size and the probability of expanding the SLR is to shift the probability densities towards larger inversions (fig.~\ref{fig:PrSLRFixFig}A--D). When the SLR is located in the middle of the chromosome arm ($\text{SLR}_{\text{pos}} = 0.5$) the probability that new inversions expanding the SLR increases until $x = 0.5$, after which it plateaus (figure \ref{fig:PrSLRFixFig}A). Intuitively, the probability that a new inversion spans the SLR increases until $x > \{\text{SLR}_{\text{pos}},\,(1 - \text{SLR}_{\text{pos}})\}$, above which the SLR will always be captured. A similar, but more exaggerated pattern favouring large inversions emerges when the SLR is located near one end of the chromosome arm (fig.~\ref{fig:PrSLRFixFig}E--H).

%%%%%%%%%%%%%%%%%%%%%%%
 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.53]{./PrFix_SpanSLR_combined_delMut}
 \caption{Effects of explicitly modeling the probability of spanning the ancestral SLR on the fixation probability of SLR-expanding inversions of different lengths. Each panel shows the overall fixation probability of new inversions of length $x$, evaluated with Eq(\ref{eq:PrSpanSLR}) substituted into Eq(\ref{eq:generalPrFix}) for each selection scenario. Panels A--D show results when the ancestral SLR is located in the exact middle of the chromosome arm ($\text{SLR}_\text{pos} = 0.5$); panels E--H show the same when the SLR is located near either the centromere or telomere ($\text{SLR}_\text{pos} = 0.1$). Point shapes indicate different average deleterious mutation loads carried by standard-arrangement chromosomes (as in fig.~\ref{fig:PrFixFig}). Dashed lines correspond to approximations of $\Pr(\text{fix} \mid x, \text{SLR})$ when ignoring the effects of partially recessive deleterious mutation pressure for each selection scenario. All other parameter values are the same as in fig.~\ref{fig:PrFixFig}.}
 \label{fig:PrSLRFixFig}
 \end{figure}
%%%%%%%%%%%%%%%%%%%%%%%

The behaviour of the models for different selection scenarios now breaks down into two basic patterns. Neutral inversions and those capturing an SA locus tightly linked with the ancestral SLR yield fixation probabilities that favour smaller inversions, particularly when the average deleterious mutation load is high (\ref{fig:PrSLRFixFig}A,C,E,G). In contrast, unconditionally beneficial inversions and those capturing an unlinked SA locus have fixation probabilities that are often biased towards intermediate to large inversions (\ref{fig:PrSLRFixFig}B,D,F,H). This effect is lessened when the deleterious mutation rate is high because of the stronger influence of deleterious mutations, which favour smaller inversions. The different relations between inversion size and fixation probability for the different selection scenarios are most easily distinguished when the mutation rate is low and/or the ancestral SLR is located near one end of the chromosome arm.






%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Expected length distributions of evolutionary strata} \label{subsec:DistFixedInv}

The above expressions for the fixation probability of SLR-expanding inversions suggest that different selection scenarios may leave behind unique signatures of evolutionary strata size. Here, we briefly explore how these fixation probabilities might influence the expected distributions of fixed SLR-expanding inversion sizes. Following \citet{vanValenLevins1968, Santos1986}, and \citet{ConnallonOlito2021}, the proportion of fixed inversions of length $x$ is given by 
\begin{linenomath*}
\begin{equation} \label{eq:generalInvSizeModel}
  g(x) = \frac{\Pr(\text{expand SLR} \mid x) f(x)} {\int \Pr(\text{expand SLR} \mid x) f(x)\,dx},
\end{equation}
\end{linenomath*}

\noindent where $f(x)$ is the probability of a new inversion of length $x$, and $\Pr(\text{fix} \mid x)$ is the fixation probability given in Eq(\ref{eq:generalPrFix}) with appropriate substitutions made for each selection scenario. $x\int \Pr(\text{fix} \mid x) f(x)\,dx$ gives the mean length of fixed inversions. Little is known about how the mutational process for new inversions shapes $f(x)$, and we therefore examine two scenarios representing plausible extremes to illustrate the spectrum of possible outcomes.

On one hand, if inversion breakpoints are distributed uniformly across the chromosome arm containing the SDR, then $f(x) = 2(1 - x)$, a scenario we refer to as the "random breakpoint" model \citep{vanValenLevins1968}. On the other hand, if inversion breakpoints tend to be clustered, for example in chromosomal regions with repetitive sequences, the resulting enrichment of smaller new inversions can be modeled phenomenologically using a truncated exponential distribution:
\begin{linenomath*}
\begin{equation} \label{eq:truncExp}
  f(x) = \frac{ \lambda e^{-\lambda x}} {1 - e^{-\lambda}},
\end{equation}
\end{linenomath*}

\noindent where $\lambda$ is the exponential rate parameter \citep{PevznerTesler2003, PengPevznerTesler2006, ChengKirkpatrick2019,ConnallonOlito2021}. For strongly skewed distributions (e.g., $\lambda > 10$, as we assume here), the truncation effect is negligible, and $f(x)$ is approximately equal to the numerator of Eq(\ref{eq:truncExp}). We refer to this other extreme as the "exponential model". 

%%%%%%%%%%%%%%%%%%%%%%%
 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.65]{./Expected_xDistribution_delMut_Ud0_02}
 \caption{Scaled probability densities for SLR-expanding inversions of different lengths (Eq[\ref{eq:generalInvSizeModel}]). Panels A, C show results for the Random Breakpoint model, while B,D show results for the Exponential model of new inversion sizes. Point shapes indicate different selection scenarios, and corresponding black points along the x-axis indicate distribution means. Results are shown for lowest deleterious mutation rate presented in other figures, which corresponds to an average deleterious mutation load on standard-arrangement chromsomes of $U/hs = 8$. The above distributions therefore represent a best case where differences between the selection scenarios are most obvious. Higher mutation loads cause the expected length distributions to become more strongly skewed and difficult to distinguish (see supplementary fig.~\hl{S.XXX}. All other parameters are the same as in fig.~\ref{fig:PrFixFig}.}
 \label{fig:ExpectedDistFig}
 \end{figure}
%%%%%%%%%%%%%%%%%%%%%%%

Several key results emerge from the expected distributions of evolutionary strata length. First, the expected length distributions for neutral inversions and those capturing a SA locus linked to the SLR are indistinguishable, regardless of the location of the ancestral SLR (fig.~\ref{fig:ExpectedDistFig}), or the average deleterious mutation load (\hl{fig.~SX, Appendix X}. Likewise, unconditionally beneficial inversions and those capturing unlinked SA loci yeild largely overlapping distributions of larger inversions; SA selection generally results in larger mean inversion lengths. Not surprisingly, the distribution of new inversion lengths can strongly influence the resulting distributions are under each selection scenario. Very little about the distribution of new inversion lengths, but our predictions clearly indicate that if it is strongly skewed (e.g., like the exponential model) the scope for different selection scenarios to leave behind unique signatures in the lenghth of evolutionary strata can become quite limited (compare fig.~\ref{fig:ExpectedDistFig}A,C with B,D). An interaction between the distribution of new inversion lengths and the location of the ancestral SLR also emerges from our predictions. When the SLR is located near the end of the chromosome arm, it appears to exaggerate differences between the models of neutral/linked SA selection and the models of beneficial/unlinked SA selection, but only under the random breakpoint model of new inversion lengths. The reverse is true under the exponential distribution, with the different selection scenarios being most distinguishable when the SLR is in the middle of the chromosome arm. 

%To summarize, while different selection scenarios are expected to result in distinct distributions under a random breakpoint model, they become increasingly difficult to distinguish under a strongly skewed exponential model of new inversion lengths. 



%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Discussion} \label{sec:Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%


%\textcolor{blue}{>> CONTINUE EDITING FROM HERE <<}

Our models reveal two major implications for the evolution of recombination suppression between sex chromosomes. The first is that different selection scenarios result in unique relations between SLR-expanding inversion length and fixation probability, suggesting that the length of evolutionary strata can reflect the selective process underlying expansion of the non-recombining SLR. Specifically, our models predict that evolutionary strata formed by the fixation of neutral inversions should be significantly smaller on average than those formed by unconditionally beneficial inversions. Inversions capturing SA loci can drive foromation of either small or large evolutionary strata, depending on the extent of linkage disequilibrium between the SLR and SA loci prior to the inversion mutation: tight linkage results in distributions of evolutionary strata length that are indistinguishable from that of neutral inversions, while capture of unlinked SA loci drives the evolution of the largest strata predicted by any of the selection scenarios we studied. Our results also show that indirect selection due to segregating partially recessive deleterious mutational variation has a strong effect on inversion evolution under all selection scenarios we studied \citep{Olito-etal-2022}.

One obvious application of our findings is to compare the lengths of early evolutionary strata (i.e., those occurring when the ancestral SLR is still quite small) identified from DNA sequence data with the expected length distributions we have derived here. The ongoing development of whole-genome sequencing technology and analyses is making the identification of genome structural variation, including fixed inversions and evolutionary strata on sex chromosomes, increasingly feasible for non-model organisms \citep[reviewed in ][]{Muyle2017, Charlesworth2018,PandayAzad2016}. Small fully sex-linked regions in chromosomes with large regions that still recombine have been found in a variety of unrelated species, including Papaya (Caricaceae) and two closely related species \citep{Wang2012, Lovene2015}, {\itshape Mercurialis annua} (\citealt{VeltsosPannell2019}), the genus {\itshape Populus}  (Salicaceae) \citep[reviewed in][]{HobzaEtAl2018}, and several fishes including African cichlids \citep{GammerdingerKocher2018} and yellowtail \citep{KoyamaEtAl2015}. \hl{Mention Anther smut mating type chromosomes here too?}. Moreover, inversions appear to be involved in the evolution of sex-linked genome regions in several of these species (but see recent work on {\itshape Salix}; \citealt{AlmeidaMank2019}). 

Our findings suggest that evolutionary strata lengths can provide indirect evidence of how selection has influenced the evolution of SLR-expanding inversions (or other recombination modifiers) in  systems like these. Clearly it is not possible to observe a distribution of evolutionary strata lengths for single species. Moreover, subsequent sequence evolution within a newly expanded SLR, including deletions, duplications, and the accumulation of transposable elements will distort comparisons. Nevertheless, the observed length of relatively undegraded evolutionary strata should often provide different levels of support for neutral vs.~selection scenarios: large evolutionary strata being more consistent with the fixation of inversions driven by capture of previously unlinked SA loci or unconditionally beneficial inversions, while small strata (possibly including gene-by-gene recombination suppression or gradual expansion of the SLR; e.g., \citealt{BergeroQiuCharlesworth2013, QiuBergeroCharlesworth2015}), is more consistent with scenarios involving neutrality, or the capture of already tightly linked SA loci. Given recent results indicating the role of genome-wide heterochiasmy in restricting recombination between sex chromosomes, coupled with within-lineage variation in the extent of recombination suppression in {\itshape Poecillid} fish and some frogs \citep[e.g.,][]{Wright2017, BergeroCharlesworth2019, DaroltiWrightMank2019, FurmanEvans2018}, it would be very interesting to critically examine parallels between heterochiasmy and the fixation of large inversions expanding the SLR.

The second major implication of our models is that physical characteristics of recombining sex chromosomes, and especially the location of the ancestral SLR, can have a strong effect on the evolution of suppressed recombination. This is a crucial difference between the process of recombination suppression between sex chromosomes, and the fixation of inversions on autosomes \citep{ConnallonOlito2021}. The effect of SLR location on the likelihood of forming different sized evolutionary strata emerges from the geometry of a functionally two dimensional chromosome arm. The resulting predictions suggest that considering physical characteristics of recombining sex chromosomes could shed light on several outstanding questions (reviewed in \citealt{Charlesworth2016, Charlesworth2017}), such as why large sex-linked regions or heteromorphic sex chromosomes have evolved in some lineages and not others, and how many recombination suppression events are involved and why this varies among lineages? Overall, our models suggest that considering the physical processes involved in recombination suppression may offer new insights into why and how restricted recombination does or does not evolve in different lineages.% than seeking evidence of past bouts of sexually antagonistic selection.

Although we have modelled the effect of SLR location explicitly, other physical characteristics of recombining sex chromosomes not included in our models also influence the process of recombination suppression. For example, it is well known that the rate of recombination at different locations along chromosomes -- the 'recombination landscape' -- can be highly variable within and among species, and that marked differences often exist between males and females \citep[reviewed in ][]{SinghalEtAl2015, SardellKirkpatrick2020}. It has also been suggested that new sex determining genes may be more likely to recruit to genome regions with already low recombination rates \citep{Charlesworth1978a,vanDoornKirkpatrick2007, vanDoornKirkpatrick2010, OttoScottOsmond2018, Charlesworth2015, Olito2019}. For example, this appears to be the case for {\itshape Rumex hastatulus} and Papaya relatives \citep{RifkinBarrettWright2020,Lovene2015}. Moreover, classical theory predicts that low recombination rates are favourable for the maintenance of sexually antagonistic polymorphism \citep{Charlesworth1978a,Olito2017,Olito2019,Charlesworth2018}. If these regions of low recombination are more likely to occur at certain locations along the chromosome arm, the possible locations of the SLR may be constrained, thereby influencing whether further recombination suppression will involve small vs.~large evolutionary strata. Given that recombination is often lower in genome regions surrounding the centromere \citep[e.g.,][]{MahtaniWillard1998,SardellKirkpatrick2020}, it would be interesting to examine how our predictions, which are limited to paracentric inversions, might change when inversions suppressing recombination are pericentric.

%There is perhaps a parallel between the evolution of divergence between sex chromosomes parallels the genomics of speciation. Early genomic analysis of hybrid species pairs suggested the existence of "genomic islands of speciation" -- restricted regions with high genetic differentiation between species -- which were speculated to contribute to adaptation and reproductive isolation \citep[e.g.,][]{Ellegren2012}. Although apparent genomic islands of divergence have been identified \citep{TavaresEtAl2018}, a number of early analyses were later shown to provide inadequate control for confounding factors such as variable levels of genetic diversity across the genome or variation in recombination rate \citep{NoorBennett2009, WolfEllegren2017}. Consequently, regions of high divergence were often erroneously ascribed to selection rather than neutral or structural factors. Both this example and the results of our models suggest that caution is warranted when inferring causation with respect to genomic differentiation, and that selective explanations, although intuitively appealing, may not always be the most parsimonious.

Finally, our results show that the shape of the distribution of new inversion lengths (e.g., random breakpoint vs.~exponential) can weaken or exaggerate differences between selection scenarios in the expected length distributions of evolutionary strata. Although little is known about the distribution of new inversion lengths (limited data from {\itshape Drosophila} mutagenesis experiments are roughly consistent with a random breakpoint model; \citealt{KrimbasPowell1992}), it will be determined, at least in part, by other physical aspects of proto sex chromosome structure, such as the density and physical location of gene duplications, chromatin structure, transposable elements (TEs) and other repetitive sequences, which create hotspots for inversion breakpoints and DNA replication errors \citep[e.g.,][]{Charlesworth1994, PevznerTesler2003, PengPevznerTesler2006, LeeBatzer2008}. Indeed, the spatial distribution of these structural features of sex chromosomes will contribute jointly to determine the whether and how expanded non-recombining regions on sex chromosomes evolve. The interaction between physical and seletive processes driving the evolution of recombination suppression between sex chromosomes offers a variety of future directions for theoretical and empirical research.



%%%%%%%%%%%%%%%%%%%%%%%%
%\subsection*{Author Contributions}
%C.O. conceived the study, developed the models, and performed the analyses. Both C.O. and J.K.A. wrote and critically revised the manuscript.


%%%%%%%%%%%%%%%%%%%%%%%%
%\subsection*{Acknowledgements}
%This research was supported by a Wenner-Gren Postdoctoral Fellowship to C.O., and ERC-StG-2015-678148 to J.K.A. The study benefitted many detailed discussions and constructive feedback from T.~Connallon, C.Y.~Jordan, C.~Venables, H.~Papoli, the SexGen research group at Lund University, the editor, and two anonymous reviewers.

%%%%%%%%%%%%%%%%%%%%%%%%
%\subsection*{Data Accessibility Statement}
%Computer code needed to reproduce the simulations and main figures is available on GitHub (\url{https://github.com/colin-olito/inversionSize-ProtoSexChrom}) and a version of record is archived on Zenodo (doi: \hl{XXX}).






%%%%%%%%%%%%%%%%%%%%%%%%
%\subsection*{Supplementary Materials}
%Requests for supplementary material and correspondence can be directed to C.O. (\url{colin.olito@gmail.com}).


%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%
\bibliography{bibliography-inversionSize-ProtoSexChrom}

\newpage


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%  Tables 

\begin{table}[htbp]
\caption{\bf Definition of terms and parameters.}
\begin{tabu}to \linewidth{l X}
\toprule
\multicolumn{2}{l}{{\itshape Key terms}} \\
\midrule
$x$ & Inversion size, expressed as fraction of the chromosome that it spans ($0 < x < 1$). \\
$n$ & Number of selected loci captured by new inversion. \\
$U$ & Chromosome-wide deleterious mutation rate ($0 < U_d$). \\
$s_{d}$ & Selection coefficient for deleterious mutations ($s_d = 0.01$ for all analyses). \\
$h_{d}$ & Dominance coefficient for deleterious mutations ($h_d = 0.25$ for all analyses). \\
$N$, $N_f$, $N_m$ & Census, and breeding male and female population sizes, respectively. \\
$N_Y$, $N_X$, & Effective population size for Y- and X-linked genes, respectively. \\
$s_b$ & Heterozygous election coefficient for an unconditionally beneficial inversion ($s_b = 0.02$). \\
$s_f,s_m$ & Sex specific selection coefficients for SA loci ($s_f = s_m = 0.05$). \\
$h_f,h_m$ & Sex specific dominance coefficients for SA loci ($h_f = h_m = 0.5$). \\
$s_I$ & Overall fitness effect for a new inversion capturing SA loci ($0 < s \ll 1$). \\
$A$ & Expected number of sexually antagonistic loci on sex chromosomes. \\
$P$ & Length of PAR region in linkage disequilibrium with SLR, expressed as fraction of total chromosome length. \\
$\lambda$ & Rate parameter for the exponential model of new inversion lengths. \\
\addlinespace
\multicolumn{2}{l}{{\itshape Deterministic 2-locus model for SA selection scenario}} \\
\midrule
$w^{f}_{ii}$, $w^{m}_{ii}$ & diploid fitness terms for each genotype in females and males. \\
$r_{SA}$ & Ancestral recombination rate between SLR and SA locus. \\
$\lambda_I$ & Leading eigenvalue associated with invasion of rare inversion genotype. \\
$\hat{q}$ & Equilibrium frequency of male-beneficial sexually antagonistic allele (when $r_{SA} = 1/2$). \\
$\hat{X}_f$, $\hat{X}_m$, $\hat{Y}$ & Equilibrium frequency of male-beneficial sexually antagonistic allele on X chromosomes in males and females, and Y chromosomes (when $r_{SA} < 1/2$). \\
\addlinespace
\multicolumn{2}{l}{{\itshape Probability of expanding SLR}} \\
\midrule
$\text{SLR}_{\text{pos}}$ & Location of the SLR on the chromosome arm, expressed as a proportion of the distance between the centromere and distal chromosome end ($0 \leq \text{SLR}_{\text{pos}} \leq 1$). \\
$y_1$, $y_2$, $y_3$  & Proportion of total length of chromosome arm falling between the centromere and SLR, spanned by the ancestral SLR, and between the SLR and distal end, respectively (we assume $y_2 \approx 0$, so that $y_1 = \text{SLR}_{\text{pos}}$). \\
\addlinespace
\bottomrule
\end{tabu}
\label{tab:Parameters}\\
\end{table}
\newpage{}


% \begin{table}[htbp]
% \centering
% \caption{\bf Fitness expressions for models of Indirect Selection.}
% \begin{tabu}to 10.5cm {X[1,l] X[2,l] X[2,l] X[2,l]}
% \toprule
% \multicolumn{4}{l}{{\textit{Sexually antagonistic selection}}} \\
% \midrule
%	Females: & $w^{f}_{11} = 1$ & $w^{f}_{12} = 1 - h_f s_f$ & $w^{f}_{22} = 1 - s_f$ \\
%	Males: & $w^{m}_{11} = 1 - s_m$ & $w^{m}_{12} = 1 - h_m s_m$ & $w^{m}_{22} = 1$ \\
% \addlinespace
% \multicolumn{3}{l}{{\textit{Ploidally-antagonistic selection}}} \\
% \midrule
 	%Diploid: & $w^{\text{sex}}_{11} = 1 - s$ & $w^{\text{sex}}_{12} = 1 - s/2$ & $w^{\text{sex}}_{22} = 1$ \\
%%	Females: & $w^{f}_{11} = 1 - s$ & $w^{f}_{12} = 1 - s/2$ & $w^{f}_{22} = 1$ \\
%%	Males:   & $w^{m}_{11} = 1 - s$ & $w^{m}_{12} = 1 - s/2$ & $w^{m}_{22} = 1$ \\
% \addlinespace
% \end{tabu}
% \begin{tabu}to 10.5cm {X[1,l] X[2,l] X[2,c] X[2,l]}
% 	Haploid: & $v^{\text{sex}}_{1} = 1$ & -- & $v^{\text{sex}}_{2} = 1 - t$ \\
%%	Female gametes: & $v^{f}_{1} = 1$ & $v^{f}_{2} = 1 - t$ \\
%%	Male gametes:   & $v^{m}_{1} = 1$ & $v^{m}_{2} = 1$ \\
% \bottomrule
% \end{tabu}
% \begin{tabu}to 10.5cm {X[1,l]}
% {\footnotesize Where $\text{sex} \in \{m,f\}$.}
% \end{tabu}
% \label{tab:fitness}\\
% \end{table}
% \newpage{}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%  Figures 




\end{document}
