%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Preamble
\documentclass{article}

%Dependencies
\usepackage[left]{lineno}
\usepackage{titlesec}
\usepackage{xcolor}

\newcommand\hl[1]{%
  \bgroup
  \hskip0pt\color{blue!80!black}%
  #1%
  \egroup
}
\usepackage{ogonek}
\usepackage{float}
%\usepackage{charter}
%\usepackage[charter]{mathdesign}
\usepackage{amsmath}
\usepackage{old-arrows}
\usepackage{enumitem}
\usepackage{wasysym}

% Other Packages
%\usepackage{times}
\RequirePackage{fullpage}
\linespread{1.5}
\RequirePackage[colorlinks=true, allcolors=black]{hyperref}
\RequirePackage[english]{babel}
\RequirePackage{amsmath,amsfonts,amssymb}
\RequirePackage[sc]{mathpazo}
\RequirePackage[T1]{fontenc}
\RequirePackage{url}
\usepackage{tabu}

% Bibliography
%\usepackage[authoryear,sectionbib,sort]{natbib}
\usepackage{natbib} \bibpunct{(}{)}{;}{author-year}{}{,}
\bibliographystyle{genetics}
\addto{\captionsenglish}{\renewcommand{\refname}{Literature Cited}}
\setlength{\bibsep}{0.0pt}

% Graphics package
\usepackage{graphicx}
\graphicspath{{../../output/figures/}.pdf}

% Change Appendix Numbering
\renewcommand\thesection{Appendix \Alph{section}}
\renewcommand\thesubsection{\Alph{section}.\arabic{subsection}}

% New commands: fonts


%\newcommand{\code}{\fontfamily{pcr}\selectfont}
%\newcommand*\chem[1]{\ensuremath{\mathrm{#1}}}
\newcommand\numberthis{\addtocounter{equation}{1}\tag{\theequation}}
%\titleformat{\subsubsection}[runin]{\bfseries\itshape}{\thesubsubsection.}{0.5em}{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\title{Supplementary Materials (Appendices A -- D) for: The evolution of suppressed recombination between sex chromosomes by chromosomal inversions. \textit{Genetics}}

\author{Colin Olito$^{\ast}$ \& Jessica K.~Abbott}
\date{\today}

\begin{document}
\maketitle

\noindent{} Department of Biology, Section for Evolutionary Ecology, Lund University, Lund 223 62, Sweden.

\noindent{} $^{\ast}$ Corresponding author e-mail: \url{colin.olito@gmail.com}
\bigskip

\noindent{} Additional R code and supplementary info available at: \url{https://github.com/colin-olito?tab=repositories}

\bigskip

\newpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Running Header
%\pagestyle{fancyplain}
%\makeatother
%\lhead{\textit{Supplement to Olito. Linkage and SA polymorphism in hermaphrodites. \textit{Evolution}.\\}}
%%\rhead{\textit{Sex antagonistic selection on phenology}}
%\renewcommand{\headrulewidth}{0pt}
%\renewcommand{\footrulewidth}{0pt}
%\addtolength{\headheight}{12pt}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Neutural inversions}\label{AppA}
\renewcommand{\theequation}{A\arabic{equation}}
\titleformat{\subsubsection}    
{\normalfont\fontsize{12pt}{17}\itshape}{\thesubsubsection}{12pt}{}
\renewcommand{\thetable}{A\arabic{table}}

%%%%%%%%%%%%
\subsection{Fixation Probability}

As described in the the main article, our models make several key assumptions, including ({\itshape i}) that deleterious mutations are at mutation-selection balance at loci on the sex chromosomes that are unlinked with the sex determining region (SDR) and one another; ({\itshape ii}) that new inversions are unlikely to fix unless they are initially free of deleterious mutations; and ({\itshape ii}) that the timescale for inversion fixation is shorter than that of gene loss/degeneration or dosage compensation due to recombination suppression between sex chromosomes (i.e., both inversion and non-inversion X and Y chromosomes have functional homologs for the same genes). Under these assumptions, a mutation-free inversion will enjoy a temporary fitness advantage over the average non-inverted chromosome in the population. Over time, however, mutations will accumulate on the inversion chromosomes until they too are at equilibrium, equalizing the fitness of inverted and non-inverted chromosomes. Any approximation of the fixation probability for neutral inversions that are initially free of deleterious mutations must therefore take into account how frequent inversion descendants would become in the population before they become selectivley neutral.

Prior to any recombination suppression between sex chromosomes, the dynamics of deleterious mutations at loci unlinked with the SDR are identical to those at autosomal loci. Hence, the equilibrium frequency of deleterious alleles at each locus will be $\hat{p} \approx \frac{\mu}{h_d s_d}$, where $\mu$ is the single-locus deleterious mutation rate. Following \citet{Nei1967} (their Eq[7] and Eq[8]), the general solution for the approximate frequency of deleterious mutations over time at a given locus is

\begin{equation}\label{eq:delMutAccum}
	p_t = \hat{p}(1 - e^{-h_d s_d}).
\end{equation}

\noindent The overall mutation rate in the chromosomal segment spanned by an inversion is equal to $U_d x = n \mu$, where $n$ is the number of loci spanned by the inversion. 


%%%%%%%%%%%%
\subsubsection*{Inversions spanning the SDR on the Y chromosome}

From Eq(\ref{eq:delMutAccum}), we can write the fitness of individuals carrying inversions spanning the SDR on a Y chromosome as follows:

 \begin{table}[htbp]\label{tab:NeutralYinvFitTab}
 \centering
 \caption{\bf Fitness of inversion and non-inversion genotypes (Y chromosome).}
 \begin{tabu}to 11cm {X[1,l] X[1,l] X[1,l]} \hline
 Generation & $X \mid Y_I$ & $X \mid Y$ \\
 \hline 
 $0$ & $e^{-U_d x}$ & $e^{-2 U_d x}$ \\
 $t$ & $e^{-U_d x(2 - e^{-h_d s_d t})}$ & $e^{-2 U_d x}$ \\
 $t$ (relative to $X \mid Y_I$) & 1 & $e^{- U_d x e^{-h_d s_d t}}$ \\
 \hline
 \multicolumn{3}{p{3.3\tabucolX}}{{\footnotesize Note: Fitness expressions are given for the $0^{\text{th}}$ and $t^{\text{th}}$ generation when an inversion capturing the SDR on a Y chromosome is initially free of deleterious mutations. Note also that these fitness expressions are approximate for $h_d \approx 1/2$.}}
 \end{tabu}
 \end{table}
 \newpage{}

\noindent Given these time-dependent fitness expressions, we can write the discrete time recursion for the frequency ($q$) of inversion descendents at time $t + 1$ as:

\begin{equation}\label{eq:NeutralYinvRec}
	q_{t+1} = \frac{ q_t e^{-U_d x(2 - e^{-h_d s_d t})} }{(1 - q_t) e^{-2 U_d x} + q_t e^{-U_d x(2 - e^{-h_d s_d t})}},
\end{equation} 

\noindent or more simply:

\begin{equation}\label{eq:NeutralYinvRecRatio}
	\frac{q_{t+1}}{1- q_{t+1}} = \frac{q_{t}}{1- q_{t}} \text{Exp}[U_d x e^{-h_d s_d t}].
\end{equation} 

\noindent The general solution to Eq(\ref{eq:NeutralYinvRecRatio}) is 

\begin{equation}\label{eq:NeutralYinvGenSol}
	q_{t} = \frac{q_{0} \, \text{Exp}\Bigg[ U_d x \frac{(1 - e^{-h_d s_d t})}{(1 - e^{-h_d s_d})} \Bigg]} {1 - q_{0} + q_0 \, \text{Exp}\Bigg[ U_d x \frac{(1 - e^{-h_d s_d t})}{(1 - e^{-h_d s_d})} \Bigg]}.
\end{equation} 

\noindent \citep[see][]{ConnallonOlito2020}. In the limit of $t \rightarrow \infty$, the frequency of inversion descendents will converge to 

\begin{equation}\label{eq:NeutralYinvGenSolLimit}
	q_{t} = \frac{q_{0} \, \text{Exp}\Big[ \frac{U_d x}{1 - e^{-h_d s_d} } \Big]} {1 - q_{0} + q_0 \, \text{Exp}\Big[ \frac{U_d x}{1 - e^{-h_d s_d} } \Big]}.
\end{equation} 

\noindent When $N_Y$ is large, a single-copy of an initially deleterious mutation-free neutral inversion will reach an 'effective frequency' of

\begin{align*}\label{eq:NeutralYinvQEff}
	q^Y_{\text{eff}} & \approx N^{-1}_Y \, \text{Exp}\Big[ \frac{U_d x}{1 - e^{-h_d s_d} } \Big] \\
	 &\approx N^{-1}_Y \, \text{Exp}\Big[ \frac{U_d x}{h_d s_d} \Big] \numberthis,
\end{align*} 

\noindent after which it will evolve neutrally. 

Multiplying $q_{\text{eff}}$ by the probability that the inversion is initially free of deleterious mutations, $\Pr(k = 0 \mid x) = e^{-\frac{U_d x}{s_d}}$, gives the overall fixation probability presented in Eq(\hl{2}a) of the main text:

\begin{align*}\label{eq:NeutralYinvPFix}
		\Pr(\text{fix} \mid x, k = 0) &\approx N^{-1}_Y \, \text{Exp}\Big[ \frac{U_d x}{h_d s_d} \Big] e^{\frac{-U_d x}{h_d s_d}} \\
		 &= N^{-1}_Y , \numberthis
\end{align*}

Overall, the fact that larger inversions are less likely to be initially free of deleterious mutations offsets the greater temporary fitness benefit of actually being mutation-free. The fixation probability for neutral inversions remains approximately unchanged in the presence of deleterious mutation pressure that is approximately equal to $q_0 = N^{-1}_Y$. increasing risk of catching overall fixation probability of an initially mutation-free neutral inversion spanning the SDR on a Y chromosome is approximately equal to $q^Y_{\text{eff}}$, as presented in Eq(\hl{2}a) of the main text.



%%%%%%%%%%%%%%%
\subsubsection*{Inversions spanning the SDR on an X chromosome}

The frequency dynamics of initially mutation-free inversions spanning the SDR on the X chromosome turn out to be nearly identical to both autosomal inversions and those on the Y chromsome. Here we briefly outline the one key difference between the results provided for inversions on the X vs.~Y chromosomes. 

We can write the fitness of individuals carrying inversions spanning the SDR on an X chromosome from Eq(\ref{eq:delMutAccum}) as follows:

 \begin{table}[htbp]\label{tab:NeutralXinvFitTab}
 \centering
 \caption{\bf Fitness of inversion and non-inversion genotypes (X chromosome).}
 \begin{tabu}to 14cm {X[1,l] X[1,l] X[1,l] X[1,l]} \hline
 \multicolumn{4}{l}{\bf{ Females}} \\
 Generation & $X_I \mid X_I$ & $X \mid X_I$ & $X \mid X$ \\
 \hline 
 $0$ & $1$ & $e^{-U_d x}$ & $e^{-2 U_d x}$ \\
 $t$ & $e^{-2 U_d x(1 - e^{-h_d s_d t})}$ & $e^{-U_d x(2 - e^{-h_d s_d t})}$ & $e^{-2 U_d x}$ \\
 $t$ (relative to $X_I \mid X_I$) & 1 & $e^{-U_d x e^{-h_d s_d t}}$ & $e^{-2 U_d x e^{-h_d s_d t}}$ \\
 $t$ (approx.) & 1 & $1 - U_d x e^{-h_d s_d t}$ & $1 - 2 U_d x e^{-h_d s_d t}$ \\
 \hline
 \multicolumn{4}{l}{\bf{Males}} \\
 Generation & $X_I \mid Y$ & $X \mid Y$ & \\
 \hline 
 $0$ & $e^{-U_d x}$ & $e^{-2 U_d x}$ & \\
 $t$ & $e^{-U_d x(2 - e^{-h_d s_d t})}$ & $e^{-2 U_d x}$ & \\
 $t$ (relative to $X \mid Y_I$) & 1 & $e^{-2 U_d x e^{-h_d s_d t}}$ & \\
 $t$ (approx.) & 1 & $1 - U_d x e^{-h_d s_d t}$ & \\
 \hline
 \multicolumn{4}{p{4.4\tabucolX}}{{\footnotesize Note: Fitness expressions are given for the $0^{\text{th}}$ and $t^{\text{th}}$ generation when an inversion capturing the SDR on an X chromosome is initially free of deleterious mutations. Note also that these fitness expressions are approximate for $h_d \approx 1/2$.}}
 \end{tabu}
 \end{table}

\noindent Unlike the model for inversions on the Y chromosome, there is no general solution to the discrete time recursion for the frequency of inversion descendents when the inversion spans the SDR on an X chromosome. Nevertheless, we can use a continuous time approximation to the general solution for the discrete time model. Under weak selection and mutation, the rate of change in the frequency of inversion descendents is roughly 

\begin{equation}\label{eq:NeutralXinvDiffEq}
	\frac{\partial q_t}{\partial t} = q_t(1 - q_t)U_d x \, e^{-h_d s_d t} + O(U_d^2).
\end{equation}

\noindent The general solution to Eq(\ref{eq:NeutralXinvDiffEq}) is

\begin{equation}\label{eq:NeutralXinvGenSol}
	q_{t} = \frac{q_{0} \, \text{Exp}\big[ \frac{U_d x(1 - e^{-h_d s_d t})}{h_d s_d } \big]} {1 - q_{0} + q_0 \, \text{Exp} \big[ \frac{U_d x(1 - e^{-h_d s_d t})}{h_d s_d } \big]}.
\end{equation} 

\noindent In the long term, the frequency of inversion descendents converges deterministically on 

\begin{equation}\label{eq:NeutralXinvqEff}
	q_{t} = \frac{q_{0} \, e^{\frac{U_d x}{h_d s_d }}} {1 - q_{0} + q_0 \, e^{\frac{U_d x}{h_d s_d }}}.
\end{equation} 

\noindent Finally, when $N_X$ is large, a single-copy of an initially deleterious mutation-free neutral inversion will reach an 'effective frequency' of

\begin{equation}\label{eq:NeutralXinvQEff}
	q^X_{\text{eff}} \approx N^{-1}_X \, e^{\frac{U_d x}{h_d s_d }},
\end{equation} 

\noindent after which it will evolve neutrally. $q^X_{\text{eff}}$ therefore provides an approximation for the fixation probability of an initially mutation-free neutral inversion spanning the SDR on an X chromosome. 

Multiplying $q_eff$ by the probability that the inversion is initially free of deleterious mutations, $\Pr(k = 0 \mid x) = e^{-\frac{U_d x}{s_d}}$, gives the overall fixation probability presented in Eq(\hl{2}b) of the main text:

\begin{align*}\label{eq:NeutralXinvPFix}
		\Pr(\text{fix} \mid x, k = 0) &\approx N^{-1}_X \, e^{\frac{U_d x}{h_d s_d}} e^{\frac{-U_d x}{h_d s_d}} \\
		&= N^{-1}_X, \numberthis
\end{align*}

Comparing Eq(\ref{eq:NeutralYinvPFix}) with Eq(\ref{eq:NeutralXinvPFix}) for equivalent effective population sizes shows the general solution for the continuous time model provides a good approximation to the discrete-time model, but works best when $U_d$, $s_d$, and $x$ are all small. 

\newpage{}





%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Sexual Antagonism}\label{AppB}
 \renewcommand{\theequation}{B\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{B\arabic{figure}}
 \setcounter{figure}{0}

In the main text we present analyses of simplified two-locus models for two scenarios: ({\itshape iii}) sexual antagonism and ({\itshape iv}) haploid and diploid selection (roman numerals from the main text). Each of these models represent modest extensions of earlier population genetic models of the PAR including \citet{Clark1987}, \citet{Otto2011}, and \citet{Otto2014, Otto2019}. We refer readers to these papers for comprehensive theoretical background on two-locus PAR models. 

We use the two-locus models to examine the invasion conditions for rare inversions spanning the SDR and other selected loci, and to approximate the overall selection coefficient for inversions, taking into account the possible effects of partial linkage between the SDR and the selected locus prior to the inversion mutation. The recursions for each of these models are structurally alike, and the evolutionary invasion analyses follow similar steps. Here, we develop the recursions for scenario ({\itshape iii}) sexual antagonism, and a detailed description of the invasion analysis. Details for the other models are presented in the Mathematica notebook files (.nb) available in the Online Supplementary Materials.


%%%%%%%%%%%%
\subsection{Recursions (Y chromosome inversion)}

Consider the two-locus genetic system described in the main text: one locus determines whether a chromosome is considered X or Y, with XX individuals being female, and XY individuals being male (YY are considered lethal); and a second locus having alleles $A$ and $a$ that may be linked to the sex-determining region and is subject to natural selection. Recombination between the two loci occurs at a rate $r$ per meiosis. Generations are discrete, and the population size is assumed to be large enough that drift is negligible. The life cycle proceeds: haploid selection $\rightarrow$ random mating $\rightarrow$ diploid selection.

When studying the invasion of an inversion capturing the SDR and one allele at a selected locus on the Y chromosome, there are three relevant female genotypes: $AA$, $Aa$, and $aa$, with frequencies denoted $x_1$, $x_2$, and $x_3$, and general fitness expressions at selection denoted $w_{f,1}$, $w_{f,2}$, $w_{f,3}$. However, there are six relevant genotypes for males: $AA$, $Aa$ (cis-), $Aa^I$ (cis-), $aA$ (trans-), $aa$, and $aa^I$, with frequencies $y_{1}$, $y_{2c}$, $y^I_{2c}$, $y_{2t}$, $y_{3}$, $y_{3I}$, but fitness expressions $w_{m,1}$, $w_{m,2}$, $w_{m,3}$. Note that male heterozygote genotype labels indicate whether the $A$ allele is located on the X chromosome ($y_{2c}$), or on the Y chromosome ($y_{2t}$), and "$I$" superscripts denote inverted haplotypes. It is assumed that recombination is suppressed between inverted and non-inverted chromosomes.

\begin{equation*}
	\text{Female genotypes}:\left( \begin{array}{cc|c}
		x_1: & XA & XA \\
		x_2: & XA & Xa \\
		x_3: & Xa & Xa 
	\end{array} \right)
\end{equation*}

\begin{equation*}
	\text{Male genotypes}:\left( \begin{array}{cc|c}
		y_1:     & XA & YA \\
		y_{2c}:   & XA & Ya \\
		y^I_{2c}: & XA & Ya^I \\
		y_{2t}:   & Xa & YA \\
		y_{3}:    & Xa & Ya \\
		y^I_{3}:  & Xa & Ya^I 
	\end{array} \right)
\end{equation*}

\noindent The genotypic frequencies among females after recombination and random mating are:
\begin{align*}
	x^m_{1} &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^m_{2} &= \Big( x_1 + \frac{x_2}{2} \Big) \big( y_{2c} \cdot r + y_{2t}(1 - r) + y_3 + y^I_3 \big)~+ \\
	&~~~~\Big( x_3 + \frac{x_2}{2} \Big) \big( y_{1} + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big)      \\
	x^m_{3} &= \Big( x_3 + \frac{x_2}{2} \Big) \big(y_{2c} \cdot r + y{2t}(1 - r) + y_{3} + y^I_3  \big) \numberthis
\end{align*}

\noindent and among males:
\begin{align*}
	y^m_{1}      &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y_1 + y_{2c} \cdot r + y_{2t}(1 - r)  \big) \\
	y^m_{2c}     &= \Big( x_1 + \frac{x_2}{2} \Big) \big( y_{2c} (1 - r) + y_{2t} \cdot r + y_3 \big) \\
	y^{I,m}_{2c} &= \Big( x_1 + \frac{x_2}{2} \Big) \big(y^I_{2c} + y^I_{3} \big)  \\
	y^m_{2t}     &= \Big( x_3 + \frac{x_2}{2} \Big) \big( y_{2c} \cdot r + y_{2t}(1 - r) + y_1 \\
	y^m_{3}      &= \Big( x_3 + \frac{x_2}{2} \Big) \big( y_{2c}(1 - r) + y_{2t} \cdot r + y_3 \\
	y^{I,m}_{3}  &= \Big( x_3 + \frac{x_2}{2} \Big) \big(y^I_{2c} + y^I_{3} \big)  \numberthis
\end{align*}

\noindent After selection, the genotypic frequencies among females will be:
\begin{align*}
	x'_{1} &= \frac{x^m_1 w_{f,1}}{\overline{w}_{f}}\\
	x'_{2} &= \frac{x^m_2 w_{f,2}}{\overline{w}_{f}}\\
	x'_{3} &= \frac{x^m_3 w_{f,3}}{\overline{w}_{f}} \numberthis
\end{align*}

\noindent where $\overline{w}_{f} = x^m_1 w_{f,1} + x^m_2 w_{f,2} + x^m_3 w_{f,3}$. For males, the genotypic frequencies after selection are:
\begin{align*}
	y'_{1}       &= \frac{y^m_1 w_{m,1}}{\overline{w}_{m}} \\
	y'_{2c}      &= \frac{y^m_{2c} w_{m,2}}{\overline{w}_{m}} \\
	y'^{I}_{2c}  &= \frac{y^{I,m}_{2c} w_{m,2}}{\overline{w}_{m}}  \\
	y'_{2t}      &= \frac{y^m_{2t} w_{m,2}}{\overline{w}_{m}} \\
	y'_{3}       &= \frac{y^m_3 w_{m,3}}{\overline{w}_{m}} \\
	y'^{I}_{3}   &= \frac{y^{I,m}_3 w_{m,3}}{\overline{w}_{m}} \numberthis
\end{align*}

\noindent where $\overline{w}_{m} = y^m_1 w_{m,1} + y^m_{2c} w_{m,2} + y^{I,m}_{2c} w_{m,2} + y^m_{2t} w_{m,2} + y^m_3 w_{m,3} + y^{I,m}_3 w_{m,3}$.



%%%%%%%%%%%%
\subsection{Eigenvalues (Y chromosome inversion)}

Under the assumptions stated in the main article, a new inversion must capture the SDR and a male-beneficial allele at the SA locus in order to invade. Accordingly, we use SA fitness expressions {\itshape sensu} \citealt{Kidwell1977,ConnallonClark2012,Otto2011}), where $w_{f,1} = 1$, $w_{f,2} = 1 - h_f s_f$, and $w_{f,3} = 1 - s_f$, and $w_{m,1} = 1 - s_m$, $w_{m,2} = 1 - h_m s_m$, and $w_{m,3} = 1$. We are interested in the evolutionary fate of rare inversion genotypes capturing the $a$ allele at the SA locus.

Substituting $x_3 = 1 - x_1 - x_2$ and $y_3 = 1 - y_1 - y_{2c} - y^I_{2c} - y_{2t} - y^I_3$, yields a system of $7$ genotypic frequency recursions. A key assumption in the analysis is that all non-inversion genotypes are initially at equilibrium when a new inversion mutation occurs. Specifically, we assume that $x_i = \hat{x}_i$ for $i \in \{1,\,2 \}$ and $y_j = \hat{y}_j$ where $j \in \{ 1,\,2c,\,2t \}$. To solve for the conditions under which a rare inversion capturing the SDR and the male-beneficial allele at the SA locus, we evaluate the Jacobian at the boundary equilibrium where the initial frequency of inverted genotypes are $y^{I}_{2c} = y^{I}_{3} = 0$:

\begin{equation*}
	\mathbb{J} = \left( \begin{array}{ccccccc}

					\frac{\partial x'_1}{\partial x_1} &
					\frac{\partial x'_2}{\partial x_1} &
					\frac{\partial y'_1}{\partial x_{1}} &
					\frac{\partial y'_{2c}}{\partial x_1} &
					\frac{\partial y'^I_{2c}}{\partial x_1} &
					\frac{\partial y'_{2t}}{\partial x_1} &
					\frac{\partial y'^I_{3}}{\partial x_1} \\[1ex]

					\frac{\partial x'_1}{\partial x_2} &
					\frac{\partial x'_2}{\partial x_2} &
					\frac{\partial y'_1}{\partial x_{2}} &
					\frac{\partial y'_{2c}}{\partial x_2} &
					\frac{\partial y'^I_{2c}}{\partial x_2} &
					\frac{\partial y'_{2t}}{\partial x_2} &
					\frac{\partial y'^I_{3}}{\partial x_2} \\[1ex]

					\frac{\partial x'_1}{\partial y_1} &
					\frac{\partial x'_2}{\partial y_1} &
					\frac{\partial y'_1}{\partial y_{1}} &
					\frac{\partial y'_{2c}}{\partial y_1} &
					\frac{\partial y'^I_{2c}}{\partial y_1} &
					\frac{\partial y'_{2t}}{\partial y_1} &
					\frac{\partial y'^I_{3}}{\partial y_1} \\[1ex]

					\frac{\partial x'_1}{\partial y_{2c}} &
					\frac{\partial x'_2}{\partial y_{2c}} &
					\frac{\partial y'_1}{\partial y_{2c}} &
					\frac{\partial y'_{2c}}{\partial y_{2c}} &
					\frac{\partial y'^I_{2c}}{\partial y_{2c}} &
					\frac{\partial y'_{2t}}{\partial y_{2c}} &
					\frac{\partial y'^I_{3}}{\partial y_{2c}} \\[1ex]

					\frac{\partial x'_1}{\partial y^I_{2c}} &
					\frac{\partial x'_2}{\partial y^I_{2c}} &
					\frac{\partial y'_1}{\partial y^I_{2c}} &
					\frac{\partial y'_{2c}}{\partial y^I_{2c}} &
					\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
					\frac{\partial y'_{2t}}{\partial y^I_{2c}} &
					\frac{\partial y'^I_{3}}{\partial y^I_{2c}} \\[1ex]

					\frac{\partial x'_1}{\partial y_{2t}} &
					\frac{\partial x'_2}{\partial y_{2t}} &
					\frac{\partial y'_1}{\partial y_{2t}} &
					\frac{\partial y'_{2c}}{\partial y_{2t}} &
					\frac{\partial y'^I_{2c}}{\partial y_{2t}} &
					\frac{\partial y'_{2t}}{\partial y_{2t}} &
					\frac{\partial y'^I_{3}}{\partial y_{2t}} \\[1ex]

					\frac{\partial x'_1}{\partial y^I_{3}} &
					\frac{\partial x'_2}{\partial y^I_{3}} &
					\frac{\partial y'_1}{\partial y^I_{3}} &
					\frac{\partial y'_{2c}}{\partial y^I_{3}} &
					\frac{\partial y'^I_{2c}}{\partial y^I_{3}} &
					\frac{\partial y'_{2t}}{\partial y^I_{3}} &
					\frac{\partial y'^I_{3}}{\partial y^I_{3}} \\[1ex]

				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										y'^I_{2c} = y^I_{3} = 0}} \numberthis
\end{equation*}

\noindent We can isolate the candidate leading eigenvalues associated with the spread of inversion genotypes by first rearranging the Jacobian using elementary row and column operations to give a block triangular matrix \cite[Supplement~to~Primer~2]{OttoDay2007}:
\begin{equation}
	\mathbb{J}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{A} = \left( \begin{array}{ccc} 
										\frac{\partial x'_1}{\partial x_1} & \cdots & \frac{\partial y'_{2t}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'_1}{\partial y_{2t}} & \cdots & \frac{\partial y'_{2t}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) &
				\mathbf{B} = \left( \begin{array}{cc} 
										\frac{\partial y'^I_{2c}}{\partial x_1} & \frac{\partial y'^I_{3}}{\partial x_1} \\[1ex]
										\vdots & \vdots \\[1ex]
										\frac{\partial y'^I_{2c}}{\partial y_{2t}} & \frac{\partial y'^I_{3}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{C} = \left( \begin{array}{ccc} 
								0 & \cdots & 0\\[1ex]
								0 & \cdots & 0\\[1ex]
							\end{array} \right) &
		\mathbf{D} = \left( \begin{array}{cc} 
								\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} & \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} \\[1ex]
								\frac{\partial y'^I_{2c}}{\partial y^I_{2c}} & \frac{\partial y'^I_{3}}{\partial y^I_{3}} \\[1ex]
							\end{array} \right) \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										y'^I_{2c} = y^I_{3} = 0}} \\[1ex]
\end{equation}

\noindent The eigenvalues of a block triangular matrix are the eigenvalues of the submatrices along the diagonal, which conveniently isolates the candidate leading eigenvalues for non-inversion genotypes (submatrix $\mathbf{A}$) and inversion genotypes (submatrix $\mathbf{D}$). When evaluated at $y'^I_{2c} = y^I_{3} = 0$, the two elements in the first row of submatrix $\mathbf{D}$ are identical ($ \alpha = \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} = \frac{\partial y'^I_{2c}}{\partial y^I_{3}}$), as are the two elements in the second row ($\beta = \frac{\partial y'^I_{3}}{\partial y^I_{2c}} = \frac{\partial y'^I_{3}}{\partial y^I_{3}}$). The candidate leading eigenvalue of submatrix $\mathbf{D}$ describing the spread of the rare inversion is:
\begin{equation}
	\lambda_{I} = \big( \alpha + \beta \big)
\end{equation}

\noindent $\lambda_I$ is cumbersome when expressed in terms of the adult genotypic frequencies. However, $\lambda_I$ can be transformed onto a new coordinate system of allele frequencies in the three chromosome classes among gametes: X chromosomes in ovules ($X_f$), X chromosomes in sperm ($X_m$), and Y chromosomes in sperm ($Y$) \citep{Clark1987,Otto2011,Otto2014}. $\lambda_I$ simplifies considerably when expressed on this new coordinate system:

\begin{equation}
	\lambda_I = \frac{w_{m,2}(1 - \hat{X}_f) + w_{m,3} \hat{X}_f} {w_{m,1}(1 - \hat{X}_f)(1 - \hat{Y}) + w_{m,3} \hat{X}_f \hat{Y} + w_{m,2} (\hat{X}_f + \hat{Y} - 2 \hat{X}_f \hat{Y})}
\end{equation}

\noindent If $\lambda_I > 1$, and all non-inversion genotypes are initially at equilibrium ($x_i = \hat{x}_i$, $y_j = \hat{y}_j$), as we have assumed, $\lambda_I$ will necessarily be the leading eigenvalue of the system of recursions. This is a strong assumption that does not necessarily hold once the system has been perturbed. We have therefore checked our use of $\lambda_I$ in subsequent approximations using both deterministic iteration of the recursions and Wright-Fisher simulations (see fig.~2 and fig.~5 in the main text). For inversions spanning the SDR on a Y chromosome, however, there are no internal equilibria; if the inversion can invade (i.e., if $\lambda_I > 1$), it will go to fixation. 

Substituting in the SA fitness expessions described above, and calculating $s_I \approx \lambda - 1$ provides an estimate of the overall selection coefficient for the rare inversion \cite{OttoYong2002}, which appears in the main text as \hl{Eq(8)}. Calculating $\lambda_I$ for $r = 1/2$ yields the approximation given in \hl{Eq(6)}.



%%%%%%%%%%%%
\subsection{Recursions (X chromosome inversion)}

The assumptions and development of the recursions for an inversion spanning the SDR and a single SA locus on an X chromosome largely parallel those for Y chromome inversions. We use the same SA fitness expressions described in the previous section. Now, our interest is focused on a rare inversion that captures the female-beneficial $A$ allele at the SA locus, and we have $6$ relevant genotypes for both males and females:

\begin{equation*}
	\text{Female genotypes}:\left( \begin{array}{cc|c}
		x_1:        & XA   & YA \\
		x^I_{1}:    & XA   & XA^I \\
		x^{II}_{1}: & XA^I & XA^I \\
		x_{2}:      & XA   & Xa \\
		x^I_{2}:    & XA^I & Xa \\
		x_{3}:      & Xa   & Xa 
	\end{array} \right)
\end{equation*}

\begin{equation*}
	\text{Male genotypes}:\left( \begin{array}{cc|c}
		y_1:      & XA   & YA \\
		y^I_{1}:  & XA^I & YA \\
		y_{2c}:   & XA   & Ya \\
		y^I_{2c}: & XA^I & Ya \\
		y_{2t}:   & Xa & YA \\
		y_{3}:    & Xa & Ya 
	\end{array} \right)
\end{equation*}


\noindent The genotypic frequencies among females after recombination and random mating are:
\begin{align*}
	x^m_{1}      &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{I,m}_{1}  &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big( y^I_{2c} + y^I_{2c} \big)~+ \\
				 &~~~~\Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{II,m}_{1} &= \Big( \frac{x^I_1}{2} + x^{II}_1 + \frac{x^I_2}{2} \Big) \big( y^I_{2c} + y^I_{2c} \big) \\
	x^m_2        &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big)~+ \\
				 &~~~~\Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_1 + y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r \big) \\
	x^{I,m}_2    &= \Big( \frac{x^I_1}{2} + x^{II}_{1} + \frac{x^I_2}{2} \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big)~+ \\
				 &~~~~\Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big( y^I_{2c} + y^I_{2c} \big) \\
	x^m_3        &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_{2c} \cdot r + y_{2t}(1 - r) + y_{3} \big) \numberthis
\end{align*}

\noindent and among males:
\begin{align*}
	y^m_{1}      &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{I,m}_{1}  &= \Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big)  \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{m}_{2c}   &= \Big( x_1 + \frac{x^I_1}{2} + \frac{x_2}{2} \Big) \big(y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big)  \\
	y^{I,m}_{2c} &= \Big( \frac{x^I_1}{2}~+ x^{II}_1 + \frac{x^I_2}{2} \Big) \big( y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big) \\
	y^m_{2t}     &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big(y_1 + y^I_1 + y_{2c} \cdot r + y_{2t}(1 - r) \big) \\
	y^{m}_{3}    &= \Big(\frac{x_2}{2} + \frac{x^I_2}{2} + x_3 \Big) \big( y_{2c}(1 - r) + y^I_{2c} + y_{2t} \cdot r + y_3 \big)  \numberthis
\end{align*}

\noindent After selection, the genotypic frequencies among females will be:
\begin{align*}
	x'_{1}     &= \frac{x^m_1 w_{f,1}     }{\overline{w}_{f}} \\
	x'^{I}_{1}  &= \frac{x^{I,m}_1 w_{f,1} }{\overline{w}_{f}} \\
	x'^{II}_{1} &= \frac{x^{II,m}_1 w_{f,1}}{\overline{w}_{f}} \\
	x'_{2}      &= \frac{x^m_2 w_{f,2}     }{\overline{w}_{f}} \\
	x'^{I}_{2}  &= \frac{x^{I,m}_2 w_{f,2} }{\overline{w}_{f}} \\
	x'_{3}      &= \frac{x^m_3 w_{f,3}     }{\overline{w}_{f}} \\
\end{align*}
    
\noindent where $\overline{w}_{f} = x^m_1 w_{f,1} + x^{I,m}_1 w_{f,1} + x^{II,m}_1 w_{f,1} + x^m_2 w_{f,2} + x^{I,m}_2 w_{f,2} + x^m_3 w_{f,3} $. The genotypic frequencies in males after selection are:
\begin{align*}
	y'_{1}       &= \frac{y^m_1 w_{m,1}       }{\overline{w}_{m}} \\
	y'^I_{1}     &= \frac{y^{I,m}_{1} w_{m,1} }{\overline{w}_{m}} \\
	y'_{2c}      &= \frac{y^{m}_{2c} w_{m,2}  }{\overline{w}_{m}}  \\
	y'^I_{2c}    &= \frac{y^{I,m}_{2c} w_{m,2}}{\overline{w}_{m}} \\
	y'_{2t}      &= \frac{y^m_{2t} w_{m,2}    }{\overline{w}_{m}} \\
	y'_{3}       &= \frac{y^{m}_3 w_{m,3}   }{\overline{w}_{m}} \numberthis
\end{align*}
  
\noindent where $\overline{w}_{m} = y^m_1 w_{f,1} + y^{I,m}_{1} w_{f,1} + y^{m}_{2c} w_{f,2} + y^{I,m}_{2c} w_{f,2} + y^m_{2t} w_{f,2} + y^{I,m}_3 w_{f,3} $.



%%%%%%%%%%%%
\subsection{Eigenvalues (X chromosome inversion)}

Again, our analysis for the model of X chromosome inversions largely paralleles that for the Y.

Substituting $x_3 = 1 - x_{1} - x^I_{1} - x^{II}_{1} - x_{2} - x^I_{2}$ and $y_3 = 1 - y_1 - y^I_1 - y_{2c} - y^I_{2c} - y_{2t}$, yields a system of $10$ genotypic frequency recursions. As before, we make the key assumption that all non-inversion genotypes are initially at equilibrium when a new inversion mutation occurs: $x_i = \hat{x}_i$ for $i \in \{1,\,2 \}$ and $y_j = \hat{y}_j$ where $j \in \{ 1,\,2c,\,2t \}$. To solve for the conditions under which a rare inversion capturing the SDR and the male-beneficial allele at the SA locus, we evaluate the Jacobian at the boundary equilibrium where the initial frequency of inverted genotypes are $x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0$:


\begin{equation*}
	\mathbb{J} = \left( \begin{array}{cccccccccc}

					\frac{\partial x'_1      }{\partial x_1} &
					 \frac{\partial x'^I_1   }{\partial x_1} &
					 \frac{\partial x'^{II}_1}{\partial x_1} &
					 \frac{\partial x'_{2}   }{\partial x_1} &
					 \frac{\partial x'^I_{2} }{\partial x_1} &
					 \frac{\partial y'_1     }{\partial x_1} &
					 \frac{\partial y'^I_1   }{\partial x_1} &
					 \frac{\partial y'_{2c}  }{\partial x_1} &
					 \frac{\partial y'^I_{2c}}{\partial x_1} &
					 \frac{\partial y'_{2t}  }{\partial x_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x^I_1} &
					 \frac{\partial x'^I_1   }{\partial x^I_1} &
					 \frac{\partial x'^{II}_1}{\partial x^I_1} &
					 \frac{\partial x'_{2}   }{\partial x^I_1} &
					 \frac{\partial x'^I_{2} }{\partial x^I_1} &
					 \frac{\partial y'_1     }{\partial x^I_1} &
					 \frac{\partial y'^I_1   }{\partial x^I_1} &
					 \frac{\partial y'_{2c}  }{\partial x^I_1} &
					 \frac{\partial y'^I_{2c}}{\partial x^I_1} &
					 \frac{\partial y'_{2t}  }{\partial x^I_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x^{II}_1} &
					 \frac{\partial x'^I_1   }{\partial x^{II}_1} &
					 \frac{\partial x'^{II}_1}{\partial x^{II}_1} &
					 \frac{\partial x'_{2}   }{\partial x^{II}_1} &
					 \frac{\partial x'^I_{2} }{\partial x^{II}_1} &
					 \frac{\partial y'_1     }{\partial x^{II}_1} &
					 \frac{\partial y'^I_1   }{\partial x^{II}_1} &
					 \frac{\partial y'_{2c}  }{\partial x^{II}_1} &
					 \frac{\partial y'^I_{2c}}{\partial x^{II}_1} &
					 \frac{\partial y'_{2t}  }{\partial x^{II}_1} \\[1ex]

					\frac{\partial x'_1      }{\partial x_2} &
					 \frac{\partial x'^I_1   }{\partial x_2} &
					 \frac{\partial x'^{II}_1}{\partial x_2} &
					 \frac{\partial x'_{2}   }{\partial x_2} &
					 \frac{\partial x'^I_{2} }{\partial x_2} &
					 \frac{\partial y'_1     }{\partial x_2} &
					 \frac{\partial y'^I_1   }{\partial x_2} &
					 \frac{\partial y'_{2c}  }{\partial x_2} &
					 \frac{\partial y'^I_{2c}}{\partial x_2} &
					 \frac{\partial y'_{2t}  }{\partial x_2} \\[1ex]

					\frac{\partial x'_1      }{\partial x'^I_{2}} &
					 \frac{\partial x'^I_1   }{\partial x'^I_{2}} &
					 \frac{\partial x'^{II}_1}{\partial x'^I_{2}} &
					 \frac{\partial x'_{2}   }{\partial x'^I_{2}} &
					 \frac{\partial x'^I_{2} }{\partial x'^I_{2}} &
					 \frac{\partial y'_1     }{\partial x'^I_{2}} &
					 \frac{\partial y'^I_1   }{\partial x'^I_{2}} &
					 \frac{\partial y'_{2c}  }{\partial x'^I_{2}} &
					 \frac{\partial y'^I_{2c}}{\partial x'^I_{2}} &
					 \frac{\partial y'_{2t}  }{\partial x'^I_{2}} \\[1ex]

					\frac{\partial x'_1      }{\partial y_1} &
					 \frac{\partial x'^I_1   }{\partial y_1} &
					 \frac{\partial x'^{II}_1}{\partial y_1} &
					 \frac{\partial x'_{2}   }{\partial y_1} &
					 \frac{\partial x'^I_{2} }{\partial y_1} &
					 \frac{\partial y'_1     }{\partial y_1} &
					 \frac{\partial y'^I_1   }{\partial y_1} &
					 \frac{\partial y'_{2c}  }{\partial y_1} &
					 \frac{\partial y'^I_{2c}}{\partial y_1} &
					 \frac{\partial y'_{2t}  }{\partial y_1} \\[1ex]

					\frac{\partial x'_1      }{\partial y^I_1} &
					 \frac{\partial x'^I_1   }{\partial y^I_1} &
					 \frac{\partial x'^{II}_1}{\partial y^I_1} &
					 \frac{\partial x'_{2}   }{\partial y^I_1} &
					 \frac{\partial x'^I_{2} }{\partial y^I_1} &
					 \frac{\partial y'_1     }{\partial y^I_1} &
					 \frac{\partial y'^I_1   }{\partial y^I_1} &
					 \frac{\partial y'_{2c}  }{\partial y^I_1} &
					 \frac{\partial y'^I_{2c}}{\partial y^I_1} &
					 \frac{\partial y'_{2t}  }{\partial y^I_1} \\[1ex]

					\frac{\partial x'_1      }{\partial y_{2c}} &
					 \frac{\partial x'^I_1   }{\partial y_{2c}} &
					 \frac{\partial x'^{II}_1}{\partial y_{2c}} &
					 \frac{\partial x'_{2}   }{\partial y_{2c}} &
					 \frac{\partial x'^I_{2} }{\partial y_{2c}} &
					 \frac{\partial y'_1     }{\partial y_{2c}} &
					 \frac{\partial y'^I_1   }{\partial y_{2c}} &
					 \frac{\partial y'_{2c}  }{\partial y_{2c}} &
					 \frac{\partial y'^I_{2c}}{\partial y_{2c}} &
					 \frac{\partial y'_{2t}  }{\partial y_{2c}} \\[1ex]

					\frac{\partial x'_1      }{\partial y^I_{2c}} &
					 \frac{\partial x'^I_1   }{\partial y^I_{2c}} &
					 \frac{\partial x'^{II}_1}{\partial y^I_{2c}} &
					 \frac{\partial x'_{2}   }{\partial y^I_{2c}} &
					 \frac{\partial x'^I_{2} }{\partial y^I_{2c}} &
					 \frac{\partial y'_1     }{\partial y^I_{2c}} &
					 \frac{\partial y'^I_1   }{\partial y^I_{2c}} &
					 \frac{\partial y'_{2c}  }{\partial y^I_{2c}} &
					 \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
					 \frac{\partial y'_{2t}  }{\partial y^I_{2c}} \\[1ex]

					\frac{\partial x'_1      }{\partial y_{2t}} &
					 \frac{\partial x'^I_1   }{\partial y_{2t}} &
					 \frac{\partial x'^{II}_1}{\partial y_{2t}} &
					 \frac{\partial x'_{2}   }{\partial y_{2t}} &
					 \frac{\partial x'^I_{2} }{\partial y_{2t}} &
					 \frac{\partial y'_1     }{\partial y_{2t}} &
					 \frac{\partial y'^I_1   }{\partial y_{2t}} &
					 \frac{\partial y'_{2c}  }{\partial y_{2t}} &
					 \frac{\partial y'^I_{2c}}{\partial y_{2t}} &
					 \frac{\partial y'_{2t}  }{\partial y_{2t}} \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation*}

\noindent As before, we can isolate the candidate leading eigenvalues associated with the spread of the inversion by rearranging the Jacobian to give the following block triangular matrix \cite[Supplement~to~Primer~2]{OttoDay2007}:

\begin{equation}
	\mathbb{J}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{A} = \left( \begin{array}{ccc} 
										\frac{\partial x'_1}{\partial x_1} & \cdots & \frac{\partial y'_{2t}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'_1}{\partial y_{2t}} & \cdots & \frac{\partial y'_{2t}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) &
				\mathbf{B} = \left( \begin{array}{ccc} 
										\frac{\partial x'^I_1}{\partial x_1} & \cdots & \frac{\partial y'^I_{2c}}{\partial x_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'^I_{1}}{\partial y_{2t}} & \cdots & \frac{\partial y'^I_{2c}}{\partial y_{2t}} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{C} = \left( \begin{array}{ccc} 
								0      & \cdots & 0     \\[1ex]
								\vdots & \ddots & \vdots\\[1ex]
								0      & \cdots & 0     \\[1ex]
							\end{array} \right) &
				\mathbf{D} = \left( \begin{array}{ccc} 
										\frac{\partial x'^I_1}{\partial x^I_1} & \cdots & \frac{\partial y'^I_{2c}}{\partial x^I_1} \\[1ex]
										\vdots & \ddots & \vdots \\[1ex]
										\frac{\partial x'^I_{1}}{\partial y^I_{2c}} & \cdots & \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} \\[1ex]
									\end{array} \right) \\[1ex]
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation}

\noindent The eigenvalues calculations can be further simplified by rearranging submatrix $\mathbf{D}$ so that it is also of block triangular form (this does not affect the eigenvalues associated with changes in non-inversion genotype frequencies from submatrix $\mathbf{A}$). Ultimately, we have 
\begin{equation}
	\mathbf{D}_{\text{BT}} = \left( \begin{array}{cc}
		\mathbf{W} = \left( \begin{array}{cc}
										\frac{\partial x'^I_1   }{\partial x^I_1} &
										 \frac{\partial x'^I_{2}}{\partial x^I_1} \\[1ex]
										\frac{\partial x'^I_1   }{\partial x^I_2} &
										 \frac{\partial x'^I_{2}}{\partial x^I_2} \\[1ex]
									\end{array} \right) &
				\mathbf{X} = \left( \begin{array}{ccc} 
										\frac{\partial y'^I_1    }{\partial x^I_1} &
										 \frac{\partial y'^I_{2c}}{\partial x^I_1} &
										 \frac{\partial x'^{II}_1}{\partial x^I_1} \\[1ex]
										\frac{\partial y'^I_1    }{\partial x^I_2} &
										 \frac{\partial y'^I_{2c}}{\partial x^I_2} &
										 \frac{\partial x'^{II}_1}{\partial x^I_2} \\[1ex]
									\end{array} \right) \\
		\!\rule{0pt}{0pt} \\
		\mathbf{Y} = \left( \begin{array}{cc} 
								0      & 0     \\[1ex]
								0      & 0     \\[1ex]
								0      & 0     \\[1ex]
							\end{array} \right) &
				\mathbf{Z} = \left( \begin{array}{ccc} 
										\frac{\partial y'^I_1    }{\partial y^I_1} &
										 \frac{\partial y'^I_{2c}}{\partial y^I_1} &
										 \frac{\partial x'^{II}_1}{\partial y^I_1} \\[1ex]
										\frac{\partial y'^I_1    }{\partial y^I_{2c}} &
										 \frac{\partial y'^I_{2c}}{\partial y^I_{2c}} &
										 \frac{\partial x'^{II}_1}{\partial y^I_{2c}} \\[1ex]
										\frac{\partial y'^I_1    }{\partial x^{II}_1} &
										 \frac{\partial y'^I_{2c}}{\partial x^{II}_1} &
										 \frac{\partial x'^{II}_1}{\partial x^{II}_1} \\[1ex]
									\end{array} \right) \\
				 \end{array} \right)_{\substack{
										x_i = \hat{x}_i \\
										y_j = \hat{y}_j \\
										x^{I}_{1} = x^{II}_{1} = x^I_2 = y^I_1 = y^I_3 = 0}} \numberthis
\end{equation}

\noindent The relevant candidate leading eigenvalue comes from submatrix $\mathbf{W}$:

\begin{equation}
	\lambda_{I} = \big( \alpha + \beta \big)
\end{equation}

where $\alpha = \frac{\partial x'^I_1}{\partial x^I_1} = \frac{\partial x'^I_{2}}{\partial x^I_1}$ and $\beta = \frac{\partial x'^I_1}{\partial x^I_2} = \frac{\partial x'^I_{2}}{\partial x^I_2}$. Transforming $\lambda_I$ onto a coordinate system of allele frequencies on the three chromosome classes, $X_f$, $X_m$, and $Y$, yields:


\begin{equation}
	\lambda_I = \frac{w_{f,2} + \hat{X}_f(w_{f,1} - w_{f,2})} {w_{f,3}(1 - \hat{X}_f)(1 - \hat{X}_m) + w_{f,1} \hat{X}_f \hat{Y} + w_{f,2} (\hat{X}_f + \hat{X}_m - 2 \hat{X}_f \hat{X}_m)}
\end{equation}

\noindent If $\lambda_I > 1$, and all non-inversion genotypes are initially at equilibrium ($x_i = \hat{x}_i$, $y_j = \hat{y}_j$), as we have assumed, $\lambda_I$ will necessarily be the leading eigenvalue of the system of recursions. However, for inversions on the X chromosome there exist internal equilibria for the inversion genotypes which we have confirmed with deterministic iteration of the recursions (\hl{e.g., see fig.~4C,D in the main text}). 

Our approximation of $s_I \approx \lambda - 1$, must be interpreted cautiously for the model of X chromosome inversions. In particular, $2 s_I$ does not provide a valid approximation of the fixation probability for the inversion, but rather an optimistic approximation of the probability that the inversion escapes stochastic loss when rare to approach an internal deterministic equilibrium frequency. In finite populations, $2 s_I$ will overestimate the probability that a new inversion is maintained as a polymorphism in the long-term due to chance extinction of inversions maintained at low equilibrium frequencies. Hence, inversions on the X chromosome are less likely to be maintained as stable polymorphisms than suggested by the deterministic simulations (\hl{ figure 4C,D})

\newpage



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Haploid \& Diploid Selection} \label{AppC}
 \renewcommand{\theequation}{C\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{C\arabic{figure}}
 \setcounter{figure}{0}

The recursions and invasion analysis for the two-locus models of inversions with selection during both haploid and diploid phases are very similar to those described above for sexual antagonism. For brevity, we refer readers to the supplementary Mathematica notebook files (.nb) available in the Online Supplementary Material. There we present the recursions and a brief walk-through of the invasion analyses for models of rare inversions spanning the SDR on the Y and X chromosomes.
\bigskip

\hl{Excerpt from Version 1}
We have so far considered selection in the diploid phase only. However, sexually reproducing eukaryotes have alternating life-cycles with a reduced (e.g., haploid) and doubled (e.g., diploid) phase \citep{Strasburger1894, Roe1975}. Moreover, haploid selection can play an important role in maintaining genetic polymorphisms \citep{ImmlerOtto2011}, as well as facilitating sex chromosome turnovers \citep{OttoScottOsmond2018} and transitions between sex determination systems \citep{MuralidharVeller2018}. The models summarized above for sexually antagonistic selection can be easily extended to incorporate haploid selection, although this opens up many new possible selection scenarios \citep{ImmlerOtto2011, OttoScottOsmond2018}. For simplicity and brevity, we briefly consider a general model of haploid and diploid selection, and illustrate the model predictions with a single representative case of ploidally antagonistic selection. The critical difference between this model and those of SA selection above with respect to the fixation probability of inversions is that $s_I$ is now a function of fitness during both haploid and diploid phases. 

Consider the simple case of a rare inversion capturing the SDR and a single selected locus on the Y chromosome. To keep the model general, and relatively simple, we retain arbitrary fitness expressions for the haploid ($v^{f}_{1}$, $v^{f}_{2}$ for female, and $v^{m}_{1}$, $v^{m}_{2}$ for male gametes, respectively) and diploid genotypes ($w^{f}_{11}$, $w^{f}_{12}$, $w^{f}_{22}$ in females, and $w^{m}_{11}$, $w^{m}_{12}$, $w^{m}_{22}$ in males). For Y-linked inversions capturing a single selected locus, the approximate selection coefficient for the rare inversion under arbitrary linkage ($0 \leq r \leq 1/2$) is:

\begin{equation}\label{eq:sI-HapDip-2LocLinked}
	s_I \approx \frac{\big( v^{f}_{2} \hat{X}_f (v^{m}_{2} w^{m}_{22} -v^{m}_{1} w^{m}_{12} ) - v^{f}_{1} (1 - \hat{X}_f) (v^{m}_{1} w^{m}_{11} - v^{m}_{2} w^{m}_{12}) \big) (1 - \hat{Y})} {v^{f}_{1} (1 - \hat{X}_f) \big( v^{m}_{1} w^{m}_{11} (1 - \hat{Y}) + v^{m}_{2} w^{m}_{12} \hat{Y} \big) + v^{f}_{2} \hat{X}_f \big( v^{m}_{2} w^{m}_{22} \hat{Y} + v^{m}_{1} (w^{m}_{12} - w^{m}_{12} \hat{Y}) \big)},
\end{equation}

\noindent where $\hat{Y}$ and $\hat{X_f}$ represent the equilibrium frequency of the $2^{\text{nd}}$ allele at the selected locus in Y chromosomes and X chromosomes in females when the inversion originates. For a rare inversion to invade, $s_I > 0$ must be satisfied for Eq(\ref{eq:sI-HapDip-2LocLinked}), which requires that the net fitness effect of the inversion across haploid and diploid phases is male-beneficial, or there is sufficient linkage disequilibrium to offset a female-bias in selection. For example, under weak ploidally antagonistic selection with additive fitness in the diploid phase (see Table.~\hl{XXX}), an inversion capturing the SDR and the $2^{\text{nd}}$ allele at the selected locus can invade when $s > 2 t + O(s^2,t^2)$.  

Calculation of the fixation probability, and the effects of deleterious mutations are the same as for the sexually antagonistic model described above, and result in qualitatively similar predictions. Selection, whether during the haploid, diploid, or both phases, influences the fixation probability of differently sized inversions similarly, and should favour small to intermediately sized inversions.

For X-linked inversions, the addition of selection during the haploid phase expands the conditions under which an inversion can be maintained as a balanced polymorphism. The overall result parallels that for sexually antagonistic selection: while X-linked inversions can contribute to reduced recombination between sex chromosomes, they are far less likely to fix and thereby form evolutionary strata than Y-linked inversions. \vspace{12pt}

\noindent {\itshape Key result: The fixation probability of different sized inversions is a similar function for selection occuring during the haploid, diploid, or both phases. Inversion length will therefore provide little insight into when during the life cycle indirect selection for suppressed recombination occurs.} 





%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Expected length distributions of evolutionary strata} \label{AppC:DistFixedInv}
 \renewcommand{\theequation}{D\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{D\arabic{figure}}
 \setcounter{figure}{0}
%%%%%%%%%%%%%%%%%%%%%%%%

With expressions for the fixation probability of new inversions under different evolutionary scenarios in hand, it is possible to derive the corresponding expected distributions of fixed inversion sizes. Following \citet{vanValenLevins1968, Santos1986}, and \citet{ConnallonOlito2020}, the proportion of fixed inversions of length $x$ is given by 

\begin{equation} \label{eq:generalInvSizeModel}
	g(x) = \frac{\Pr(\text{fix} \mid x) f(x)} {\int \Pr(\text{fix} \mid x) f(x)\,dx},
\end{equation}

\noindent where $f(x)$ is the probability of a new inversion of length $x$, and $\Pr(\text{fix} \mid x)$ is the fixation probability given in Eq(\ref{eq:generalPrFix}) with appropriate substitutions made for each selection scenario. $x\int \Pr(\text{fix} \mid x) f(x)\,dx$ gives the mean length of fixed inversions. Little is known about how the mutational process for new inversions shapes $f(x)$, and we therefore examine two scenarios representing plausible extremes to illustrate the spectrum of possible outcomes.

On one hand, if inversion breakpoints are distributed uniformly across the chromosome arm containing the SLR, then $f(x) = 2(1 - x)$, an extreme scenario we refer to as the "random breakpoint" model \citep{vanValenLevins1968}. On the other hand, if inversion breakpoints tend to be clustered, for example in chromosomal regions with repetitive sequences, the resulting enrichment of smaller new inversions can be modeled phenomenologically using a truncated exponential distribution:

\begin{equation} \label{eq:truncExp}
	f(x) = \frac{ \lambda e^{-\lambda x}} {1 - e^{-\lambda}},
\end{equation}

\noindent where $\lambda$ is the exponential rate parameter \citep{PevznerTesler2003, PengPevznerTesler2006, ChengKirkpatrick2019,ConnallonOlito2020}. For strongly skewed distributions (e.g., $\lambda > 10$, as we assume here), the truncation effect is negligible, and $f(x)$ is approximately equal to the numerator of Eq(\ref{eq:truncExp}). We refer to this other extreme as the "exponential model". Two key results emerge from the expected distributions of evolutionary strata length.


 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.65]{./expDistFig}
 \caption{Probability density functions for fixed inversions expanding the SLR on Y chromosomes ($g(x)$ from Eq[\ref{eq:generalInvSizeModel}]). For clarity, we show results for the model of Sexual Anatagonism with an initially unlinked SA locus ($r = 1/2$). Results are shown for the same parameter values as in Fig.~\ref{fig:fixProbFig} and Fig.~\ref{fig:probCatchFixFig}: $U_d = 0.2$, $s_d = 0.02$, and $s_I = 0.02$ for Neutral and Beneficial inversion scenarios, and $s_f = s_m = 0.05$, $U_d = 0.1$, $s_d = 0.01$, $A = 1$ for the Sex Antagonism scenario.}
 \label{fig:ExpectedDistFig}
 \end{figure}


First, the results are again strongly influenced by the location of the SLR. When the SLR is located in the center of the chromosome arm, neutral inversions are expected to give rise to a triangular distribution of evolutionary strata lengths, with a mean at $x = 1/2$ (Fig.\ref{fig:ExpectedDistFig}A). Both beneficial inversions and those capturing SA loci yield largely overlapping distributions of smaller inversions, although the distribution for sex antagonism has a distinctly heavier tail under the random breakpoint model. The differences between the distributions for neutral and selected inversions becomes exaggerated when the SLR is located near one end of the chromosome arm (Fig.\ref{fig:ExpectedDistFig}C). The distribution for neutral inversions now has three distinct regions, yielding a plateau shape, while those for beneficial and sex antagonistic inversions become increasingly skewed and overlapping. The unusual form of the length distributions for neutral inversions under the random breakpoint model results from the appearance of $(1 - x)$ terms in both $f(x)$ and $\Pr(\text{SLR} \mid x)$, which cancel in different ranges of $x$ depending on the location of the SLR.  

Second, the expected length distributions of evolutionary strata are sensitive to the form of $f(x)$. In contrast to the random breakpoints model, when inversion breakpoints are clustered the predicted distributions of strata length are highly overlapping for all three selection scenarios, and are practically indistinguishable when the SLR is located near the end of the chromosome arm (Fig.~\ref{fig:ExpectedDistFig}B,D). \vspace{12pt}

\noindent {\itshape Key result: Two dominant factors influence the expected length distribution of fixed inversions expanding the SLR: the location of the ancestral SLR on the chromosome arm, and the length distribution of new inversions. While different selection scenarios are expected to result in distinct distributions under a random breakpoint model, the length distributions become practically indistinguishable under an exponential model of new inversion lengths.} 

% \begin{figure}[h!]
% \includegraphics[scale=0.7]{./recSimFig_add}
% \caption{Haplotype recursions evaluated at QE, and the resulting invasion conditions, approximate the evolutionary trajectory of the full genotypic recursions very well under additive allelic effects, even under strong selection. Predicted regions of SA polymorphism based on deterministic simulations using the genotypic recursions Eq(2) are compared against the outcome of the invasion analysis for the haplotype recursions using the QE approximation (Eq 1) across a gradient of selfing ($C$) and recombination ($r$) rates. Green points indicate parameter conditions where deterministic simulations of the genotypic recursions (Sim.) and the invasion analysis based on eigenvalues for the QE haplotype approximations (Eig.) both predicted polymorphism. Red points indicate regions where the Sim.~predicted polymorphism but the Eig.~did not; and blue points indicate the opposite. The proportion of each outcome is shown in the upper left corner of each panel. Black solid lines show the invasion conditions based on the haplotype recursions using the QE approximation for the given values of $C$ and $r$, with lines drawn for both the % two-locus and single-locus invasion conditions when $r > 0$.}
% \label{fig:addSim}
% \end{figure}
% \newpage{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Supplementary Figures} \label{SuppFigs}
 \renewcommand{\theequation}{S\arabic{equation}}
 \setcounter{equation}{0}
 \renewcommand{\thefigure}{S\arabic{figure}}
 \setcounter{figure}{0}

\begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.56]{./SchematicFig-all}
 \caption{The spread of inversions capturing the SDR on a Y chromosome under three of the main scenarios described in the main text: \textbf{(A)} beutral inversions, \textbf{(B)} directly beneficial inversions (e.g., beneficial break-point effects), and \textbf{(C)} indirectly beneficial (sexual antagonism). From left to right, each illustration depicts a sample of Y chromosomes at three time points during the spread of an inversion, highlighting several key features of the theoretical models. The left-hand panels show key outcomes when new inversions first arise, and emphasize our assumption that inversions capturing deleterious mutations (panels A-C) or female-beneficial alleles (panel C) are unlikely to spread (indicated by skull and crossbones). The center panels illustrate the spread of the inversion and highlight that the fitness advantage enjoyed by mutation-free inversions decays over time as they accumulate new deleterious mutations. Finally, the right-hand panels illustrate the eventual fixation of the inversion and the resulting expansion of the SDR region. In all panels, the chromosomal region where $0 \leq r < 1/2$ (the sl-PAR) is indicated by blue shading.}
 \label{fig:diagramFig}
 \end{figure}

\newpage


\begin{figure}[htbp]
 \centering
 \includegraphics[width=\linewidth]{./SuppFigPrCatchSDR}
 \caption{Probability that a new inversion spans the SDR as a function of inversion size, $x$, and the location of the SDR on the chromosome arm (see Eq(16) and corresponding assumptions in the main text). We illustrate the same scenarios analysed in the main article: \textbf{(A)} the SDR is located at the exact middle of the chromosome arm, $\text{SDR}_\text{loc} = 1/2$, and \textbf{(B)} the SDR is located closer to the centromere $\text{SDR}_\text{loc} = 1/10$ (this choice is arbitrary, and results are identical if the SDR is located equally close to the telomere, $\text{SDR}_\text{loc} = 9/10$). To aid comparison, an illustration of a Y chromosome arm with the corresponding SDR locations are drawn above each plot. The form of $\Pr (\text{SDR} \mid x)$ changes as $x$ increases, and shaded regions indicate the relevant regions of parameter space where the piecewise function changes form. Darker grey indicates where $x \leq y_1,y_3$, light grey indicates $y_1 < x < y_3$, and no shading indicates $x > y_1, y_3$. Note that the region corresponding to $y_1 > x > y_3$ is not applicable for the values of $\text{SDR}_\text{loc}$ examined here. The corresponding piecewise function for each region is shown in boxes.}
 \label{fig:PrCatchFixFig}
 \end{figure}


%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%
\bibliography{../bibliography-inversionSize-ProtoSexChrom}

\newpage


\end{document}
